{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Validation results\n",
    "\n",
    "In this notebook we examine how well Oxford Nanopore SNP calling agrees with SNP calling from Illumina data.\n",
    "\n",
    "The parameters we use are:\n",
    "    * nanopolish: 5-mer alignment, 6-mer variant calling, reverse HMM fix"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "ROOTDIR=\"/mnt/ebola/ebov\"\n",
    "import os\n",
    "os.environ['PATH'] = os.getenv('PATH') + ':' + ROOTDIR + '/scripts'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/ebola/ebov\n"
     ]
    }
   ],
   "source": [
    "%cd $ROOTDIR"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reading ../models/6mer/ont_complement.pop1.model\n",
      "reading ../models/6mer/ont_complement.pop2.model\n",
      "reading ../models/6mer/ont_template.model\n",
      "reading ../models/6mer/ont_complement.pop1.model\n",
      "reading ../models/6mer/ont_complement.pop2.model\n",
      "reading ../models/6mer/ont_template.model\n",
      "reading ../models/6mer/ont_complement.pop1.model\n",
      "reading ../models/6mer/ont_complement.pop2.model\n",
      "reading ../models/6mer/ont_template.model\n",
      "reading ../models/6mer/ont_complement.pop1.model\n",
      "reading ../models/6mer/ont_complement.pop2.model\n",
      "reading ../models/6mer/ont_template.model\n",
      "reading ../models/6mer/ont_complement.pop1.model\n",
      "reading ../models/6mer/ont_complement.pop2.model\n",
      "reading ../models/6mer/ont_template.model\n",
      "reading ../models/6mer/ont_complement.pop1.model\n",
      "reading ../models/6mer/ont_complement.pop2.model\n",
      "reading ../models/6mer/ont_template.model\n",
      "reading ../models/6mer/ont_complement.pop1.model\n",
      "reading ../models/6mer/ont_complement.pop2.model\n",
      "reading ../models/6mer/ont_template.model\n",
      "reading ../models/6mer/ont_complement.pop1.model\n",
      "reading ../models/6mer/ont_complement.pop2.model\n",
      "reading ../models/6mer/ont_template.model\n",
      "reading ../models/6mer/ont_complement.pop1.model\n",
      "reading ../models/6mer/ont_complement.pop2.model\n",
      "reading ../models/6mer/ont_template.model\n",
      "reading ../models/6mer/ont_complement.pop1.model\n",
      "reading ../models/6mer/ont_complement.pop2.model\n",
      "reading ../models/6mer/ont_template.model\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "+ ref_prefix=EM_079517\n",
      "+ sample=076383_180Genomes_11rx\n",
      "+ poretools_dir=076383_180Genomes_11rx\n",
      "+ sample_tag=076383_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "++ tempfile\n",
      "+ tmp=/tmp/fileNIZFVX\n",
      "+ poretools fastq --type 2D ../data/076383_180Genomes_11rx/pass\n",
      "+ poretools fasta --type 2D ../data/076383_180Genomes_11rx/pass\n",
      "+ '[' hq == hq ']'\n",
      "+ poretools fastq --type 2D --high-quality ../data/076383_180Genomes_11rx/fail\n",
      "+ poretools fasta --type 2D --high-quality ../data/076383_180Genomes_11rx/fail\n",
      "+ '[' '!' na == na ']'\n",
      "+ mv /tmp/fileNIZFVX 076383_180Genomes_11rx_hq.fastq\n",
      "+ mv /tmp/fileNIZFVX.fasta 076383_180Genomes_11rx_hq.fasta\n",
      "+ '[' '!' -e ../refs/EM_079517.fasta.amb ']'\n",
      "+ rm -rf jobTree_align_EM_079517_076383_180Genomes_11rx_hq\n",
      "+ marginAlign 076383_180Genomes_11rx_hq.fastq ../refs/EM_079517.fasta EM_079517_076383_180Genomes_11rx_hq.sam --jobTree jobTree_align_EM_079517_076383_180Genomes_11rx_hq --inputModel ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446200766.47 : Going to chain sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_076383_180Genomes_11rx_hq/jobs/gTD0/tmp_45cOls4PUm/temp.sam\n",
      "Got message from job at time: 1446200766.48 : Going to realign sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_076383_180Genomes_11rx_hq/jobs/gTD1/tmp_KwkejJAYJG/temp.sam to create output sam file: EM_079517_076383_180Genomes_11rx_hq.sam     with match gamma 0.5 and gap gamma 0.0 and model ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446200843.28 : Realigned sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_076383_180Genomes_11rx_hq/jobs/gTD1/tmp_KwkejJAYJG/temp.sam to create output sam file: EM_079517_076383_180Genomes_11rx_hq.sam\n",
      "+ align_trim.py\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_076383_180Genomes_11rx_hq_marginalign.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_076383_180Genomes_11rx_hq_marginalign.sorted.bam\n",
      "+ rm 076383_180Genomes_11rx_hq.fasta.fast5.fofn\n",
      "+ nanopolish eventalign --reads 076383_180Genomes_11rx_hq.fasta -b EM_079517_076383_180Genomes_11rx_hq_marginalign.sorted.bam -g ../refs/EM_079517.fasta --sam\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_076383_180Genomes_11rx_hq_np.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_076383_180Genomes_11rx_hq_np.sorted.bam\n",
      "+ nanopolish variants --models-fofn offset_models.fofn --progress -t 1 --reads 076383_180Genomes_11rx_hq.fasta -o np_EM_079517_076383_180Genomes_11rx_hq.vcf -b EM_079517_076383_180Genomes_11rx_hq_marginalign.sorted.bam -e EM_079517_076383_180Genomes_11rx_hq_np.sorted.bam -g ../refs/EM_079517.fasta -vv -w EM_079517:0-20000 --snp\n",
      "TODO: train model\n",
      "TODO: filter data\n",
      "EM_079517\t40\t.\tG\tT\t3.9\tPASS\tBaseCalledReadsWithVariant=115;BaseCalledFrequency=0.416667;TotalReads=2;SupportingReads=2;SupportFraction=1;Template=1.975,-nan;Comp.P1=-nan,-nan;Comp.P2=1.93,-nan;ReadCounts=1,0,0,0,1,0\n",
      "EM_079517\t213\t.\tG\tA\t3102.9\tPASS\tBaseCalledReadsWithVariant=198;BaseCalledFrequency=0.709677;TotalReads=698;SupportingReads=626;SupportFraction=0.896848;Template=7.273,3.91;Comp.P1=2.572,-0.3316;Comp.P2=4.442,2.586;ReadCounts=167,182,9,16,173,151\n",
      "EM_079517\t800\t.\tC\tT\t1151.4\tPASS\tBaseCalledReadsWithVariant=56;BaseCalledFrequency=0.861538;TotalReads=184;SupportingReads=174;SupportFraction=0.945652;Template=8.199,2.543;Comp.P1=4.95,1.175;Comp.P2=11.21,3.607;ReadCounts=42,50,2,8,48,34\n",
      "EM_079517\t1849\t.\tT\tC\t5489.2\tPASS\tBaseCalledReadsWithVariant=253;BaseCalledFrequency=0.702778;TotalReads=1072;SupportingReads=838;SupportFraction=0.781716;Template=8.992,-0.7616;Comp.P1=5.771,-0.9448;Comp.P2=10.54,1.845;ReadCounts=281,255,8,4,247,277\n",
      "EM_079517\t2124\t.\tG\tA\t2549.2\tPASS\tBaseCalledReadsWithVariant=145;BaseCalledFrequency=0.767196;TotalReads=406;SupportingReads=332;SupportFraction=0.817734;Template=11.13,2.202;Comp.P1=4.625,3.294;Comp.P2=5.182,4.916;ReadCounts=126,77,24,17,53,109\n",
      "EM_079517\t2968\t.\tT\tC\t2484.8\tPASS\tBaseCalledReadsWithVariant=112;BaseCalledFrequency=0.823529;TotalReads=290;SupportingReads=264;SupportFraction=0.910345;Template=12.47,8.942;Comp.P1=5.699,7.508;Comp.P2=7.996,4.775;ReadCounts=78,67,18,17,49,61\n",
      "EM_079517\t6056\t.\tA\tC\t869.6\tPASS\tBaseCalledReadsWithVariant=58;BaseCalledFrequency=0.90625;TotalReads=140;SupportingReads=131;SupportFraction=0.935714;Template=7.671,2.818;Comp.P1=3.94,-nan;Comp.P2=6.261,8.702;ReadCounts=34,36,6,0,30,34\n",
      "EM_079517\t6283\t.\tC\tT\t655.7\tPASS\tBaseCalledReadsWithVariant=47;BaseCalledFrequency=0.839286;TotalReads=136;SupportingReads=127;SupportFraction=0.933824;Template=8.094,2.507;Comp.P1=4.963,-nan;Comp.P2=7.418,1.7;ReadCounts=34,34,6,0,28,34\n",
      "EM_079517\t6909\t.\tA\tT\t1071.7\tPASS\tBaseCalledReadsWithVariant=66;BaseCalledFrequency=0.942857;TotalReads=144;SupportingReads=138;SupportFraction=0.958333;Template=7.114,8.574;Comp.P1=5.321,-nan;Comp.P2=7.415,7.181;ReadCounts=37,35,8,0,27,37\n",
      "EM_079517\t8928\t.\tA\tC\t975.1\tPASS\tBaseCalledReadsWithVariant=114;BaseCalledFrequency=0.612903;TotalReads=390;SupportingReads=327;SupportFraction=0.838462;Template=2.388,5.195;Comp.P1=2.014,3.154;Comp.P2=1.458,0.7828;ReadCounts=101,94,6,11,88,90\n",
      "EM_079517\t9264\t.\tA\tG\t1544.3\tPASS\tBaseCalledReadsWithVariant=75;BaseCalledFrequency=0.78125;TotalReads=388;SupportingReads=333;SupportFraction=0.858247;Template=2.285,6.173;Comp.P1=-1.465,8.763;Comp.P2=1.595,5.696;ReadCounts=103,91,6,11,85,92\n",
      "EM_079517\t10315\t.\tC\tT\t8108.9\tPASS\tBaseCalledReadsWithVariant=559;BaseCalledFrequency=0.699625;TotalReads=1884;SupportingReads=1779;SupportFraction=0.944268;Template=3.593,3.464;Comp.P1=3.528,2.761;Comp.P2=5.307,5.212;ReadCounts=550,392,80,12,312,538\n",
      "EM_079517\t13856\t.\tA\tG\t4857.5\tPASS\tBaseCalledReadsWithVariant=250;BaseCalledFrequency=0.905797;TotalReads=630;SupportingReads=599;SupportFraction=0.950794;Template=5.606,9.389;Comp.P1=4.397,7.376;Comp.P2=7.86,8.968;ReadCounts=168,147,31,12,116,156\n",
      "EM_079517\t15660\t.\tT\tC\t10972.9\tPASS\tBaseCalledReadsWithVariant=711;BaseCalledFrequency=0.711;TotalReads=2137;SupportingReads=1896;SupportFraction=0.887225;Template=8.966,1.627;Comp.P1=2.661,1.363;Comp.P2=8.517,2.297;ReadCounts=532,537,73,39,464,492\n",
      "EM_079517\t15963\t.\tG\tA\t1734.1\tPASS\tBaseCalledReadsWithVariant=49;BaseCalledFrequency=0.890909;TotalReads=210;SupportingReads=202;SupportFraction=0.961905;Template=3.438,12.94;Comp.P1=1.526,5.874;Comp.P2=5.003,12.64;ReadCounts=49,56,9,4,47,45\n",
      "EM_079517\t16514\t.\tG\tA\t2766.2\tPASS\tBaseCalledReadsWithVariant=114;BaseCalledFrequency=0.876923;TotalReads=271;SupportingReads=253;SupportFraction=0.933579;Template=6.568,15.22;Comp.P1=2.596,9.992;Comp.P2=4.255,15.95;ReadCounts=70,65,9,11,57,59\n",
      "EM_079517\t17142\t.\tT\tC\t4079.7\tPASS\tBaseCalledReadsWithVariant=195;BaseCalledFrequency=0.859031;TotalReads=498;SupportingReads=469;SupportFraction=0.941767;Template=13.11,8.266;Comp.P1=4.138,4.06;Comp.P2=6.113,6.449;ReadCounts=107,142,7,17,135,90\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=076383_180Genomes_11rx\n",
      "+ poretools_dir=076383_180Genomes_11rx\n",
      "+ sample_tag=076383_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "++ tempfile\n",
      "+ tmp=/tmp/file4QGaGX\n",
      "+ poretools fastq --type 2D ../data/076383_180Genomes_11rx/pass\n",
      "+ poretools fasta --type 2D ../data/076383_180Genomes_11rx/pass\n",
      "+ '[' hq == hq ']'\n",
      "+ poretools fastq --type 2D --high-quality ../data/076383_180Genomes_11rx/fail\n",
      "+ poretools fasta --type 2D --high-quality ../data/076383_180Genomes_11rx/fail\n",
      "+ '[' '!' na == na ']'\n",
      "+ mv /tmp/file4QGaGX 076383_180Genomes_11rx_hq.fastq\n",
      "+ mv /tmp/file4QGaGX.fasta 076383_180Genomes_11rx_hq.fasta\n",
      "+ '[' '!' -e ../refs/EM_079517_mut30_2.fasta.amb ']'\n",
      "+ rm -rf jobTree_align_EM_079517_mut30_2_076383_180Genomes_11rx_hq\n",
      "+ marginAlign 076383_180Genomes_11rx_hq.fastq ../refs/EM_079517_mut30_2.fasta EM_079517_mut30_2_076383_180Genomes_11rx_hq.sam --jobTree jobTree_align_EM_079517_mut30_2_076383_180Genomes_11rx_hq --inputModel ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446200767.16 : Going to chain sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_076383_180Genomes_11rx_hq/jobs/gTD0/tmp_YNifQEPnom/temp.sam\n",
      "Got message from job at time: 1446200767.16 : Going to realign sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_076383_180Genomes_11rx_hq/jobs/gTD1/tmp_yKiV6kwEAf/temp.sam to create output sam file: EM_079517_mut30_2_076383_180Genomes_11rx_hq.sam     with match gamma 0.5 and gap gamma 0.0 and model ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446200843.42 : Realigned sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_076383_180Genomes_11rx_hq/jobs/gTD1/tmp_yKiV6kwEAf/temp.sam to create output sam file: EM_079517_mut30_2_076383_180Genomes_11rx_hq.sam\n",
      "+ samtools view -bS -\n",
      "+ align_trim.py\n",
      "+ samtools sort - EM_079517_mut30_2_076383_180Genomes_11rx_hq_marginalign.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_mut30_2_076383_180Genomes_11rx_hq_marginalign.sorted.bam\n",
      "+ rm 076383_180Genomes_11rx_hq.fasta.fast5.fofn\n",
      "rm: cannot remove ‘076383_180Genomes_11rx_hq.fasta.fast5.fofn’: No such file or directory\n",
      "+ nanopolish eventalign --reads 076383_180Genomes_11rx_hq.fasta -b EM_079517_mut30_2_076383_180Genomes_11rx_hq_marginalign.sorted.bam -g ../refs/EM_079517_mut30_2.fasta --sam\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_mut30_2_076383_180Genomes_11rx_hq_np.sorted.bam\n",
      "+ nanopolish variants --models-fofn offset_models.fofn --progress -t 1 --reads 076383_180Genomes_11rx_hq.fasta -o np_EM_079517_mut30_2_076383_180Genomes_11rx_hq.vcf -b EM_079517_mut30_2_076383_180Genomes_11rx_hq_marginalign.sorted.bam -e EM_079517_mut30_2_076383_180Genomes_11rx_hq_np.sorted.bam -g ../refs/EM_079517_mut30_2.fasta -vv -w EM_079517:0-20000 --snp\n",
      "TODO: train model\n",
      "TODO: filter data\n",
      "EM_079517\t40\t.\tG\tT\t3.9\tPASS\tBaseCalledReadsWithVariant=115;BaseCalledFrequency=0.416667;TotalReads=2;SupportingReads=2;SupportFraction=1;Template=1.975,-nan;Comp.P1=-nan,-nan;Comp.P2=1.93,-nan;ReadCounts=1,0,0,0,1,0\n",
      "EM_079517\t213\t.\tG\tA\t3102.9\tPASS\tBaseCalledReadsWithVariant=198;BaseCalledFrequency=0.709677;TotalReads=698;SupportingReads=626;SupportFraction=0.896848;Template=7.273,3.91;Comp.P1=2.572,-0.3316;Comp.P2=4.442,2.586;ReadCounts=167,182,9,16,173,151\n",
      "EM_079517\t672\t.\tT\tA\t1860.0\tPASS\tBaseCalledReadsWithVariant=62;BaseCalledFrequency=0.861111;TotalReads=204;SupportingReads=193;SupportFraction=0.946078;Template=12.11,5.801;Comp.P1=9.368,3.515;Comp.P2=12.76,6.449;ReadCounts=49,53,2,9,51,40\n",
      "EM_079517\t800\t.\tC\tT\t1151.4\tPASS\tBaseCalledReadsWithVariant=56;BaseCalledFrequency=0.861538;TotalReads=184;SupportingReads=174;SupportFraction=0.945652;Template=8.199,2.543;Comp.P1=4.95,1.175;Comp.P2=11.21,3.607;ReadCounts=42,50,2,8,48,34\n",
      "EM_079517\t927\t.\tC\tT\t1746.4\tPASS\tBaseCalledReadsWithVariant=71;BaseCalledFrequency=0.922078;TotalReads=168;SupportingReads=156;SupportFraction=0.928571;Template=18.11,4.572;Comp.P1=9.011,3.97;Comp.P2=16.66,2.333;ReadCounts=38,46,2,5,44,33\n",
      "EM_079517\t1018\t.\tC\tA\t1416.4\tPASS\tBaseCalledReadsWithVariant=70;BaseCalledFrequency=0.897436;TotalReads=164;SupportingReads=156;SupportFraction=0.95122;Template=3.365,13.36;Comp.P1=2.21,9.159;Comp.P2=4.066,14.39;ReadCounts=37,45,2,4,43,33\n",
      "EM_079517\t1849\t.\tT\tC\t5489.2\tPASS\tBaseCalledReadsWithVariant=253;BaseCalledFrequency=0.702778;TotalReads=1072;SupportingReads=838;SupportFraction=0.781716;Template=8.992,-0.7616;Comp.P1=5.771,-0.9448;Comp.P2=10.54,1.845;ReadCounts=281,255,8,4,247,277\n",
      "EM_079517\t2003\t.\tG\tC\t2301.0\tPASS\tBaseCalledReadsWithVariant=137;BaseCalledFrequency=0.820359;TotalReads=358;SupportingReads=337;SupportFraction=0.941341;Template=10.28,3.757;Comp.P1=7.689,5.688;Comp.P2=6.548,3.584;ReadCounts=115,64,19,12,45,103\n",
      "EM_079517\t2116\t.\tC\tA\t5420.2\tPASS\tBaseCalledReadsWithVariant=120;BaseCalledFrequency=0.714286;TotalReads=400;SupportingReads=373;SupportFraction=0.9325;Template=20.83,7.81;Comp.P1=8.826,4.503;Comp.P2=11.26,12.71;ReadCounts=124,76,23,16,53,108\n",
      "EM_079517\t2124\t.\tG\tA\t5420.2\tPASS\tBaseCalledReadsWithVariant=149;BaseCalledFrequency=0.776042;TotalReads=400;SupportingReads=373;SupportFraction=0.9325;Template=20.83,7.81;Comp.P1=8.826,4.503;Comp.P2=11.26,12.71;ReadCounts=124,76,23,16,53,108\n",
      "EM_079517\t2287\t.\tT\tA\t1045.7\tPASS\tBaseCalledReadsWithVariant=61;BaseCalledFrequency=0.54955;TotalReads=278;SupportingReads=224;SupportFraction=0.805755;Template=4.057,3.55;Comp.P1=0.6993,2.93;Comp.P2=0.9808,7.24;ReadCounts=73,66,17,17,49,56\n",
      "EM_079517\t2418\t.\tC\tA\t1175.6\tPASS\tBaseCalledReadsWithVariant=83;BaseCalledFrequency=0.83;TotalReads=280;SupportingReads=227;SupportFraction=0.810714;Template=5.45,1.438;Comp.P1=1.787,2.85;Comp.P2=9.656,2.199;ReadCounts=74,66,17,17,49,57\n",
      "EM_079517\t2535\t.\tA\tC\t2536.4\tPASS\tBaseCalledReadsWithVariant=94;BaseCalledFrequency=0.79661;TotalReads=288;SupportingReads=275;SupportFraction=0.954861;Template=11.61,12.27;Comp.P1=5.907,9.92;Comp.P2=4.499,5.432;ReadCounts=78,66,17,17,49,61\n",
      "EM_079517\t2811\t.\tT\tC\t2820.6\tPASS\tBaseCalledReadsWithVariant=119;BaseCalledFrequency=0.881481;TotalReads=290;SupportingReads=270;SupportFraction=0.931034;Template=9.795,12.95;Comp.P1=6.119,13.1;Comp.P2=8.998,6.801;ReadCounts=78,67,18,17,49,61\n",
      "EM_079517\t2968\t.\tT\tC\t2486.3\tPASS\tBaseCalledReadsWithVariant=112;BaseCalledFrequency=0.823529;TotalReads=290;SupportingReads=264;SupportFraction=0.910345;Template=12.47,8.942;Comp.P1=5.699,7.508;Comp.P2=7.996,4.8;ReadCounts=78,67,18,17,49,61\n",
      "EM_079517\t4360\t.\tC\tA\t8682.0\tPASS\tBaseCalledReadsWithVariant=296;BaseCalledFrequency=0.741855;TotalReads=836;SupportingReads=804;SupportFraction=0.961722;Template=12.3,12.6;Comp.P1=6.255,8.909;Comp.P2=9.345,8.03;ReadCounts=203,215,54,2,161,201\n",
      "EM_079517\t4550\t.\tT\tA\t6809.6\tPASS\tBaseCalledReadsWithVariant=369;BaseCalledFrequency=0.856148;TotalReads=879;SupportingReads=809;SupportFraction=0.920364;Template=8.898,8.728;Comp.P1=3.589,10.83;Comp.P2=5.797,8.207;ReadCounts=214,225,60,2,165,213\n",
      "EM_079517\t5042\t.\tC\tT\t5459.6\tPASS\tBaseCalledReadsWithVariant=397;BaseCalledFrequency=0.894144;TotalReads=956;SupportingReads=837;SupportFraction=0.875523;Template=10.35,2.79;Comp.P1=4.878,2.491;Comp.P2=8.09,2.609;ReadCounts=231,247,69,2,178,229\n",
      "EM_079517\t6056\t.\tA\tC\t869.6\tPASS\tBaseCalledReadsWithVariant=58;BaseCalledFrequency=0.90625;TotalReads=140;SupportingReads=131;SupportFraction=0.935714;Template=7.671,2.818;Comp.P1=3.94,-nan;Comp.P2=6.261,8.702;ReadCounts=34,36,6,0,30,34\n",
      "EM_079517\t6283\t.\tC\tT\t655.7\tPASS\tBaseCalledReadsWithVariant=47;BaseCalledFrequency=0.839286;TotalReads=136;SupportingReads=127;SupportFraction=0.933824;Template=8.094,2.507;Comp.P1=4.963,-nan;Comp.P2=7.418,1.7;ReadCounts=34,34,6,0,28,34\n",
      "EM_079517\t6619\t.\tC\tT\t1159.0\tPASS\tBaseCalledReadsWithVariant=62;BaseCalledFrequency=0.898551;TotalReads=142;SupportingReads=130;SupportFraction=0.915493;Template=13.93,4.455;Comp.P1=2.799,-nan;Comp.P2=10.27,5.484;ReadCounts=37,34,8,0,26,37\n",
      "EM_079517\t6895\t.\tT\tG\t830.4\tPASS\tBaseCalledReadsWithVariant=50;BaseCalledFrequency=0.877193;TotalReads=144;SupportingReads=127;SupportFraction=0.881944;Template=9.979,3.491;Comp.P1=1.752,-nan;Comp.P2=7.189,3.536;ReadCounts=37,35,8,0,27,37\n",
      "EM_079517\t6909\t.\tA\tT\t1072.4\tPASS\tBaseCalledReadsWithVariant=66;BaseCalledFrequency=0.942857;TotalReads=144;SupportingReads=138;SupportFraction=0.958333;Template=7.114,8.574;Comp.P1=5.399,-nan;Comp.P2=7.415,7.181;ReadCounts=37,35,8,0,27,37\n",
      "EM_079517\t7956\t.\tT\tA\t969.9\tPASS\tBaseCalledReadsWithVariant=44;BaseCalledFrequency=0.758621;TotalReads=136;SupportingReads=126;SupportFraction=0.926471;Template=7.786,9.053;Comp.P1=4.36,0.08723;Comp.P2=6.234,6.412;ReadCounts=28,40,4,5,36,23\n",
      "EM_079517\t8625\t.\tG\tC\t2305.9\tPASS\tBaseCalledReadsWithVariant=150;BaseCalledFrequency=0.867052;TotalReads=370;SupportingReads=341;SupportFraction=0.921622;Template=4.225,10.35;Comp.P1=0.9475,3.702;Comp.P2=3.891,7.172;ReadCounts=96,89,6,11,83,85\n",
      "EM_079517\t8778\t.\tG\tC\t1488.4\tPASS\tBaseCalledReadsWithVariant=112;BaseCalledFrequency=0.854962;TotalReads=386;SupportingReads=373;SupportFraction=0.966321;Template=2.817,3.902;Comp.P1=1.73,2.688;Comp.P2=4.601,4.547;ReadCounts=101,92,6,11,86,90\n",
      "EM_079517\t8928\t.\tA\tC\t975.1\tPASS\tBaseCalledReadsWithVariant=114;BaseCalledFrequency=0.612903;TotalReads=390;SupportingReads=327;SupportFraction=0.838462;Template=2.388,5.195;Comp.P1=2.014,3.154;Comp.P2=1.458,0.7828;ReadCounts=101,94,6,11,88,90\n",
      "EM_079517\t9264\t.\tA\tG\t1544.3\tPASS\tBaseCalledReadsWithVariant=75;BaseCalledFrequency=0.78125;TotalReads=388;SupportingReads=333;SupportFraction=0.858247;Template=2.285,6.173;Comp.P1=-1.465,8.763;Comp.P2=1.595,5.696;ReadCounts=103,91,6,11,85,92\n",
      "EM_079517\t9780\t.\tT\tG\t19913.9\tPASS\tBaseCalledReadsWithVariant=820;BaseCalledFrequency=0.870488;TotalReads=2364;SupportingReads=2222;SupportFraction=0.939932;Template=15.8,3.617;Comp.P1=0.5214,2.179;Comp.P2=11.3,3.969;ReadCounts=677,505,89,25,416,652\n",
      "EM_079517\t10096\t.\tT\tG\t14928.3\tPASS\tBaseCalledReadsWithVariant=389;BaseCalledFrequency=0.590288;TotalReads=1878;SupportingReads=1741;SupportFraction=0.92705;Template=13.36,2.77;Comp.P1=7.646,5.688;Comp.P2=13.9,2.81;ReadCounts=550,389,78,12,311,538\n",
      "EM_079517\t10315\t.\tC\tT\t8104.4\tPASS\tBaseCalledReadsWithVariant=559;BaseCalledFrequency=0.69875;TotalReads=1884;SupportingReads=1778;SupportFraction=0.943737;Template=3.593,3.463;Comp.P1=3.477,2.761;Comp.P2=5.307,5.212;ReadCounts=550,392,80,12,312,538\n",
      "EM_079517\t10407\t.\tG\tT\t9095.2\tPASS\tBaseCalledReadsWithVariant=460;BaseCalledFrequency=0.719875;TotalReads=1892;SupportingReads=1736;SupportFraction=0.917548;Template=6.86,3.665;Comp.P1=3.534,3.569;Comp.P2=7.89,1.976;ReadCounts=554,392,80,12,312,542\n",
      "EM_079517\t10451\t.\tT\tA\t13479.6\tPASS\tBaseCalledReadsWithVariant=708;BaseCalledFrequency=0.851986;TotalReads=1892;SupportingReads=1647;SupportFraction=0.870507;Template=1.379,15.27;Comp.P1=0.8086,9.397;Comp.P2=3.216,10.24;ReadCounts=554,392,80,12,312,542\n",
      "EM_079517\t10755\t.\tG\tA\t8550.7\tPASS\tBaseCalledReadsWithVariant=758;BaseCalledFrequency=0.85073;TotalReads=1856;SupportingReads=1532;SupportFraction=0.825431;Template=6.47,9.455;Comp.P1=2.076,6.147;Comp.P2=1.163,1.508;ReadCounts=541,387,77,12,310,529\n",
      "EM_079517\t11106\t.\tT\tA\t3566.3\tPASS\tBaseCalledReadsWithVariant=206;BaseCalledFrequency=0.95814;TotalReads=454;SupportingReads=426;SupportFraction=0.938326;Template=9.647,5.997;Comp.P1=8.395,-0.9308;Comp.P2=11.24,2.005;ReadCounts=59,168,3,3,165,56\n",
      "EM_079517\t11215\t.\tG\tC\t5489.2\tPASS\tBaseCalledReadsWithVariant=215;BaseCalledFrequency=0.951327;TotalReads=460;SupportingReads=441;SupportFraction=0.958696;Template=16.17,12.8;Comp.P1=5.688,0.3469;Comp.P2=9.651,12.58;ReadCounts=60,170,4,3,166,57\n",
      "EM_079517\t12568\t.\tT\tA\t3415.5\tPASS\tBaseCalledReadsWithVariant=147;BaseCalledFrequency=0.738693;TotalReads=616;SupportingReads=573;SupportFraction=0.930195;Template=5.89,7.801;Comp.P1=3.279,4.978;Comp.P2=2.282,5.978;ReadCounts=163,145,30,11,115,152\n",
      "EM_079517\t13856\t.\tA\tG\t4857.5\tPASS\tBaseCalledReadsWithVariant=250;BaseCalledFrequency=0.905797;TotalReads=630;SupportingReads=599;SupportFraction=0.950794;Template=5.606,9.389;Comp.P1=4.397,7.376;Comp.P2=7.86,8.968;ReadCounts=168,147,31,12,116,156\n",
      "EM_079517\t15652\t.\tC\tG\t23600.7\tPASS\tBaseCalledReadsWithVariant=875;BaseCalledFrequency=0.892857;TotalReads=2123;SupportingReads=2035;SupportFraction=0.958549;Template=19.71,6.861;Comp.P1=4.446,5.505;Comp.P2=14.43,4.795;ReadCounts=527,534,72,40,462,488\n",
      "EM_079517\t15660\t.\tT\tC\t23600.7\tPASS\tBaseCalledReadsWithVariant=709;BaseCalledFrequency=0.706175;TotalReads=2123;SupportingReads=2035;SupportFraction=0.958549;Template=19.71,6.861;Comp.P1=4.446,5.505;Comp.P2=14.43,4.795;ReadCounts=527,534,72,40,462,488\n",
      "EM_079517\t15802\t.\tC\tA\t955.3\tPASS\tBaseCalledReadsWithVariant=80;BaseCalledFrequency=0.792079;TotalReads=204;SupportingReads=188;SupportFraction=0.921569;Template=5.647,3.512;Comp.P1=3.351,-1.38;Comp.P2=5.52,4.999;ReadCounts=49,53,9,4,44,45\n",
      "EM_079517\t15963\t.\tG\tA\t1735.6\tPASS\tBaseCalledReadsWithVariant=49;BaseCalledFrequency=0.890909;TotalReads=210;SupportingReads=203;SupportFraction=0.966667;Template=3.438,12.94;Comp.P1=1.865,5.874;Comp.P2=4.97,12.64;ReadCounts=49,56,9,4,47,45\n",
      "EM_079517\t16119\t.\tC\tA\t1226.8\tPASS\tBaseCalledReadsWithVariant=90;BaseCalledFrequency=0.873786;TotalReads=216;SupportingReads=206;SupportFraction=0.953704;Template=5.128,6.625;Comp.P1=2.459,1.136;Comp.P2=3.932,8.067;ReadCounts=51,57,9,5,48,46\n",
      "EM_079517\t16352\t.\tG\tA\t916.8\tPASS\tBaseCalledReadsWithVariant=60;BaseCalledFrequency=0.625;TotalReads=216;SupportingReads=195;SupportFraction=0.902778;Template=2.925,3.282;Comp.P1=0.3188,2.862;Comp.P2=5.242,6.778;ReadCounts=51,57,9,5,48,46\n",
      "EM_079517\t16514\t.\tG\tA\t2766.2\tPASS\tBaseCalledReadsWithVariant=114;BaseCalledFrequency=0.876923;TotalReads=271;SupportingReads=253;SupportFraction=0.933579;Template=6.568,15.22;Comp.P1=2.596,9.992;Comp.P2=4.255,15.95;ReadCounts=70,65,9,11,57,59\n",
      "EM_079517\t17142\t.\tT\tC\t4079.7\tPASS\tBaseCalledReadsWithVariant=195;BaseCalledFrequency=0.859031;TotalReads=498;SupportingReads=469;SupportFraction=0.941767;Template=13.11,8.266;Comp.P1=4.138,4.06;Comp.P2=6.113,6.449;ReadCounts=107,142,7,17,135,90\n",
      "EM_079517\t17864\t.\tT\tA\t2738.4\tPASS\tBaseCalledReadsWithVariant=175;BaseCalledFrequency=0.945946;TotalReads=486;SupportingReads=419;SupportFraction=0.86214;Template=8.744,3.423;Comp.P1=6.032,2.94;Comp.P2=8.634,1.383;ReadCounts=104,139,6,16,133,88\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=076534_180Genomes_11rx\n",
      "+ poretools_dir=076534_180Genomes_11rx\n",
      "+ sample_tag=076534_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "++ tempfile\n",
      "+ tmp=/tmp/fileN04pLY\n",
      "+ poretools fastq --type 2D ../data/076534_180Genomes_11rx/pass\n",
      "+ poretools fasta --type 2D ../data/076534_180Genomes_11rx/pass\n",
      "+ '[' hq == hq ']'\n",
      "+ poretools fastq --type 2D --high-quality ../data/076534_180Genomes_11rx/fail\n",
      "+ poretools fasta --type 2D --high-quality ../data/076534_180Genomes_11rx/fail\n",
      "+ '[' '!' na == na ']'\n",
      "+ mv /tmp/fileN04pLY 076534_180Genomes_11rx_hq.fastq\n",
      "+ mv /tmp/fileN04pLY.fasta 076534_180Genomes_11rx_hq.fasta\n",
      "+ '[' '!' -e ../refs/EM_079517.fasta.amb ']'\n",
      "+ rm -rf jobTree_align_EM_079517_076534_180Genomes_11rx_hq\n",
      "+ marginAlign 076534_180Genomes_11rx_hq.fastq ../refs/EM_079517.fasta EM_079517_076534_180Genomes_11rx_hq.sam --jobTree jobTree_align_EM_079517_076534_180Genomes_11rx_hq --inputModel ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446200830.32 : Going to chain sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_076534_180Genomes_11rx_hq/jobs/gTD0/tmp_nn4jIOOU3I/temp.sam\n",
      "Got message from job at time: 1446200830.32 : Going to realign sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_076534_180Genomes_11rx_hq/jobs/gTD1/tmp_sSHzDsGeWW/temp.sam to create output sam file: EM_079517_076534_180Genomes_11rx_hq.sam     with match gamma 0.5 and gap gamma 0.0 and model ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446200905.2 : Realigned sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_076534_180Genomes_11rx_hq/jobs/gTD1/tmp_sSHzDsGeWW/temp.sam to create output sam file: EM_079517_076534_180Genomes_11rx_hq.sam\n",
      "+ align_trim.py\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_076534_180Genomes_11rx_hq_marginalign.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_076534_180Genomes_11rx_hq_marginalign.sorted.bam\n",
      "+ rm 076534_180Genomes_11rx_hq.fasta.fast5.fofn\n",
      "rm: cannot remove ‘076534_180Genomes_11rx_hq.fasta.fast5.fofn’: No such file or directory\n",
      "+ nanopolish eventalign --reads 076534_180Genomes_11rx_hq.fasta -b EM_079517_076534_180Genomes_11rx_hq_marginalign.sorted.bam -g ../refs/EM_079517.fasta --sam\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_076534_180Genomes_11rx_hq_np.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_076534_180Genomes_11rx_hq_np.sorted.bam\n",
      "+ nanopolish variants --models-fofn offset_models.fofn --progress -t 1 --reads 076534_180Genomes_11rx_hq.fasta -o np_EM_079517_076534_180Genomes_11rx_hq.vcf -b EM_079517_076534_180Genomes_11rx_hq_marginalign.sorted.bam -e EM_079517_076534_180Genomes_11rx_hq_np.sorted.bam -g ../refs/EM_079517.fasta -vv -w EM_079517:0-20000 --snp\n",
      "TODO: train model\n",
      "TODO: filter data\n",
      "EM_079517\t213\t.\tG\tA\t4797.7\tPASS\tBaseCalledReadsWithVariant=306;BaseCalledFrequency=0.748166;TotalReads=994;SupportingReads=906;SupportFraction=0.911469;Template=6.923,4.646;Comp.P1=3.689,0.8228;Comp.P2=5.331,2.752;ReadCounts=251,246,6,39,240,212\n",
      "EM_079517\t800\t.\tC\tT\t3189.4\tPASS\tBaseCalledReadsWithVariant=189;BaseCalledFrequency=0.895735;TotalReads=552;SupportingReads=513;SupportFraction=0.929348;Template=6.925,2.114;Comp.P1=6.149,0.6432;Comp.P2=11.23,3.741;ReadCounts=140,136,2,35,134,105\n",
      "EM_079517\t1849\t.\tT\tC\t3453.2\tPASS\tBaseCalledReadsWithVariant=154;BaseCalledFrequency=0.684444;TotalReads=668;SupportingReads=491;SupportFraction=0.73503;Template=9.482,-1.54;Comp.P1=4.619,0.07317;Comp.P2=11.26,1.587;ReadCounts=175,159,4,2,155,173\n",
      "EM_079517\t2124\t.\tG\tA\t5673.5\tPASS\tBaseCalledReadsWithVariant=306;BaseCalledFrequency=0.847645;TotalReads=769;SupportingReads=686;SupportFraction=0.892068;Template=12.48,3.664;Comp.P1=5.93,4.107;Comp.P2=7.052,5.377;ReadCounts=229,156,26,41,129,188\n",
      "EM_079517\t6283\t.\tC\tT\t296.7\tPASS\tBaseCalledReadsWithVariant=19;BaseCalledFrequency=0.730769;TotalReads=64;SupportingReads=60;SupportFraction=0.9375;Template=8.383,1.564;Comp.P1=3.2,-0.5991;Comp.P2=5.823,2.091;ReadCounts=23,9,1,3,8,20\n",
      "EM_079517\t6909\t.\tA\tT\t2143.7\tPASS\tBaseCalledReadsWithVariant=113;BaseCalledFrequency=0.882812;TotalReads=274;SupportingReads=261;SupportFraction=0.952555;Template=8.046,9.52;Comp.P1=6.831,5.984;Comp.P2=7.193,7.039;ReadCounts=87,50,9,1,41,86\n",
      "EM_079517\t8928\t.\tA\tC\t1413.6\tPASS\tBaseCalledReadsWithVariant=132;BaseCalledFrequency=0.647059;TotalReads=426;SupportingReads=373;SupportFraction=0.875587;Template=4.825,5.455;Comp.P1=2.915,3.171;Comp.P2=2.177,0.9694;ReadCounts=127,86,8,7,78,120\n",
      "EM_079517\t11204\t.\tT\tG\t4667.5\tPASS\tBaseCalledReadsWithVariant=259;BaseCalledFrequency=0.846405;TotalReads=632;SupportingReads=595;SupportFraction=0.941456;Template=3.665,11.85;Comp.P1=2.234,2.032;Comp.P2=4.136,9.017;ReadCounts=100,216,10,5,206,95\n",
      "EM_079517\t12208\t.\tA\tG\t4013.5\tPASS\tBaseCalledReadsWithVariant=236;BaseCalledFrequency=0.874074;TotalReads=660;SupportingReads=571;SupportFraction=0.865152;Template=11.6,1.697;Comp.P1=-0.7591,3.039;Comp.P2=9.162,3.934;ReadCounts=110,220,8,5,212,105\n",
      "EM_079517\t13390\t.\tA\tG\t5807.1\tPASS\tBaseCalledReadsWithVariant=419;BaseCalledFrequency=0.912854;TotalReads=954;SupportingReads=913;SupportFraction=0.957023;Template=5.797,6.389;Comp.P1=3.043,2.667;Comp.P2=8.103,4.829;ReadCounts=242,235,22,27,213,215\n",
      "EM_079517\t13856\t.\tA\tG\t7374.2\tPASS\tBaseCalledReadsWithVariant=399;BaseCalledFrequency=0.95;TotalReads=946;SupportingReads=896;SupportFraction=0.947146;Template=4.83,9.137;Comp.P1=5.726,8.149;Comp.P2=8.146,9.46;ReadCounts=240,233,20,26,213,214\n",
      "EM_079517\t15313\t.\tG\tC\t22016.0\tPASS\tBaseCalledReadsWithVariant=841;BaseCalledFrequency=0.822092;TotalReads=2108;SupportingReads=2035;SupportFraction=0.96537;Template=11.8,7.902;Comp.P1=5.952,3.759;Comp.P2=10.34,12.23;ReadCounts=518,536,35,6,501,512\n",
      "EM_079517\t15660\t.\tT\tC\t11352.2\tPASS\tBaseCalledReadsWithVariant=682;BaseCalledFrequency=0.728632;TotalReads=2004;SupportingReads=1727;SupportFraction=0.861776;Template=10.07,1.77;Comp.P1=2.413,2.497;Comp.P2=9.459,1.795;ReadCounts=489,513,30,5,483,484\n",
      "EM_079517\t17142\t.\tT\tC\t14301.0\tPASS\tBaseCalledReadsWithVariant=711;BaseCalledFrequency=0.876695;TotalReads=1746;SupportingReads=1644;SupportFraction=0.941581;Template=13.17,7.494;Comp.P1=7.976,3.193;Comp.P2=6.247,7.382;ReadCounts=358,515,10,36,505,322\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=076534_180Genomes_11rx\n",
      "+ poretools_dir=076534_180Genomes_11rx\n",
      "+ sample_tag=076534_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "++ tempfile\n",
      "+ tmp=/tmp/fileQW0sSZ\n",
      "+ poretools fastq --type 2D ../data/076534_180Genomes_11rx/pass\n",
      "+ poretools fasta --type 2D ../data/076534_180Genomes_11rx/pass\n",
      "+ '[' hq == hq ']'\n",
      "+ poretools fastq --type 2D --high-quality ../data/076534_180Genomes_11rx/fail\n",
      "+ poretools fasta --type 2D --high-quality ../data/076534_180Genomes_11rx/fail\n",
      "+ '[' '!' na == na ']'\n",
      "+ mv /tmp/fileQW0sSZ 076534_180Genomes_11rx_hq.fastq\n",
      "+ mv /tmp/fileQW0sSZ.fasta 076534_180Genomes_11rx_hq.fasta\n",
      "+ '[' '!' -e ../refs/EM_079517_mut30_2.fasta.amb ']'\n",
      "+ rm -rf jobTree_align_EM_079517_mut30_2_076534_180Genomes_11rx_hq\n",
      "+ marginAlign 076534_180Genomes_11rx_hq.fastq ../refs/EM_079517_mut30_2.fasta EM_079517_mut30_2_076534_180Genomes_11rx_hq.sam --jobTree jobTree_align_EM_079517_mut30_2_076534_180Genomes_11rx_hq --inputModel ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446200830.55 : Going to chain sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_076534_180Genomes_11rx_hq/jobs/gTD0/tmp_w53ht9Vf3l/temp.sam\n",
      "Got message from job at time: 1446200830.55 : Going to realign sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_076534_180Genomes_11rx_hq/jobs/gTD1/tmp_80jgzdJ29y/temp.sam to create output sam file: EM_079517_mut30_2_076534_180Genomes_11rx_hq.sam     with match gamma 0.5 and gap gamma 0.0 and model ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446200905.29 : Realigned sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_076534_180Genomes_11rx_hq/jobs/gTD1/tmp_80jgzdJ29y/temp.sam to create output sam file: EM_079517_mut30_2_076534_180Genomes_11rx_hq.sam\n",
      "+ align_trim.py\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_076534_180Genomes_11rx_hq_marginalign.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_mut30_2_076534_180Genomes_11rx_hq_marginalign.sorted.bam\n",
      "+ rm 076534_180Genomes_11rx_hq.fasta.fast5.fofn\n",
      "rm: cannot remove ‘076534_180Genomes_11rx_hq.fasta.fast5.fofn’: No such file or directory\n",
      "+ nanopolish eventalign --reads 076534_180Genomes_11rx_hq.fasta -b EM_079517_mut30_2_076534_180Genomes_11rx_hq_marginalign.sorted.bam -g ../refs/EM_079517_mut30_2.fasta --sam\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_076534_180Genomes_11rx_hq_np.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_mut30_2_076534_180Genomes_11rx_hq_np.sorted.bam\n",
      "+ nanopolish variants --models-fofn offset_models.fofn --progress -t 1 --reads 076534_180Genomes_11rx_hq.fasta -o np_EM_079517_mut30_2_076534_180Genomes_11rx_hq.vcf -b EM_079517_mut30_2_076534_180Genomes_11rx_hq_marginalign.sorted.bam -e EM_079517_mut30_2_076534_180Genomes_11rx_hq_np.sorted.bam -g ../refs/EM_079517_mut30_2.fasta -vv -w EM_079517:0-20000 --snp\n",
      "TODO: train model\n",
      "TODO: filter data\n",
      "EM_079517\t213\t.\tG\tA\t4797.7\tPASS\tBaseCalledReadsWithVariant=306;BaseCalledFrequency=0.748166;TotalReads=994;SupportingReads=906;SupportFraction=0.911469;Template=6.923,4.646;Comp.P1=3.689,0.8228;Comp.P2=5.331,2.752;ReadCounts=251,246,6,39,240,212\n",
      "EM_079517\t672\t.\tT\tA\t5095.3\tPASS\tBaseCalledReadsWithVariant=165;BaseCalledFrequency=0.932203;TotalReads=574;SupportingReads=557;SupportFraction=0.970383;Template=12.12,4.865;Comp.P1=6.472,4.298;Comp.P2=13.11,5.97;ReadCounts=145,142,3,36,139,109\n",
      "EM_079517\t800\t.\tC\tT\t3192.4\tPASS\tBaseCalledReadsWithVariant=189;BaseCalledFrequency=0.895735;TotalReads=552;SupportingReads=513;SupportFraction=0.929348;Template=6.925,2.117;Comp.P1=6.149,0.6432;Comp.P2=11.25,3.741;ReadCounts=140,136,2,35,134,105\n",
      "EM_079517\t927\t.\tC\tT\t5598.5\tPASS\tBaseCalledReadsWithVariant=217;BaseCalledFrequency=0.93133;TotalReads=520;SupportingReads=485;SupportFraction=0.932692;Template=19.1,2.942;Comp.P1=-3.115,4.638;Comp.P2=18.35,2.459;ReadCounts=134,126,2,32,124,102\n",
      "EM_079517\t1018\t.\tC\tA\t2833.8\tPASS\tBaseCalledReadsWithVariant=159;BaseCalledFrequency=0.828125;TotalReads=362;SupportingReads=346;SupportFraction=0.955801;Template=3.841,13.57;Comp.P1=2.71,7.305;Comp.P2=5.159,9.274;ReadCounts=92,89,1,19,88,73\n",
      "EM_079517\t1849\t.\tT\tC\t3453.2\tPASS\tBaseCalledReadsWithVariant=154;BaseCalledFrequency=0.684444;TotalReads=668;SupportingReads=491;SupportFraction=0.73503;Template=9.482,-1.54;Comp.P1=4.619,0.07317;Comp.P2=11.26,1.587;ReadCounts=175,159,4,2,155,173\n",
      "EM_079517\t2003\t.\tG\tC\t4079.5\tPASS\tBaseCalledReadsWithVariant=244;BaseCalledFrequency=0.802632;TotalReads=672;SupportingReads=615;SupportFraction=0.915179;Template=10.17,2.372;Comp.P1=8.489,3.85;Comp.P2=6.384,3.745;ReadCounts=211,125,22,34,103,177\n",
      "EM_079517\t2116\t.\tC\tA\t13457.0\tPASS\tBaseCalledReadsWithVariant=246;BaseCalledFrequency=0.75;TotalReads=766;SupportingReads=735;SupportFraction=0.95953;Template=27.32,9.293;Comp.P1=12.16,8.808;Comp.P2=17.64,15;ReadCounts=230,153,26,41,127,189\n",
      "EM_079517\t2124\t.\tG\tA\t13457.0\tPASS\tBaseCalledReadsWithVariant=311;BaseCalledFrequency=0.847411;TotalReads=766;SupportingReads=735;SupportFraction=0.95953;Template=27.32,9.293;Comp.P1=12.16,8.808;Comp.P2=17.64,15;ReadCounts=230,153,26,41,127,189\n",
      "EM_079517\t2287\t.\tT\tA\t2621.6\tPASS\tBaseCalledReadsWithVariant=171;BaseCalledFrequency=0.5;TotalReads=809;SupportingReads=623;SupportFraction=0.770087;Template=4.433,-0.8465;Comp.P1=1.417,-0.1059;Comp.P2=1.717,7.255;ReadCounts=239,165,27,42,139,197\n",
      "EM_079517\t2418\t.\tC\tA\t3363.1\tPASS\tBaseCalledReadsWithVariant=228;BaseCalledFrequency=0.808511;TotalReads=824;SupportingReads=658;SupportFraction=0.798544;Template=6.479,0.4442;Comp.P1=2.172,2.49;Comp.P2=7.758,2.261;ReadCounts=243,169,28,42,141,201\n",
      "EM_079517\t2535\t.\tA\tC\t7864.5\tPASS\tBaseCalledReadsWithVariant=303;BaseCalledFrequency=0.839335;TotalReads=830;SupportingReads=802;SupportFraction=0.966265;Template=13.54,12.82;Comp.P1=4.854,9.192;Comp.P2=4.879,5.67;ReadCounts=246,169,28,42,141,204\n",
      "EM_079517\t2811\t.\tT\tC\t8210.5\tPASS\tBaseCalledReadsWithVariant=356;BaseCalledFrequency=0.896725;TotalReads=832;SupportingReads=794;SupportFraction=0.954327;Template=9.641,12.13;Comp.P1=3.887,13.29;Comp.P2=10.05,8.248;ReadCounts=246,170,28,42,142,204\n",
      "EM_079517\t6283\t.\tC\tT\t296.7\tPASS\tBaseCalledReadsWithVariant=19;BaseCalledFrequency=0.730769;TotalReads=64;SupportingReads=60;SupportFraction=0.9375;Template=8.383,1.564;Comp.P1=3.2,-0.5991;Comp.P2=5.823,2.091;ReadCounts=23,9,1,3,8,20\n",
      "EM_079517\t6619\t.\tC\tT\t2434.8\tPASS\tBaseCalledReadsWithVariant=113;BaseCalledFrequency=0.896825;TotalReads=266;SupportingReads=245;SupportFraction=0.921053;Template=14.13,4.205;Comp.P1=5.227,-0.1381;Comp.P2=12.29,5.992;ReadCounts=87,46,8,3,38,84\n",
      "EM_079517\t6895\t.\tT\tG\t1662.3\tPASS\tBaseCalledReadsWithVariant=69;BaseCalledFrequency=0.734043;TotalReads=274;SupportingReads=246;SupportFraction=0.89781;Template=9.77,2.778;Comp.P1=2.357,3.9;Comp.P2=8.86,3.314;ReadCounts=87,50,9,1,41,86\n",
      "EM_079517\t6909\t.\tA\tT\t2147.1\tPASS\tBaseCalledReadsWithVariant=115;BaseCalledFrequency=0.884615;TotalReads=274;SupportingReads=261;SupportFraction=0.952555;Template=8.046,9.52;Comp.P1=6.831,5.984;Comp.P2=7.276,7.039;ReadCounts=87,50,9,1,41,86\n",
      "EM_079517\t7956\t.\tT\tA\t2824.0\tPASS\tBaseCalledReadsWithVariant=125;BaseCalledFrequency=0.874126;TotalReads=366;SupportingReads=353;SupportFraction=0.964481;Template=10.05,8.791;Comp.P1=7.043,5.153;Comp.P2=6.018,5.741;ReadCounts=107,76,11,5,65,102\n",
      "EM_079517\t8625\t.\tG\tC\t2835.9\tPASS\tBaseCalledReadsWithVariant=177;BaseCalledFrequency=0.867647;TotalReads=424;SupportingReads=390;SupportFraction=0.919811;Template=4.953,10.38;Comp.P1=-0.1625,1.724;Comp.P2=4.056,8.334;ReadCounts=126,86,8,7,78,119\n",
      "EM_079517\t8778\t.\tG\tC\t1815.3\tPASS\tBaseCalledReadsWithVariant=117;BaseCalledFrequency=0.879699;TotalReads=424;SupportingReads=412;SupportFraction=0.971698;Template=2.96,4.166;Comp.P1=2.692,2.387;Comp.P2=5.331,5.294;ReadCounts=126,86,8,7,78,119\n",
      "EM_079517\t8928\t.\tA\tC\t1413.6\tPASS\tBaseCalledReadsWithVariant=132;BaseCalledFrequency=0.647059;TotalReads=426;SupportingReads=373;SupportFraction=0.875587;Template=4.825,5.455;Comp.P1=2.915,3.171;Comp.P2=2.177,0.9694;ReadCounts=127,86,8,7,78,120\n",
      "EM_079517\t9780\t.\tT\tG\t29199.8\tPASS\tBaseCalledReadsWithVariant=1113;BaseCalledFrequency=0.894695;TotalReads=3170;SupportingReads=3002;SupportFraction=0.947003;Template=16.3,2.977;Comp.P1=-0.1015,1.661;Comp.P2=13.54,4.507;ReadCounts=948,637,74,12,563,936\n",
      "EM_079517\t10096\t.\tT\tG\t19511.4\tPASS\tBaseCalledReadsWithVariant=543;BaseCalledFrequency=0.610799;TotalReads=2380;SupportingReads=2232;SupportFraction=0.937815;Template=13.18,2.707;Comp.P1=7.614,6.099;Comp.P2=15.24,2.864;ReadCounts=721,469,63,4,406,717\n",
      "EM_079517\t10407\t.\tG\tT\t12100.6\tPASS\tBaseCalledReadsWithVariant=634;BaseCalledFrequency=0.802532;TotalReads=2386;SupportingReads=2213;SupportFraction=0.927494;Template=7.217,3.217;Comp.P1=3.228,3.515;Comp.P2=8.571,2.317;ReadCounts=721,472,64,4,408,717\n",
      "EM_079517\t10451\t.\tT\tA\t18629.5\tPASS\tBaseCalledReadsWithVariant=940;BaseCalledFrequency=0.874419;TotalReads=2386;SupportingReads=2149;SupportFraction=0.900671;Template=0.9782,15.89;Comp.P1=0.4252,19.04;Comp.P2=4.06,12.08;ReadCounts=720,473,64,4,409,716\n",
      "EM_079517\t10755\t.\tG\tA\t10303.6\tPASS\tBaseCalledReadsWithVariant=993;BaseCalledFrequency=0.876434;TotalReads=2362;SupportingReads=2012;SupportFraction=0.85182;Template=5.788,8.143;Comp.P1=1.516,7.168;Comp.P2=1.378,2.374;ReadCounts=713,468,62,4,406,709\n",
      "EM_079517\t11106\t.\tT\tA\t4878.2\tPASS\tBaseCalledReadsWithVariant=280;BaseCalledFrequency=0.942761;TotalReads=627;SupportingReads=580;SupportFraction=0.92504;Template=9.289,5.75;Comp.P1=2.308,1.781;Comp.P2=12.32,1.695;ReadCounts=98,216,10,5,206,92\n",
      "EM_079517\t11204\t.\tT\tG\t4664.3\tPASS\tBaseCalledReadsWithVariant=259;BaseCalledFrequency=0.846405;TotalReads=632;SupportingReads=595;SupportFraction=0.941456;Template=3.667,11.84;Comp.P1=2.201,2.062;Comp.P2=4.129,9.032;ReadCounts=100,216,10,5,206,95\n",
      "EM_079517\t11215\t.\tG\tC\t8034.8\tPASS\tBaseCalledReadsWithVariant=286;BaseCalledFrequency=0.940789;TotalReads=634;SupportingReads=616;SupportFraction=0.971609;Template=16.59,13.37;Comp.P1=5.427,3.293;Comp.P2=10.06,13.91;ReadCounts=100,217,10,5,207,95\n",
      "EM_079517\t12208\t.\tA\tG\t4013.5\tPASS\tBaseCalledReadsWithVariant=236;BaseCalledFrequency=0.874074;TotalReads=660;SupportingReads=571;SupportFraction=0.865152;Template=11.6,1.697;Comp.P1=-0.7591,3.039;Comp.P2=9.162,3.934;ReadCounts=110,220,8,5,212,105\n",
      "EM_079517\t12568\t.\tT\tA\t5625.4\tPASS\tBaseCalledReadsWithVariant=206;BaseCalledFrequency=0.746377;TotalReads=954;SupportingReads=898;SupportFraction=0.9413;Template=6.015,9.034;Comp.P1=3.349,2.001;Comp.P2=2.778,6.174;ReadCounts=245,232,20,27,212,218\n",
      "EM_079517\t13390\t.\tA\tG\t5807.1\tPASS\tBaseCalledReadsWithVariant=419;BaseCalledFrequency=0.912854;TotalReads=954;SupportingReads=913;SupportFraction=0.957023;Template=5.797,6.389;Comp.P1=3.043,2.667;Comp.P2=8.103,4.829;ReadCounts=242,235,22,27,213,215\n",
      "EM_079517\t13856\t.\tA\tG\t7374.2\tPASS\tBaseCalledReadsWithVariant=399;BaseCalledFrequency=0.95;TotalReads=946;SupportingReads=896;SupportFraction=0.947146;Template=4.83,9.137;Comp.P1=5.726,8.149;Comp.P2=8.146,9.46;ReadCounts=240,233,20,26,213,214\n",
      "EM_079517\t15313\t.\tG\tC\t22016.0\tPASS\tBaseCalledReadsWithVariant=841;BaseCalledFrequency=0.822092;TotalReads=2108;SupportingReads=2035;SupportFraction=0.96537;Template=11.8,7.902;Comp.P1=5.952,3.759;Comp.P2=10.34,12.23;ReadCounts=518,536,35,6,501,512\n",
      "EM_079517\t15652\t.\tC\tG\t23941.4\tPASS\tBaseCalledReadsWithVariant=795;BaseCalledFrequency=0.89527;TotalReads=2000;SupportingReads=1894;SupportFraction=0.947;Template=21.95,6.689;Comp.P1=3.727,5.689;Comp.P2=15.74,4.29;ReadCounts=487,513,29,5,484,482\n",
      "EM_079517\t15660\t.\tT\tC\t23941.4\tPASS\tBaseCalledReadsWithVariant=683;BaseCalledFrequency=0.726596;TotalReads=2000;SupportingReads=1894;SupportFraction=0.947;Template=21.95,6.689;Comp.P1=3.727,5.689;Comp.P2=15.74,4.29;ReadCounts=487,513,29,5,484,482\n",
      "EM_079517\t17142\t.\tT\tC\t14301.0\tPASS\tBaseCalledReadsWithVariant=711;BaseCalledFrequency=0.876695;TotalReads=1746;SupportingReads=1644;SupportFraction=0.941581;Template=13.17,7.494;Comp.P1=7.976,3.193;Comp.P2=6.247,7.382;ReadCounts=358,515,10,36,505,322\n",
      "EM_079517\t17864\t.\tT\tA\t10790.8\tPASS\tBaseCalledReadsWithVariant=657;BaseCalledFrequency=0.913769;TotalReads=1746;SupportingReads=1575;SupportFraction=0.902062;Template=9.219,3.366;Comp.P1=1.361,3.196;Comp.P2=9.65,2.341;ReadCounts=357,516,10,36,506,321\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=076769_180Genomes_19Rx\n",
      "+ poretools_dir=076769_180Genomes_19Rx\n",
      "+ sample_tag=076769_180Genomes_19Rx_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "++ tempfile\n",
      "+ tmp=/tmp/fileqNy1n3\n",
      "+ poretools fastq --type 2D ../data/076769_180Genomes_19Rx/pass\n",
      "+ poretools fasta --type 2D ../data/076769_180Genomes_19Rx/pass\n",
      "+ '[' hq == hq ']'\n",
      "+ poretools fastq --type 2D --high-quality ../data/076769_180Genomes_19Rx/fail\n",
      "+ poretools fasta --type 2D --high-quality ../data/076769_180Genomes_19Rx/fail\n",
      "+ '[' '!' na == na ']'\n",
      "+ mv /tmp/fileqNy1n3 076769_180Genomes_19Rx_hq.fastq\n",
      "+ mv /tmp/fileqNy1n3.fasta 076769_180Genomes_19Rx_hq.fasta\n",
      "+ '[' '!' -e ../refs/EM_079517.fasta.amb ']'\n",
      "+ rm -rf jobTree_align_EM_079517_076769_180Genomes_19Rx_hq\n",
      "+ marginAlign 076769_180Genomes_19Rx_hq.fastq ../refs/EM_079517.fasta EM_079517_076769_180Genomes_19Rx_hq.sam --jobTree jobTree_align_EM_079517_076769_180Genomes_19Rx_hq --inputModel ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446201154.23 : Going to chain sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_076769_180Genomes_19Rx_hq/jobs/gTD0/tmp_0SGQ7K1aAb/temp.sam\n",
      "Got message from job at time: 1446201154.23 : Going to realign sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_076769_180Genomes_19Rx_hq/jobs/gTD1/tmp_WLd3sMwrBH/temp.sam to create output sam file: EM_079517_076769_180Genomes_19Rx_hq.sam     with match gamma 0.5 and gap gamma 0.0 and model ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446201313.81 : Realigned sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_076769_180Genomes_19Rx_hq/jobs/gTD1/tmp_WLd3sMwrBH/temp.sam to create output sam file: EM_079517_076769_180Genomes_19Rx_hq.sam\n",
      "+ samtools view -bS -\n",
      "+ align_trim.py\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_076769_180Genomes_19Rx_hq_marginalign.sorted.bam\n",
      "+ rm 076769_180Genomes_19Rx_hq.fasta.fast5.fofn\n",
      "rm: cannot remove ‘076769_180Genomes_19Rx_hq.fasta.fast5.fofn’: No such file or directory\n",
      "+ nanopolish eventalign --reads 076769_180Genomes_19Rx_hq.fasta -b EM_079517_076769_180Genomes_19Rx_hq_marginalign.sorted.bam -g ../refs/EM_079517.fasta --sam\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_076769_180Genomes_19Rx_hq_np.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_076769_180Genomes_19Rx_hq_np.sorted.bam\n",
      "+ nanopolish variants --models-fofn offset_models.fofn --progress -t 1 --reads 076769_180Genomes_19Rx_hq.fasta -o np_EM_079517_076769_180Genomes_19Rx_hq.vcf -b EM_079517_076769_180Genomes_19Rx_hq_marginalign.sorted.bam -e EM_079517_076769_180Genomes_19Rx_hq_np.sorted.bam -g ../refs/EM_079517.fasta -vv -w EM_079517:0-20000 --snp\n",
      "TODO: train model\n",
      "TODO: filter data\n",
      "EM_079517\t213\t.\tG\tA\t6239.3\tPASS\tBaseCalledReadsWithVariant=440;BaseCalledFrequency=0.861057;TotalReads=1218;SupportingReads=1146;SupportFraction=0.940887;Template=7.373,5.691;Comp.P1=6.375,3.822;Comp.P2=4.86,2.325;ReadCounts=254,355,12,8,343,246\n",
      "EM_079517\t800\t.\tC\tT\t8298.8\tPASS\tBaseCalledReadsWithVariant=416;BaseCalledFrequency=0.906318;TotalReads=1282;SupportingReads=1223;SupportFraction=0.953978;Template=7.934,2.964;Comp.P1=4.453,0.5804;Comp.P2=11.38,3.623;ReadCounts=274,367,14,12,353,262\n",
      "EM_079517\t1849\t.\tT\tC\t15449.7\tPASS\tBaseCalledReadsWithVariant=693;BaseCalledFrequency=0.729474;TotalReads=2670;SupportingReads=2270;SupportFraction=0.850187;Template=9.673,-0.4307;Comp.P1=3.704,-0.2849;Comp.P2=10.63,3.186;ReadCounts=764,571,24,9,547,755\n",
      "EM_079517\t2124\t.\tG\tA\t18645.3\tPASS\tBaseCalledReadsWithVariant=969;BaseCalledFrequency=0.814286;TotalReads=2535;SupportingReads=2262;SupportFraction=0.892308;Template=11.45,1.854;Comp.P1=4.217,3.564;Comp.P2=7.767,5.777;ReadCounts=867,401,13,43,388,823\n",
      "EM_079517\t3361\t.\tT\tC\t4316.3\tPASS\tBaseCalledReadsWithVariant=340;BaseCalledFrequency=0.892388;TotalReads=804;SupportingReads=714;SupportFraction=0.88806;Template=10.66,2.075;Comp.P1=3.985,-nan;Comp.P2=8.967,1.69;ReadCounts=131,271,59,0,212,131\n",
      "EM_079517\t3375\t.\tT\tC\t4438.1\tPASS\tBaseCalledReadsWithVariant=285;BaseCalledFrequency=0.800562;TotalReads=804;SupportingReads=750;SupportFraction=0.932836;Template=10.7,1.977;Comp.P1=1.114,-nan;Comp.P2=9.346,3.461;ReadCounts=131,271,59,0,212,131\n",
      "EM_079517\t3447\t.\tT\tC\t2533.6\tPASS\tBaseCalledReadsWithVariant=262;BaseCalledFrequency=0.698667;TotalReads=806;SupportingReads=433;SupportFraction=0.537221;Template=8.611,-2.313;Comp.P1=2.363,-nan;Comp.P2=9.474,-0.9356;ReadCounts=131,272,59,0,213,131\n",
      "EM_079517\t3596\t.\tT\tC\t4544.2\tPASS\tBaseCalledReadsWithVariant=291;BaseCalledFrequency=0.826705;TotalReads=806;SupportingReads=770;SupportFraction=0.955335;Template=5.38,3.085;Comp.P1=3.096,-nan;Comp.P2=9.324,6.3;ReadCounts=131,272,58,0,214,131\n",
      "EM_079517\t5675\t.\tT\tC\t4034.3\tPASS\tBaseCalledReadsWithVariant=326;BaseCalledFrequency=0.810945;TotalReads=860;SupportingReads=814;SupportFraction=0.946512;Template=3.923,6.035;Comp.P1=-1.046,-nan;Comp.P2=3.569,5.404;ReadCounts=161,269,11,0,258,161\n",
      "EM_079517\t5939\t.\tT\tC\t7164.9\tPASS\tBaseCalledReadsWithVariant=309;BaseCalledFrequency=0.911504;TotalReads=792;SupportingReads=757;SupportFraction=0.955808;Template=10.18,10.46;Comp.P1=-0.3131,4.241;Comp.P2=8.215,7.746;ReadCounts=224,172,1,15,171,209\n",
      "EM_079517\t6283\t.\tC\tT\t4140.9\tPASS\tBaseCalledReadsWithVariant=293;BaseCalledFrequency=0.861765;TotalReads=796;SupportingReads=756;SupportFraction=0.949749;Template=7.752,3.991;Comp.P1=0.862,1.417;Comp.P2=7.694,1.731;ReadCounts=226,172,1,17,171,209\n",
      "EM_079517\t6909\t.\tA\tT\t3904.2\tPASS\tBaseCalledReadsWithVariant=212;BaseCalledFrequency=0.933921;TotalReads=482;SupportingReads=475;SupportFraction=0.985477;Template=9.22,7.486;Comp.P1=3.227,-nan;Comp.P2=7.999,7.869;ReadCounts=164,77,19,0,58,164\n",
      "EM_079517\t8928\t.\tA\tC\t3123.8\tPASS\tBaseCalledReadsWithVariant=363;BaseCalledFrequency=0.72167;TotalReads=1048;SupportingReads=860;SupportFraction=0.820611;Template=4.867,5.423;Comp.P1=0.2512,0.9619;Comp.P2=1.96,0.1065;ReadCounts=312,212,4,16,208,296\n",
      "EM_079517\t13856\t.\tA\tG\t7631.9\tPASS\tBaseCalledReadsWithVariant=386;BaseCalledFrequency=0.881279;TotalReads=1034;SupportingReads=954;SupportFraction=0.922631;Template=5.042,8.763;Comp.P1=5.063,9.475;Comp.P2=5.971,9.869;ReadCounts=210,307,20,12,287,198\n",
      "EM_079517\t15660\t.\tT\tC\t12461.9\tPASS\tBaseCalledReadsWithVariant=852;BaseCalledFrequency=0.785253;TotalReads=2254;SupportingReads=1831;SupportFraction=0.812334;Template=9.875,0.3626;Comp.P1=3.368,1.393;Comp.P2=9.689,3.14;ReadCounts=421,706,7,40,699,381\n",
      "EM_079517\t15675\t.\tT\tC\t9071.5\tPASS\tBaseCalledReadsWithVariant=590;BaseCalledFrequency=0.598377;TotalReads=2202;SupportingReads=2002;SupportFraction=0.909173;Template=5.426,4.303;Comp.P1=4.02,1.636;Comp.P2=3.154,4.384;ReadCounts=408,693,7,38,686,370\n",
      "EM_079517\t15963\t.\tG\tA\t10222.2\tPASS\tBaseCalledReadsWithVariant=329;BaseCalledFrequency=0.826633;TotalReads=1222;SupportingReads=1183;SupportFraction=0.968085;Template=3.24,12.42;Comp.P1=0.8659,11.48;Comp.P2=5.192,11.89;ReadCounts=189,422,15,14,407,175\n",
      "EM_079517\t16749\t.\tT\tC\t10781.6\tPASS\tBaseCalledReadsWithVariant=746;BaseCalledFrequency=0.748245;TotalReads=2154;SupportingReads=1951;SupportFraction=0.905757;Template=5.345,4.589;Comp.P1=3.839,0.7588;Comp.P2=2.955,7.17;ReadCounts=650,427,10,91,417,559\n",
      "EM_079517\t17142\t.\tT\tC\t18920.2\tPASS\tBaseCalledReadsWithVariant=903;BaseCalledFrequency=0.872464;TotalReads=2208;SupportingReads=2086;SupportFraction=0.944746;Template=12.6,7.199;Comp.P1=4.601,1.691;Comp.P2=6.186,8.056;ReadCounts=654,450,11,93,439,561\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=076769_180Genomes_19Rx\n",
      "+ poretools_dir=076769_180Genomes_19Rx\n",
      "+ sample_tag=076769_180Genomes_19Rx_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "++ tempfile\n",
      "+ tmp=/tmp/filejMZZv5\n",
      "+ poretools fastq --type 2D ../data/076769_180Genomes_19Rx/pass\n",
      "+ poretools fasta --type 2D ../data/076769_180Genomes_19Rx/pass\n",
      "+ '[' hq == hq ']'\n",
      "+ poretools fastq --type 2D --high-quality ../data/076769_180Genomes_19Rx/fail\n",
      "+ poretools fasta --type 2D --high-quality ../data/076769_180Genomes_19Rx/fail\n",
      "+ '[' '!' na == na ']'\n",
      "+ mv /tmp/filejMZZv5 076769_180Genomes_19Rx_hq.fastq\n",
      "+ mv /tmp/filejMZZv5.fasta 076769_180Genomes_19Rx_hq.fasta\n",
      "+ '[' '!' -e ../refs/EM_079517_mut30_2.fasta.amb ']'\n",
      "+ rm -rf jobTree_align_EM_079517_mut30_2_076769_180Genomes_19Rx_hq\n",
      "+ marginAlign 076769_180Genomes_19Rx_hq.fastq ../refs/EM_079517_mut30_2.fasta EM_079517_mut30_2_076769_180Genomes_19Rx_hq.sam --jobTree jobTree_align_EM_079517_mut30_2_076769_180Genomes_19Rx_hq --inputModel ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446201154.23 : Going to chain sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_076769_180Genomes_19Rx_hq/jobs/gTD0/tmp_9FJRp6YJuZ/temp.sam\n",
      "Got message from job at time: 1446201154.23 : Going to realign sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_076769_180Genomes_19Rx_hq/jobs/gTD1/tmp_agMFHkHeys/temp.sam to create output sam file: EM_079517_mut30_2_076769_180Genomes_19Rx_hq.sam     with match gamma 0.5 and gap gamma 0.0 and model ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446201313.63 : Realigned sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_076769_180Genomes_19Rx_hq/jobs/gTD1/tmp_agMFHkHeys/temp.sam to create output sam file: EM_079517_mut30_2_076769_180Genomes_19Rx_hq.sam\n",
      "+ align_trim.py\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_076769_180Genomes_19Rx_hq_marginalign.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_mut30_2_076769_180Genomes_19Rx_hq_marginalign.sorted.bam\n",
      "+ rm 076769_180Genomes_19Rx_hq.fasta.fast5.fofn\n",
      "rm: cannot remove ‘076769_180Genomes_19Rx_hq.fasta.fast5.fofn’: No such file or directory\n",
      "+ nanopolish eventalign --reads 076769_180Genomes_19Rx_hq.fasta -b EM_079517_mut30_2_076769_180Genomes_19Rx_hq_marginalign.sorted.bam -g ../refs/EM_079517_mut30_2.fasta --sam\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_076769_180Genomes_19Rx_hq_np.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_mut30_2_076769_180Genomes_19Rx_hq_np.sorted.bam\n",
      "+ nanopolish variants --models-fofn offset_models.fofn --progress -t 1 --reads 076769_180Genomes_19Rx_hq.fasta -o np_EM_079517_mut30_2_076769_180Genomes_19Rx_hq.vcf -b EM_079517_mut30_2_076769_180Genomes_19Rx_hq_marginalign.sorted.bam -e EM_079517_mut30_2_076769_180Genomes_19Rx_hq_np.sorted.bam -g ../refs/EM_079517_mut30_2.fasta -vv -w EM_079517:0-20000 --snp\n",
      "TODO: train model\n",
      "TODO: filter data\n",
      "EM_079517\t213\t.\tG\tA\t6239.3\tPASS\tBaseCalledReadsWithVariant=440;BaseCalledFrequency=0.861057;TotalReads=1218;SupportingReads=1146;SupportFraction=0.940887;Template=7.373,5.691;Comp.P1=6.375,3.822;Comp.P2=4.86,2.325;ReadCounts=254,355,12,8,343,246\n",
      "EM_079517\t672\t.\tT\tA\t12646.3\tPASS\tBaseCalledReadsWithVariant=419;BaseCalledFrequency=0.952273;TotalReads=1278;SupportingReads=1253;SupportFraction=0.980438;Template=12.07,7.746;Comp.P1=7.256,5.674;Comp.P2=12.97,6.807;ReadCounts=274,365,13,12,352,262\n",
      "EM_079517\t800\t.\tC\tT\t8300.6\tPASS\tBaseCalledReadsWithVariant=416;BaseCalledFrequency=0.906318;TotalReads=1282;SupportingReads=1224;SupportFraction=0.954758;Template=7.934,2.964;Comp.P1=4.453,0.5804;Comp.P2=11.38,3.63;ReadCounts=274,367,14,12,353,262\n",
      "EM_079517\t927\t.\tC\tT\t14358.5\tPASS\tBaseCalledReadsWithVariant=566;BaseCalledFrequency=0.948074;TotalReads=1282;SupportingReads=1229;SupportFraction=0.958658;Template=18.65,4.836;Comp.P1=9.138,2.946;Comp.P2=18.16,3.425;ReadCounts=273,368,14,12,354,261\n",
      "EM_079517\t1018\t.\tC\tA\t23985.9\tPASS\tBaseCalledReadsWithVariant=1149;BaseCalledFrequency=0.942576;TotalReads=2570;SupportingReads=2467;SupportFraction=0.959922;Template=3.107,14.8;Comp.P1=2.036,7.219;Comp.P2=3.726,15.96;ReadCounts=680,605,32,15,573,665\n",
      "EM_079517\t1849\t.\tT\tC\t15449.7\tPASS\tBaseCalledReadsWithVariant=693;BaseCalledFrequency=0.729474;TotalReads=2670;SupportingReads=2270;SupportFraction=0.850187;Template=9.673,-0.4307;Comp.P1=3.704,-0.2849;Comp.P2=10.63,3.186;ReadCounts=764,571,24,9,547,755\n",
      "EM_079517\t2003\t.\tG\tC\t14511.3\tPASS\tBaseCalledReadsWithVariant=879;BaseCalledFrequency=0.861765;TotalReads=2264;SupportingReads=2033;SupportFraction=0.897968;Template=9.862,0.948;Comp.P1=7.761,6.058;Comp.P2=8.472,4.142;ReadCounts=826,306,10,36,296,790\n",
      "EM_079517\t2116\t.\tC\tA\t37643.7\tPASS\tBaseCalledReadsWithVariant=806;BaseCalledFrequency=0.735401;TotalReads=2524;SupportingReads=2387;SupportFraction=0.945721;Template=20.81,4.145;Comp.P1=8.127,5.704;Comp.P2=15.12,14.43;ReadCounts=862,400,12,41,388,821\n",
      "EM_079517\t2124\t.\tG\tA\t37643.7\tPASS\tBaseCalledReadsWithVariant=981;BaseCalledFrequency=0.818865;TotalReads=2524;SupportingReads=2387;SupportFraction=0.945721;Template=20.81,4.145;Comp.P1=8.127,5.704;Comp.P2=15.12,14.43;ReadCounts=862,400,12,41,388,821\n",
      "EM_079517\t2282\t.\tG\tT\t7546.8\tPASS\tBaseCalledReadsWithVariant=546;BaseCalledFrequency=0.54491;TotalReads=2592;SupportingReads=1835;SupportFraction=0.707948;Template=3.138,-2.997;Comp.P1=-1.124,-0.1455;Comp.P2=0.01821,7.255;ReadCounts=878,418,13,43,405,835\n",
      "EM_079517\t2287\t.\tT\tA\t7546.8\tPASS\tBaseCalledReadsWithVariant=609;BaseCalledFrequency=0.545699;TotalReads=2592;SupportingReads=1835;SupportFraction=0.707948;Template=3.138,-2.997;Comp.P1=-1.124,-0.1455;Comp.P2=0.01821,7.255;ReadCounts=878,418,13,43,405,835\n",
      "EM_079517\t2418\t.\tC\tA\t10040.6\tPASS\tBaseCalledReadsWithVariant=772;BaseCalledFrequency=0.855876;TotalReads=2604;SupportingReads=2070;SupportFraction=0.794931;Template=5.077,-0.4494;Comp.P1=0.2207,0.08931;Comp.P2=8.945,2.51;ReadCounts=879,423,13,45,410,834\n",
      "EM_079517\t2535\t.\tA\tC\t23649.5\tPASS\tBaseCalledReadsWithVariant=1015;BaseCalledFrequency=0.871993;TotalReads=2614;SupportingReads=2561;SupportFraction=0.979725;Template=11.62,10.7;Comp.P1=3.472,6.079;Comp.P2=7.006,6.747;ReadCounts=882,425,13,47,412,835\n",
      "EM_079517\t2811\t.\tT\tC\t22318.9\tPASS\tBaseCalledReadsWithVariant=1010;BaseCalledFrequency=0.840966;TotalReads=2520;SupportingReads=2391;SupportFraction=0.94881;Template=8.048,8.717;Comp.P1=3.567,7.925;Comp.P2=10.22,9.203;ReadCounts=846,414,9,45,405,801\n",
      "EM_079517\t3361\t.\tT\tC\t4316.3\tPASS\tBaseCalledReadsWithVariant=340;BaseCalledFrequency=0.892388;TotalReads=804;SupportingReads=714;SupportFraction=0.88806;Template=10.66,2.075;Comp.P1=3.985,-nan;Comp.P2=8.967,1.69;ReadCounts=131,271,59,0,212,131\n",
      "EM_079517\t3375\t.\tT\tC\t4438.1\tPASS\tBaseCalledReadsWithVariant=285;BaseCalledFrequency=0.800562;TotalReads=804;SupportingReads=750;SupportFraction=0.932836;Template=10.7,1.977;Comp.P1=1.114,-nan;Comp.P2=9.346,3.461;ReadCounts=131,271,59,0,212,131\n",
      "EM_079517\t3447\t.\tT\tC\t2533.6\tPASS\tBaseCalledReadsWithVariant=262;BaseCalledFrequency=0.698667;TotalReads=806;SupportingReads=433;SupportFraction=0.537221;Template=8.611,-2.313;Comp.P1=2.363,-nan;Comp.P2=9.474,-0.9356;ReadCounts=131,272,59,0,213,131\n",
      "EM_079517\t3596\t.\tT\tC\t4544.2\tPASS\tBaseCalledReadsWithVariant=291;BaseCalledFrequency=0.826705;TotalReads=806;SupportingReads=770;SupportFraction=0.955335;Template=5.38,3.085;Comp.P1=3.096,-nan;Comp.P2=9.324,6.3;ReadCounts=131,272,58,0,214,131\n",
      "EM_079517\t4360\t.\tC\tA\t20228.9\tPASS\tBaseCalledReadsWithVariant=710;BaseCalledFrequency=0.815155;TotalReads=1788;SupportingReads=1701;SupportFraction=0.951342;Template=13.13,14.51;Comp.P1=1.655,6.649;Comp.P2=9.258,9.825;ReadCounts=427,467,87,4,380,423\n",
      "EM_079517\t4550\t.\tT\tA\t16643.6\tPASS\tBaseCalledReadsWithVariant=754;BaseCalledFrequency=0.871676;TotalReads=1786;SupportingReads=1660;SupportFraction=0.929451;Template=11.13,12.07;Comp.P1=-0.5213,3.656;Comp.P2=6.063,9.444;ReadCounts=426,467,87,3,380,423\n",
      "EM_079517\t5042\t.\tC\tT\t5114.5\tPASS\tBaseCalledReadsWithVariant=372;BaseCalledFrequency=0.934673;TotalReads=852;SupportingReads=781;SupportFraction=0.916667;Template=10.46,2.785;Comp.P1=1.532,-nan;Comp.P2=8.946,2.517;ReadCounts=159,267,11,0,256,159\n",
      "EM_079517\t5675\t.\tT\tC\t4034.3\tPASS\tBaseCalledReadsWithVariant=326;BaseCalledFrequency=0.810945;TotalReads=860;SupportingReads=814;SupportFraction=0.946512;Template=3.923,6.035;Comp.P1=-1.046,-nan;Comp.P2=3.569,5.404;ReadCounts=161,269,11,0,258,161\n",
      "EM_079517\t5939\t.\tT\tC\t7164.9\tPASS\tBaseCalledReadsWithVariant=309;BaseCalledFrequency=0.911504;TotalReads=792;SupportingReads=757;SupportFraction=0.955808;Template=10.18,10.46;Comp.P1=-0.3131,4.241;Comp.P2=8.215,7.746;ReadCounts=224,172,1,15,171,209\n",
      "EM_079517\t6283\t.\tC\tT\t4140.9\tPASS\tBaseCalledReadsWithVariant=293;BaseCalledFrequency=0.861765;TotalReads=796;SupportingReads=756;SupportFraction=0.949749;Template=7.752,3.991;Comp.P1=0.862,1.417;Comp.P2=7.694,1.731;ReadCounts=226,172,1,17,171,209\n",
      "EM_079517\t6619\t.\tC\tT\t8136.4\tPASS\tBaseCalledReadsWithVariant=397;BaseCalledFrequency=0.896163;TotalReads=942;SupportingReads=874;SupportFraction=0.927813;Template=12.81,7.058;Comp.P1=1.825,2.683;Comp.P2=11.49,3.684;ReadCounts=256,215,5,24,210,232\n",
      "EM_079517\t6895\t.\tT\tG\t2788.5\tPASS\tBaseCalledReadsWithVariant=135;BaseCalledFrequency=0.745856;TotalReads=476;SupportingReads=451;SupportFraction=0.947479;Template=9.597,3.143;Comp.P1=0.0876,-nan;Comp.P2=7.919,3.322;ReadCounts=164,74,19,0,55,164\n",
      "EM_079517\t6909\t.\tA\tT\t3882.0\tPASS\tBaseCalledReadsWithVariant=211;BaseCalledFrequency=0.933628;TotalReads=478;SupportingReads=472;SupportFraction=0.987448;Template=9.22,7.56;Comp.P1=3.227,-nan;Comp.P2=8.059,7.868;ReadCounts=164,75,19,0,56,164\n",
      "EM_079517\t7956\t.\tT\tA\t5774.4\tPASS\tBaseCalledReadsWithVariant=272;BaseCalledFrequency=0.855346;TotalReads=797;SupportingReads=761;SupportFraction=0.954831;Template=9.353,7.872;Comp.P1=7.234,8.927;Comp.P2=6.63,4.921;ReadCounts=204,195,26,6,169,197\n",
      "EM_079517\t8625\t.\tG\tC\t7442.1\tPASS\tBaseCalledReadsWithVariant=422;BaseCalledFrequency=0.909483;TotalReads=974;SupportingReads=923;SupportFraction=0.947639;Template=5.892,13.02;Comp.P1=0.02319,-0.8503;Comp.P2=3.108,9.177;ReadCounts=289,198,2,10,196,279\n",
      "EM_079517\t8778\t.\tG\tC\t4341.0\tPASS\tBaseCalledReadsWithVariant=282;BaseCalledFrequency=0.831858;TotalReads=1008;SupportingReads=977;SupportFraction=0.969246;Template=3.116,3.988;Comp.P1=2.17,0.1459;Comp.P2=5.506,5.149;ReadCounts=300,204,3,13,201,287\n",
      "EM_079517\t8928\t.\tA\tC\t3123.7\tPASS\tBaseCalledReadsWithVariant=363;BaseCalledFrequency=0.72167;TotalReads=1048;SupportingReads=860;SupportFraction=0.820611;Template=4.867,5.423;Comp.P1=0.2512,0.9565;Comp.P2=1.96,0.1065;ReadCounts=312,212,4,16,208,296\n",
      "EM_079517\t9780\t.\tT\tG\t9364.4\tPASS\tBaseCalledReadsWithVariant=383;BaseCalledFrequency=0.886574;TotalReads=1052;SupportingReads=997;SupportFraction=0.947719;Template=14.44,3.704;Comp.P1=0.3814,1.667;Comp.P2=13.56,4.142;ReadCounts=311,215,7,14,208,297\n",
      "EM_079517\t10096\t.\tT\tG\t29440.6\tPASS\tBaseCalledReadsWithVariant=820;BaseCalledFrequency=0.608309;TotalReads=3599;SupportingReads=3382;SupportFraction=0.939705;Template=12.99,2.845;Comp.P1=6.347,2.564;Comp.P2=14.56,3.487;ReadCounts=874,925,130,3,796,871\n",
      "EM_079517\t10407\t.\tG\tT\t19710.7\tPASS\tBaseCalledReadsWithVariant=1039;BaseCalledFrequency=0.778277;TotalReads=3856;SupportingReads=3612;SupportFraction=0.936722;Template=7.077,3.32;Comp.P1=2.708,4.875;Comp.P2=8.692,2.335;ReadCounts=916,1012,180,4,832,912\n",
      "EM_079517\t10451\t.\tT\tA\t31857.5\tPASS\tBaseCalledReadsWithVariant=1463;BaseCalledFrequency=0.858568;TotalReads=3862;SupportingReads=3539;SupportFraction=0.916365;Template=0.9892,15.09;Comp.P1=0.4239,9.755;Comp.P2=4.836,12.61;ReadCounts=919,1012,180,4,832,915\n",
      "EM_079517\t10755\t.\tG\tA\t18840.8\tPASS\tBaseCalledReadsWithVariant=1671;BaseCalledFrequency=0.901294;TotalReads=3860;SupportingReads=3424;SupportFraction=0.887047;Template=5.934,8.263;Comp.P1=2.518,7.264;Comp.P2=1.783,3.355;ReadCounts=919,1011,180,4,831,915\n",
      "EM_079517\t11106\t.\tT\tA\t1354.6\tPASS\tBaseCalledReadsWithVariant=82;BaseCalledFrequency=0.953488;TotalReads=178;SupportingReads=164;SupportFraction=0.921348;Template=10.03,5.667;Comp.P1=2.535,8.043;Comp.P2=13.14,3.26;ReadCounts=53,36,3,1,33,52\n",
      "EM_079517\t11215\t.\tG\tC\t2558.6\tPASS\tBaseCalledReadsWithVariant=82;BaseCalledFrequency=0.931818;TotalReads=180;SupportingReads=177;SupportFraction=0.983333;Template=17.58,12.93;Comp.P1=5.751,6.387;Comp.P2=10.78,14.58;ReadCounts=53,37,3,1,34,52\n",
      "EM_079517\t12568\t.\tT\tA\t20769.4\tPASS\tBaseCalledReadsWithVariant=675;BaseCalledFrequency=0.771429;TotalReads=3280;SupportingReads=3189;SupportFraction=0.972256;Template=6.026,9.44;Comp.P1=3.416,4.073;Comp.P2=3.896,6.063;ReadCounts=823,817,33,30,784,793\n",
      "EM_079517\t13856\t.\tA\tG\t7631.9\tPASS\tBaseCalledReadsWithVariant=386;BaseCalledFrequency=0.881279;TotalReads=1034;SupportingReads=954;SupportFraction=0.922631;Template=5.042,8.763;Comp.P1=5.063,9.475;Comp.P2=5.971,9.869;ReadCounts=210,307,20,12,287,198\n",
      "EM_079517\t15652\t.\tC\tG\t25886.8\tPASS\tBaseCalledReadsWithVariant=963;BaseCalledFrequency=0.91453;TotalReads=2242;SupportingReads=2094;SupportFraction=0.933988;Template=21.61,3.534;Comp.P1=6.123,4.99;Comp.P2=16.9,6.193;ReadCounts=421,700,7,40,693,381\n",
      "EM_079517\t15660\t.\tT\tC\t25886.8\tPASS\tBaseCalledReadsWithVariant=855;BaseCalledFrequency=0.790203;TotalReads=2242;SupportingReads=2094;SupportFraction=0.933988;Template=21.61,3.534;Comp.P1=6.123,4.99;Comp.P2=16.9,6.193;ReadCounts=421,700,7,40,693,381\n",
      "EM_079517\t15675\t.\tT\tC\t9065.2\tPASS\tBaseCalledReadsWithVariant=590;BaseCalledFrequency=0.600815;TotalReads=2192;SupportingReads=1994;SupportFraction=0.909672;Template=5.426,4.344;Comp.P1=4.02,1.609;Comp.P2=3.16,4.384;ReadCounts=408,688,7,38,681,370\n",
      "EM_079517\t15802\t.\tC\tA\t5832.3\tPASS\tBaseCalledReadsWithVariant=477;BaseCalledFrequency=0.859459;TotalReads=1170;SupportingReads=1078;SupportFraction=0.921368;Template=5.453,4.045;Comp.P1=3.002,1.439;Comp.P2=5.351,6.304;ReadCounts=185,400,14,14,386,171\n",
      "EM_079517\t15963\t.\tG\tA\t10222.4\tPASS\tBaseCalledReadsWithVariant=329;BaseCalledFrequency=0.826633;TotalReads=1222;SupportingReads=1183;SupportFraction=0.968085;Template=3.24,12.42;Comp.P1=0.8659,11.48;Comp.P2=5.192,11.89;ReadCounts=189,422,15,14,407,175\n",
      "EM_079517\t16119\t.\tC\tA\t7234.1\tPASS\tBaseCalledReadsWithVariant=530;BaseCalledFrequency=0.880399;TotalReads=1232;SupportingReads=1168;SupportFraction=0.948052;Template=5.721,6.687;Comp.P1=1.406,1.922;Comp.P2=4.964,6.87;ReadCounts=193,423,14,15,409,178\n",
      "EM_079517\t16352\t.\tG\tA\t5639.6\tPASS\tBaseCalledReadsWithVariant=328;BaseCalledFrequency=0.601835;TotalReads=1228;SupportingReads=1119;SupportFraction=0.911238;Template=2.977,3.867;Comp.P1=0.6453,5.683;Comp.P2=5.665,5.85;ReadCounts=192,422,15,14,407,178\n",
      "EM_079517\t16749\t.\tT\tC\t10781.6\tPASS\tBaseCalledReadsWithVariant=746;BaseCalledFrequency=0.748245;TotalReads=2154;SupportingReads=1951;SupportFraction=0.905757;Template=5.345,4.589;Comp.P1=3.839,0.7588;Comp.P2=2.955,7.17;ReadCounts=650,427,10,91,417,559\n",
      "EM_079517\t17142\t.\tT\tC\t18920.2\tPASS\tBaseCalledReadsWithVariant=903;BaseCalledFrequency=0.872464;TotalReads=2208;SupportingReads=2086;SupportFraction=0.944746;Template=12.6,7.199;Comp.P1=4.601,1.691;Comp.P2=6.186,8.056;ReadCounts=654,450,11,93,439,561\n",
      "EM_079517\t17864\t.\tT\tA\t18308.8\tPASS\tBaseCalledReadsWithVariant=1086;BaseCalledFrequency=0.918782;TotalReads=2868;SupportingReads=2612;SupportFraction=0.910739;Template=7.969,3.793;Comp.P1=1.186,2.118;Comp.P2=10.23,2.845;ReadCounts=532,902,8,23,894,509\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=078416_180Genomes_11rx\n",
      "+ poretools_dir=078416_180Genomes_11rx\n",
      "+ sample_tag=078416_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "++ tempfile\n",
      "+ tmp=/tmp/filevJSZOX\n",
      "+ poretools fastq --type 2D ../data/078416_180Genomes_11rx/pass\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/pass/NBPOR1138_078416_180Genomes_11rx_3607_1_ch475_file15_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ poretools fasta --type 2D ../data/078416_180Genomes_11rx/pass\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/pass/NBPOR1138_078416_180Genomes_11rx_3607_1_ch475_file15_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ '[' hq == hq ']'\n",
      "+ poretools fastq --type 2D --high-quality ../data/078416_180Genomes_11rx/fail\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch460_file78_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch488_file141_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch475_file105_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch475_file164_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch479_file18_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch466_file47_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch479_file86_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch467_file124_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch478_file180_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ poretools fasta --type 2D --high-quality ../data/078416_180Genomes_11rx/fail\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch460_file78_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch488_file141_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch475_file105_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch475_file164_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch479_file18_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch466_file47_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch479_file86_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch467_file124_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch478_file180_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ '[' '!' na == na ']'\n",
      "+ mv /tmp/filevJSZOX 078416_180Genomes_11rx_hq.fastq\n",
      "+ mv /tmp/filevJSZOX.fasta 078416_180Genomes_11rx_hq.fasta\n",
      "+ '[' '!' -e ../refs/EM_079517.fasta.amb ']'\n",
      "+ rm -rf jobTree_align_EM_079517_078416_180Genomes_11rx_hq\n",
      "+ marginAlign 078416_180Genomes_11rx_hq.fastq ../refs/EM_079517.fasta EM_079517_078416_180Genomes_11rx_hq.sam --jobTree jobTree_align_EM_079517_078416_180Genomes_11rx_hq --inputModel ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446201917.75 : Going to chain sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_078416_180Genomes_11rx_hq/jobs/gTD0/tmp_VVgf95gCbl/temp.sam\n",
      "Got message from job at time: 1446201917.75 : Going to realign sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_078416_180Genomes_11rx_hq/jobs/gTD1/tmp_IYsQ4c2hy0/temp.sam to create output sam file: EM_079517_078416_180Genomes_11rx_hq.sam     with match gamma 0.5 and gap gamma 0.0 and model ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446202462.88 : Realigned sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_078416_180Genomes_11rx_hq/jobs/gTD1/tmp_IYsQ4c2hy0/temp.sam to create output sam file: EM_079517_078416_180Genomes_11rx_hq.sam\n",
      "+ align_trim.py\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_078416_180Genomes_11rx_hq_marginalign.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_078416_180Genomes_11rx_hq_marginalign.sorted.bam\n",
      "+ rm 078416_180Genomes_11rx_hq.fasta.fast5.fofn\n",
      "rm: cannot remove ‘078416_180Genomes_11rx_hq.fasta.fast5.fofn’: No such file or directory\n",
      "+ nanopolish eventalign --reads 078416_180Genomes_11rx_hq.fasta -b EM_079517_078416_180Genomes_11rx_hq_marginalign.sorted.bam -g ../refs/EM_079517.fasta --sam\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_078416_180Genomes_11rx_hq_np.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_078416_180Genomes_11rx_hq_np.sorted.bam\n",
      "+ nanopolish variants --models-fofn offset_models.fofn --progress -t 1 --reads 078416_180Genomes_11rx_hq.fasta -o np_EM_079517_078416_180Genomes_11rx_hq.vcf -b EM_079517_078416_180Genomes_11rx_hq_marginalign.sorted.bam -e EM_079517_078416_180Genomes_11rx_hq_np.sorted.bam -g ../refs/EM_079517.fasta -vv -w EM_079517:0-20000 --snp\n",
      "TODO: train model\n",
      "TODO: filter data\n",
      "EM_079517\t213\t.\tG\tA\t6228.6\tPASS\tBaseCalledReadsWithVariant=415;BaseCalledFrequency=0.616642;TotalReads=1714;SupportingReads=1543;SupportFraction=0.900233;Template=5.057,3.478;Comp.P1=1.777,-0.4096;Comp.P2=4.191,2.213;ReadCounts=438,419,7,58,412,380\n",
      "EM_079517\t800\t.\tC\tT\t8379.6\tPASS\tBaseCalledReadsWithVariant=488;BaseCalledFrequency=0.834188;TotalReads=1688;SupportingReads=1571;SupportFraction=0.930687;Template=6.605,1.303;Comp.P1=5.752,1.578;Comp.P2=9.459,2.8;ReadCounts=439,405,9,61,396,378\n",
      "EM_079517\t1849\t.\tT\tC\t8288.9\tPASS\tBaseCalledReadsWithVariant=393;BaseCalledFrequency=0.614062;TotalReads=1908;SupportingReads=1429;SupportFraction=0.748952;Template=8.007,-1.008;Comp.P1=4.755,0.01886;Comp.P2=9.245,1.229;ReadCounts=515,439,11,28,428,487\n",
      "EM_079517\t2124\t.\tG\tA\t2286.3\tPASS\tBaseCalledReadsWithVariant=131;BaseCalledFrequency=0.731844;TotalReads=398;SupportingReads=329;SupportFraction=0.826633;Template=10.01,2.035;Comp.P1=5.264,2.707;Comp.P2=6.015,3.793;ReadCounts=126,73,13,28,60,98\n",
      "EM_079517\t2282\t.\tG\tT\t86.6\tPASS\tBaseCalledReadsWithVariant=79;BaseCalledFrequency=0.467456;TotalReads=418;SupportingReads=236;SupportFraction=0.564593;Template=0.5309,0.5396;Comp.P1=-1.259,0.6073;Comp.P2=-0.9205,0.3787;ReadCounts=128,81,16,28,65,100\n",
      "EM_079517\t6056\t.\tA\tC\t1367.0\tPASS\tBaseCalledReadsWithVariant=97;BaseCalledFrequency=0.941748;TotalReads=234;SupportingReads=222;SupportFraction=0.948718;Template=7.379,2.594;Comp.P1=1.271,0.8061;Comp.P2=4.5,7.627;ReadCounts=70,47,1,2,46,68\n",
      "EM_079517\t6283\t.\tC\tT\t1120.3\tPASS\tBaseCalledReadsWithVariant=80;BaseCalledFrequency=0.833333;TotalReads=234;SupportingReads=221;SupportFraction=0.944444;Template=8.189,2.138;Comp.P1=7.139,4.067;Comp.P2=7.282,1.454;ReadCounts=70,47,1,1,46,69\n",
      "EM_079517\t6909\t.\tA\tT\t3649.7\tPASS\tBaseCalledReadsWithVariant=193;BaseCalledFrequency=0.861607;TotalReads=492;SupportingReads=480;SupportFraction=0.97561;Template=7.422,8.785;Comp.P1=4.826,8.654;Comp.P2=8.144,6.408;ReadCounts=149,97,18,3,79,146\n",
      "EM_079517\t8928\t.\tA\tC\t5091.2\tPASS\tBaseCalledReadsWithVariant=453;BaseCalledFrequency=0.557882;TotalReads=1752;SupportingReads=1563;SupportFraction=0.892123;Template=3.722,4.513;Comp.P1=1.694,4.486;Comp.P2=1.859,1.356;ReadCounts=499,377,67,53,310,446\n",
      "EM_079517\t9264\t.\tA\tG\t7855.4\tPASS\tBaseCalledReadsWithVariant=287;BaseCalledFrequency=0.747396;TotalReads=1730;SupportingReads=1592;SupportFraction=0.920231;Template=4.797,5.419;Comp.P1=0.04438,7.149;Comp.P2=2.891,4.985;ReadCounts=495,370,61,52,309,443\n",
      "EM_079517\t9821\t.\tC\tT\t81195.6\tPASS\tBaseCalledReadsWithVariant=4469;BaseCalledFrequency=0.782525;TotalReads=12622;SupportingReads=11936;SupportFraction=0.94565;Template=4.819,8.51;Comp.P1=0.405,5.424;Comp.P2=4.635,8.528;ReadCounts=3668,2642,489,142,2153,3528\n",
      "EM_079517\t10315\t.\tC\tT\t42002.3\tPASS\tBaseCalledReadsWithVariant=2465;BaseCalledFrequency=0.54961;TotalReads=11220;SupportingReads=10590;SupportFraction=0.94385;Template=3.304,1.88;Comp.P1=3.394,2.637;Comp.P2=5.169,4.826;ReadCounts=3255,2355,484,93,1871,3162\n",
      "EM_079517\t13856\t.\tA\tG\t28880.2\tPASS\tBaseCalledReadsWithVariant=1485;BaseCalledFrequency=0.896198;TotalReads=4146;SupportingReads=3892;SupportFraction=0.938736;Template=4.402,8.072;Comp.P1=3.357,6.748;Comp.P2=6.406,9.59;ReadCounts=1098,975,136,65,839,1033\n",
      "EM_079517\t15660\t.\tT\tC\t62827.5\tPASS\tBaseCalledReadsWithVariant=4523;BaseCalledFrequency=0.639745;TotalReads=15030;SupportingReads=11667;SupportFraction=0.776248;Template=7.414,0.1832;Comp.P1=2.239,1.042;Comp.P2=8.502,1.121;ReadCounts=3645,3870,298,318,3572,3327\n",
      "EM_079517\t15963\t.\tG\tA\t2252.1\tPASS\tBaseCalledReadsWithVariant=61;BaseCalledFrequency=0.598039;TotalReads=396;SupportingReads=372;SupportFraction=0.939394;Template=2.057,7.455;Comp.P1=0.2825,5.837;Comp.P2=4.377,9.197;ReadCounts=75,123,5,8,118,67\n",
      "EM_079517\t16514\t.\tG\tA\t3106.0\tPASS\tBaseCalledReadsWithVariant=122;BaseCalledFrequency=0.635417;TotalReads=408;SupportingReads=373;SupportFraction=0.914216;Template=3.882,9.823;Comp.P1=2.013,4.131;Comp.P2=3.835,15.28;ReadCounts=76,128,5,8,123,68\n",
      "EM_079517\t17142\t.\tT\tC\t23948.3\tPASS\tBaseCalledReadsWithVariant=1340;BaseCalledFrequency=0.803357;TotalReads=3676;SupportingReads=3401;SupportFraction=0.92519;Template=9.817,4.857;Comp.P1=7.607,3.077;Comp.P2=6.038,6.329;ReadCounts=802,1036,42,109,994,693\n",
      "EM_079517\t17661\t.\tT\tC\t18922.9\tPASS\tBaseCalledReadsWithVariant=1345;BaseCalledFrequency=0.823133;TotalReads=3704;SupportingReads=3476;SupportFraction=0.938445;Template=4.714,4.387;Comp.P1=4.76,1.533;Comp.P2=4.165,8.6;ReadCounts=801,1051,44,107,1007,694\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=078416_180Genomes_11rx\n",
      "+ poretools_dir=078416_180Genomes_11rx\n",
      "+ sample_tag=078416_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "++ tempfile\n",
      "+ tmp=/tmp/fileK9yuKY\n",
      "+ poretools fastq --type 2D ../data/078416_180Genomes_11rx/pass\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/pass/NBPOR1138_078416_180Genomes_11rx_3607_1_ch475_file15_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ poretools fasta --type 2D ../data/078416_180Genomes_11rx/pass\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/pass/NBPOR1138_078416_180Genomes_11rx_3607_1_ch475_file15_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ '[' hq == hq ']'\n",
      "+ poretools fastq --type 2D --high-quality ../data/078416_180Genomes_11rx/fail\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch460_file78_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch488_file141_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch475_file105_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch475_file164_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch479_file18_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch466_file47_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch479_file86_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch467_file124_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch478_file180_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ poretools fasta --type 2D --high-quality ../data/078416_180Genomes_11rx/fail\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch460_file78_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch488_file141_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch475_file105_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch475_file164_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch479_file18_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch466_file47_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch479_file86_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch467_file124_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/078416_180Genomes_11rx/fail/NBPOR1138_078416_180Genomes_11rx_3607_1_ch478_file180_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ '[' '!' na == na ']'\n",
      "+ mv /tmp/fileK9yuKY 078416_180Genomes_11rx_hq.fastq\n",
      "+ mv /tmp/fileK9yuKY.fasta 078416_180Genomes_11rx_hq.fasta\n",
      "+ '[' '!' -e ../refs/EM_079517_mut30_2.fasta.amb ']'\n",
      "+ rm -rf jobTree_align_EM_079517_mut30_2_078416_180Genomes_11rx_hq\n",
      "+ marginAlign 078416_180Genomes_11rx_hq.fastq ../refs/EM_079517_mut30_2.fasta EM_079517_mut30_2_078416_180Genomes_11rx_hq.sam --jobTree jobTree_align_EM_079517_mut30_2_078416_180Genomes_11rx_hq --inputModel ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446201917.75 : Going to chain sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_078416_180Genomes_11rx_hq/jobs/gTD0/tmp_0XZemSdmWe/temp.sam\n",
      "Got message from job at time: 1446201917.75 : Going to realign sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_078416_180Genomes_11rx_hq/jobs/gTD1/tmp_I1MSJeNPzC/temp.sam to create output sam file: EM_079517_mut30_2_078416_180Genomes_11rx_hq.sam     with match gamma 0.5 and gap gamma 0.0 and model ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446202465.29 : Realigned sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_078416_180Genomes_11rx_hq/jobs/gTD1/tmp_I1MSJeNPzC/temp.sam to create output sam file: EM_079517_mut30_2_078416_180Genomes_11rx_hq.sam\n",
      "+ align_trim.py\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_078416_180Genomes_11rx_hq_marginalign.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_mut30_2_078416_180Genomes_11rx_hq_marginalign.sorted.bam\n",
      "+ rm 078416_180Genomes_11rx_hq.fasta.fast5.fofn\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_078416_180Genomes_11rx_hq_np.sorted\n",
      "+ nanopolish eventalign --reads 078416_180Genomes_11rx_hq.fasta -b EM_079517_mut30_2_078416_180Genomes_11rx_hq_marginalign.sorted.bam -g ../refs/EM_079517_mut30_2.fasta --sam\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_mut30_2_078416_180Genomes_11rx_hq_np.sorted.bam\n",
      "+ nanopolish variants --models-fofn offset_models.fofn --progress -t 1 --reads 078416_180Genomes_11rx_hq.fasta -o np_EM_079517_mut30_2_078416_180Genomes_11rx_hq.vcf -b EM_079517_mut30_2_078416_180Genomes_11rx_hq_marginalign.sorted.bam -e EM_079517_mut30_2_078416_180Genomes_11rx_hq_np.sorted.bam -g ../refs/EM_079517_mut30_2.fasta -vv -w EM_079517:0-20000 --snp\n",
      "TODO: train model\n",
      "TODO: filter data\n",
      "EM_079517\t213\t.\tG\tA\t6228.6\tPASS\tBaseCalledReadsWithVariant=415;BaseCalledFrequency=0.616642;TotalReads=1714;SupportingReads=1543;SupportFraction=0.900233;Template=5.057,3.478;Comp.P1=1.777,-0.4096;Comp.P2=4.191,2.213;ReadCounts=438,419,7,58,412,380\n",
      "EM_079517\t672\t.\tT\tA\t14388.5\tPASS\tBaseCalledReadsWithVariant=477;BaseCalledFrequency=0.854839;TotalReads=1822;SupportingReads=1732;SupportFraction=0.950604;Template=10.82,3.668;Comp.P1=9.599,4.597;Comp.P2=11.99,5.241;ReadCounts=474,437,11,69,426,405\n",
      "EM_079517\t800\t.\tC\tT\t8378.8\tPASS\tBaseCalledReadsWithVariant=488;BaseCalledFrequency=0.834188;TotalReads=1688;SupportingReads=1571;SupportFraction=0.930687;Template=6.603,1.303;Comp.P1=5.752,1.578;Comp.P2=9.459,2.8;ReadCounts=439,405,9,61,396,378\n",
      "EM_079517\t927\t.\tC\tT\t12360.6\tPASS\tBaseCalledReadsWithVariant=588;BaseCalledFrequency=0.923077;TotalReads=1442;SupportingReads=1286;SupportFraction=0.891817;Template=14.98,1.869;Comp.P1=10.34,2.911;Comp.P2=15.95,1.534;ReadCounts=383,338,8,42,330,341\n",
      "EM_079517\t1018\t.\tC\tA\t10894.6\tPASS\tBaseCalledReadsWithVariant=555;BaseCalledFrequency=0.878165;TotalReads=1374;SupportingReads=1311;SupportFraction=0.954148;Template=3.198,11.02;Comp.P1=2.201,6.692;Comp.P2=4.078,13.98;ReadCounts=361,326,8,29,318,332\n",
      "EM_079517\t1849\t.\tT\tC\t8288.9\tPASS\tBaseCalledReadsWithVariant=393;BaseCalledFrequency=0.614062;TotalReads=1908;SupportingReads=1429;SupportFraction=0.748952;Template=8.007,-1.008;Comp.P1=4.755,0.01886;Comp.P2=9.245,1.229;ReadCounts=515,439,11,28,428,487\n",
      "EM_079517\t2003\t.\tG\tC\t1651.9\tPASS\tBaseCalledReadsWithVariant=115;BaseCalledFrequency=0.815603;TotalReads=310;SupportingReads=276;SupportFraction=0.890323;Template=8.065,2.153;Comp.P1=9.667,4.251;Comp.P2=5.617,3.28;ReadCounts=111,44,7,23,37,88\n",
      "EM_079517\t2116\t.\tC\tA\t5293.3\tPASS\tBaseCalledReadsWithVariant=118;BaseCalledFrequency=0.698225;TotalReads=380;SupportingReads=350;SupportFraction=0.921053;Template=22.04,5.54;Comp.P1=11.3,5.207;Comp.P2=14.52,11.69;ReadCounts=124,66,10,28,56,96\n",
      "EM_079517\t2124\t.\tG\tA\t5293.3\tPASS\tBaseCalledReadsWithVariant=134;BaseCalledFrequency=0.728261;TotalReads=380;SupportingReads=350;SupportFraction=0.921053;Template=22.04,5.54;Comp.P1=11.3,5.207;Comp.P2=14.52,11.69;ReadCounts=124,66,10,28,56,96\n",
      "EM_079517\t2282\t.\tG\tT\t1144.1\tPASS\tBaseCalledReadsWithVariant=75;BaseCalledFrequency=0.460123;TotalReads=414;SupportingReads=296;SupportFraction=0.714976;Template=4.304,-0.7739;Comp.P1=0.181,0.7971;Comp.P2=0.6239,6.003;ReadCounts=127,80,16,28,64,99\n",
      "EM_079517\t2287\t.\tT\tA\t1144.1\tPASS\tBaseCalledReadsWithVariant=61;BaseCalledFrequency=0.346591;TotalReads=414;SupportingReads=296;SupportFraction=0.714976;Template=4.304,-0.7739;Comp.P1=0.181,0.7971;Comp.P2=0.6239,6.003;ReadCounts=127,80,16,28,64,99\n",
      "EM_079517\t2418\t.\tC\tA\t1450.8\tPASS\tBaseCalledReadsWithVariant=102;BaseCalledFrequency=0.69863;TotalReads=422;SupportingReads=306;SupportFraction=0.725118;Template=6.209,-1.081;Comp.P1=4.216,0.5112;Comp.P2=8.344,1.049;ReadCounts=130,81,16,28,65,102\n",
      "EM_079517\t2535\t.\tA\tC\t3326.0\tPASS\tBaseCalledReadsWithVariant=147;BaseCalledFrequency=0.777778;TotalReads=430;SupportingReads=415;SupportFraction=0.965116;Template=9.397,11.73;Comp.P1=4.393,7.969;Comp.P2=4.141,5.203;ReadCounts=132,83,17,28,66,104\n",
      "EM_079517\t2811\t.\tT\tC\t3642.2\tPASS\tBaseCalledReadsWithVariant=172;BaseCalledFrequency=0.847291;TotalReads=434;SupportingReads=405;SupportFraction=0.93318;Template=7.745,9.934;Comp.P1=4.71,12.03;Comp.P2=8.808,7.374;ReadCounts=134,83,17,28,66,106\n",
      "EM_079517\t4360\t.\tC\tA\t50908.7\tPASS\tBaseCalledReadsWithVariant=2193;BaseCalledFrequency=0.729784;TotalReads=6316;SupportingReads=5966;SupportFraction=0.944585;Template=9.809,9.075;Comp.P1=7.131,6.556;Comp.P2=7.323,6.083;ReadCounts=1596,1562,434,31,1128,1565\n",
      "EM_079517\t4550\t.\tT\tA\t53590.5\tPASS\tBaseCalledReadsWithVariant=2681;BaseCalledFrequency=0.842023;TotalReads=6759;SupportingReads=6363;SupportFraction=0.941411;Template=9.313,8.759;Comp.P1=4.473,6.868;Comp.P2=6.146,7.964;ReadCounts=1704,1675,469,33,1207,1671\n",
      "EM_079517\t5042\t.\tC\tT\t37157.2\tPASS\tBaseCalledReadsWithVariant=2724;BaseCalledFrequency=0.866688;TotalReads=6909;SupportingReads=6141;SupportFraction=0.888841;Template=9.321,2.184;Comp.P1=3.81,1.606;Comp.P2=8.61,2.766;ReadCounts=1741,1714,488,32,1225,1709\n",
      "EM_079517\t6056\t.\tA\tC\t1367.0\tPASS\tBaseCalledReadsWithVariant=97;BaseCalledFrequency=0.941748;TotalReads=234;SupportingReads=222;SupportFraction=0.948718;Template=7.379,2.594;Comp.P1=1.271,0.8061;Comp.P2=4.5,7.627;ReadCounts=70,47,1,2,46,68\n",
      "EM_079517\t6283\t.\tC\tT\t1120.3\tPASS\tBaseCalledReadsWithVariant=80;BaseCalledFrequency=0.833333;TotalReads=234;SupportingReads=221;SupportFraction=0.944444;Template=8.189,2.138;Comp.P1=7.139,4.067;Comp.P2=7.282,1.454;ReadCounts=70,47,1,1,46,69\n",
      "EM_079517\t6619\t.\tC\tT\t3517.0\tPASS\tBaseCalledReadsWithVariant=199;BaseCalledFrequency=0.876652;TotalReads=492;SupportingReads=462;SupportFraction=0.939024;Template=10.74,3.289;Comp.P1=5.141,1.77;Comp.P2=10.16,4.767;ReadCounts=150,96,18,3,78,147\n",
      "EM_079517\t6895\t.\tT\tG\t2877.4\tPASS\tBaseCalledReadsWithVariant=142;BaseCalledFrequency=0.835294;TotalReads=492;SupportingReads=453;SupportFraction=0.920732;Template=10.63,2.706;Comp.P1=4.907,2.199;Comp.P2=7.017,2.617;ReadCounts=149,97,18,3,79,146\n",
      "EM_079517\t6909\t.\tA\tT\t3642.9\tPASS\tBaseCalledReadsWithVariant=193;BaseCalledFrequency=0.869369;TotalReads=492;SupportingReads=480;SupportFraction=0.97561;Template=7.372,8.785;Comp.P1=4.835,8.654;Comp.P2=8.144,6.411;ReadCounts=149,97,18,3,79,146\n",
      "EM_079517\t7956\t.\tT\tA\t10865.2\tPASS\tBaseCalledReadsWithVariant=488;BaseCalledFrequency=0.788368;TotalReads=1698;SupportingReads=1618;SupportFraction=0.952886;Template=8.608,5.933;Comp.P1=6.704,4.635;Comp.P2=5.615,5.035;ReadCounts=486,363,59,53,304,433\n",
      "EM_079517\t8625\t.\tG\tC\t8418.9\tPASS\tBaseCalledReadsWithVariant=635;BaseCalledFrequency=0.789801;TotalReads=1750;SupportingReads=1591;SupportFraction=0.909143;Template=3.477,7.057;Comp.P1=1.7,5.583;Comp.P2=3.519,5.683;ReadCounts=500,375,67,53,308,447\n",
      "EM_079517\t8778\t.\tG\tC\t6013.2\tPASS\tBaseCalledReadsWithVariant=283;BaseCalledFrequency=0.624724;TotalReads=1750;SupportingReads=1667;SupportFraction=0.952571;Template=2.085,3.521;Comp.P1=2.14,3.223;Comp.P2=4.864,4.107;ReadCounts=499,376,67,53,309,446\n",
      "EM_079517\t8928\t.\tA\tC\t5095.3\tPASS\tBaseCalledReadsWithVariant=454;BaseCalledFrequency=0.558426;TotalReads=1754;SupportingReads=1565;SupportFraction=0.892246;Template=3.72,4.513;Comp.P1=1.702,4.478;Comp.P2=1.859,1.356;ReadCounts=500,377,67,53,310,447\n",
      "EM_079517\t9264\t.\tA\tG\t7866.8\tPASS\tBaseCalledReadsWithVariant=287;BaseCalledFrequency=0.747396;TotalReads=1732;SupportingReads=1594;SupportFraction=0.920323;Template=4.799,5.419;Comp.P1=0.04438,7.117;Comp.P2=2.891,4.985;ReadCounts=496,370,61,53,309,443\n",
      "EM_079517\t9780\t.\tT\tG\t90644.2\tPASS\tBaseCalledReadsWithVariant=4168;BaseCalledFrequency=0.823065;TotalReads=12549;SupportingReads=11442;SupportFraction=0.911786;Template=14.04,1.331;Comp.P1=2.225,2.271;Comp.P2=11.09,3.104;ReadCounts=3633,2642,490,142,2152,3490\n",
      "EM_079517\t9821\t.\tC\tT\t81180.5\tPASS\tBaseCalledReadsWithVariant=4469;BaseCalledFrequency=0.784035;TotalReads=12616;SupportingReads=11931;SupportFraction=0.945704;Template=4.815,8.505;Comp.P1=0.4095,5.404;Comp.P2=4.634,8.546;ReadCounts=3663,2644,490,143,2154,3522\n",
      "EM_079517\t10096\t.\tT\tG\t75910.8\tPASS\tBaseCalledReadsWithVariant=2101;BaseCalledFrequency=0.515204;TotalReads=11110;SupportingReads=10444;SupportFraction=0.940054;Template=11.38,1.909;Comp.P1=6.674,3.831;Comp.P2=12.51,2.529;ReadCounts=3233,2322,454,92,1868,3141\n",
      "EM_079517\t10315\t.\tC\tT\t41963.6\tPASS\tBaseCalledReadsWithVariant=2464;BaseCalledFrequency=0.549755;TotalReads=11212;SupportingReads=10580;SupportFraction=0.943632;Template=3.304,1.879;Comp.P1=3.379,2.637;Comp.P2=5.168,4.826;ReadCounts=3254,2352,482,93,1870,3161\n",
      "EM_079517\t10407\t.\tG\tT\t43031.8\tPASS\tBaseCalledReadsWithVariant=2241;BaseCalledFrequency=0.634305;TotalReads=11270;SupportingReads=10289;SupportFraction=0.912955;Template=5.326,1.981;Comp.P1=3.067,2.867;Comp.P2=7.17,1.799;ReadCounts=3268,2367,491,93,1876,3175\n",
      "EM_079517\t10451\t.\tT\tA\t80382.5\tPASS\tBaseCalledReadsWithVariant=3593;BaseCalledFrequency=0.767735;TotalReads=11290;SupportingReads=10026;SupportFraction=0.888043;Template=0.9218,13.02;Comp.P1=0.3621,9.713;Comp.P2=3.602,12.15;ReadCounts=3272,2373,491,93,1882,3179\n",
      "EM_079517\t10755\t.\tG\tA\t28891.7\tPASS\tBaseCalledReadsWithVariant=3922;BaseCalledFrequency=0.77053;TotalReads=11132;SupportingReads=8899;SupportFraction=0.799407;Template=3.454,4.685;Comp.P1=1.188,2.885;Comp.P2=0.7758,1.44;ReadCounts=3235,2331,469,94,1862,3141\n",
      "EM_079517\t11106\t.\tT\tA\t13699.2\tPASS\tBaseCalledReadsWithVariant=992;BaseCalledFrequency=0.915974;TotalReads=2396;SupportingReads=2057;SupportFraction=0.858514;Template=7.802,2.412;Comp.P1=4.356,0.5249;Comp.P2=10.48,0.9378;ReadCounts=335,863,57,12,806,323\n",
      "EM_079517\t11215\t.\tG\tC\t25747.8\tPASS\tBaseCalledReadsWithVariant=1042;BaseCalledFrequency=0.886054;TotalReads=2412;SupportingReads=2340;SupportFraction=0.970149;Template=12.77,11.55;Comp.P1=6.368,6.513;Comp.P2=8.107,13.5;ReadCounts=336,870,58,11,812,325\n",
      "EM_079517\t12568\t.\tT\tA\t18779.5\tPASS\tBaseCalledReadsWithVariant=793;BaseCalledFrequency=0.699912;TotalReads=3731;SupportingReads=3550;SupportFraction=0.951488;Template=5.267,6.924;Comp.P1=3.281,1.243;Comp.P2=2.66,5.4;ReadCounts=999,866,121,62,745,938\n",
      "EM_079517\t13856\t.\tA\tG\t28880.2\tPASS\tBaseCalledReadsWithVariant=1485;BaseCalledFrequency=0.896198;TotalReads=4146;SupportingReads=3892;SupportFraction=0.938736;Template=4.402,8.072;Comp.P1=3.357,6.748;Comp.P2=6.406,9.59;ReadCounts=1098,975,136,65,839,1033\n",
      "EM_079517\t15652\t.\tC\tG\t133311.5\tPASS\tBaseCalledReadsWithVariant=5596;BaseCalledFrequency=0.856574;TotalReads=14958;SupportingReads=13875;SupportFraction=0.927597;Template=15.94,3.273;Comp.P1=4.573,5.404;Comp.P2=13.84,3.18;ReadCounts=3632,3847,288,315,3559,3317\n",
      "EM_079517\t15660\t.\tT\tC\t133311.5\tPASS\tBaseCalledReadsWithVariant=4508;BaseCalledFrequency=0.641252;TotalReads=14958;SupportingReads=13875;SupportFraction=0.927597;Template=15.94,3.273;Comp.P1=4.573,5.404;Comp.P2=13.84,3.18;ReadCounts=3632,3847,288,315,3559,3317\n",
      "EM_079517\t15802\t.\tC\tA\t1468.8\tPASS\tBaseCalledReadsWithVariant=133;BaseCalledFrequency=0.773256;TotalReads=358;SupportingReads=293;SupportFraction=0.818436;Template=4.999,1.261;Comp.P1=4.236,-2.144;Comp.P2=5.108,6.874;ReadCounts=72,107,5,7,102,65\n",
      "EM_079517\t15963\t.\tG\tA\t2249.6\tPASS\tBaseCalledReadsWithVariant=61;BaseCalledFrequency=0.60396;TotalReads=392;SupportingReads=370;SupportFraction=0.943878;Template=2.057,7.573;Comp.P1=0.2825,5.837;Comp.P2=4.437,9.197;ReadCounts=75,121,5,8,116,67\n",
      "EM_079517\t16119\t.\tC\tA\t1610.4\tPASS\tBaseCalledReadsWithVariant=153;BaseCalledFrequency=0.827027;TotalReads=394;SupportingReads=363;SupportFraction=0.92132;Template=3.675,2.804;Comp.P1=2.642,-1.597;Comp.P2=4.623,6.737;ReadCounts=75,122,5,8,117,67\n",
      "EM_079517\t16352\t.\tG\tA\t1582.5\tPASS\tBaseCalledReadsWithVariant=103;BaseCalledFrequency=0.620482;TotalReads=398;SupportingReads=354;SupportFraction=0.889447;Template=3.08,2.055;Comp.P1=1.778,0.4456;Comp.P2=5.74,5.988;ReadCounts=75,124,5,8,119,67\n",
      "EM_079517\t16514\t.\tG\tA\t3098.3\tPASS\tBaseCalledReadsWithVariant=122;BaseCalledFrequency=0.638743;TotalReads=406;SupportingReads=372;SupportFraction=0.916256;Template=3.882,9.819;Comp.P1=2.013,4.131;Comp.P2=3.887,15.28;ReadCounts=76,127,5,8,122,68\n",
      "EM_079517\t17142\t.\tT\tC\t23954.2\tPASS\tBaseCalledReadsWithVariant=1341;BaseCalledFrequency=0.803475;TotalReads=3678;SupportingReads=3403;SupportFraction=0.925231;Template=9.809,4.857;Comp.P1=7.607,3.077;Comp.P2=6.038,6.324;ReadCounts=803,1036,42,109,994,694\n",
      "EM_079517\t17661\t.\tT\tC\t18922.7\tPASS\tBaseCalledReadsWithVariant=1345;BaseCalledFrequency=0.82263;TotalReads=3706;SupportingReads=3477;SupportFraction=0.938208;Template=4.707,4.387;Comp.P1=4.76,1.533;Comp.P2=4.165,8.588;ReadCounts=802,1051,44,107,1007,695\n",
      "EM_079517\t17864\t.\tT\tA\t18043.5\tPASS\tBaseCalledReadsWithVariant=1233;BaseCalledFrequency=0.864656;TotalReads=3700;SupportingReads=3072;SupportFraction=0.83027;Template=7.906,1.169;Comp.P1=4.196,3.229;Comp.P2=8.857,1.507;ReadCounts=801,1049,42,106,1007,695\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=076533_180Genomes_11rx\n",
      "+ poretools_dir=076533_180Genomes_11rx\n",
      "+ sample_tag=076533_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "++ tempfile\n",
      "+ tmp=/tmp/fileCxJXsZ\n",
      "+ poretools fastq --type 2D ../data/076533_180Genomes_11rx/pass\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/pass/NBPOR1137_076533_180Genomes_11rx_3815_1_ch393_file43_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/pass/NBPOR1137_076533_180Genomes_11rx_3815_1_ch385_file68_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ poretools fasta --type 2D ../data/076533_180Genomes_11rx/pass\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/pass/NBPOR1137_076533_180Genomes_11rx_3815_1_ch393_file43_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/pass/NBPOR1137_076533_180Genomes_11rx_3815_1_ch385_file68_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ '[' hq == hq ']'\n",
      "+ poretools fastq --type 2D --high-quality ../data/076533_180Genomes_11rx/fail\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch397_file198_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch3_file26_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch402_file180_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch395_file8_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch3_file3_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch403_file59_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch402_file102_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch38_file45_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch400_file91_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ poretools fasta --type 2D --high-quality ../data/076533_180Genomes_11rx/fail\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch397_file198_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch3_file26_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch402_file180_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch395_file8_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch3_file3_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch403_file59_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch402_file102_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch38_file45_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch400_file91_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ '[' '!' na == na ']'\n",
      "+ mv /tmp/fileCxJXsZ 076533_180Genomes_11rx_hq.fastq\n",
      "+ mv /tmp/fileCxJXsZ.fasta 076533_180Genomes_11rx_hq.fasta\n",
      "+ '[' '!' -e ../refs/EM_079517.fasta.amb ']'\n",
      "+ rm -rf jobTree_align_EM_079517_076533_180Genomes_11rx_hq\n",
      "+ marginAlign 076533_180Genomes_11rx_hq.fastq ../refs/EM_079517.fasta EM_079517_076533_180Genomes_11rx_hq.sam --jobTree jobTree_align_EM_079517_076533_180Genomes_11rx_hq --inputModel ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446201966.79 : Going to chain sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_076533_180Genomes_11rx_hq/jobs/gTD0/tmp_Hxv22wG7g5/temp.sam\n",
      "Got message from job at time: 1446201966.82 : Going to realign sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_076533_180Genomes_11rx_hq/jobs/gTD1/tmp_lmtGK0Lc2r/temp.sam to create output sam file: EM_079517_076533_180Genomes_11rx_hq.sam     with match gamma 0.5 and gap gamma 0.0 and model ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446202388.07 : Realigned sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_076533_180Genomes_11rx_hq/jobs/gTD1/tmp_lmtGK0Lc2r/temp.sam to create output sam file: EM_079517_076533_180Genomes_11rx_hq.sam\n",
      "+ align_trim.py\n",
      "+ samtools sort - EM_079517_076533_180Genomes_11rx_hq_marginalign.sorted\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_076533_180Genomes_11rx_hq_marginalign.sorted.bam\n",
      "+ rm 076533_180Genomes_11rx_hq.fasta.fast5.fofn\n",
      "+ nanopolish eventalign --reads 076533_180Genomes_11rx_hq.fasta -b EM_079517_076533_180Genomes_11rx_hq_marginalign.sorted.bam -g ../refs/EM_079517.fasta --sam\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_076533_180Genomes_11rx_hq_np.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_076533_180Genomes_11rx_hq_np.sorted.bam\n",
      "+ nanopolish variants --models-fofn offset_models.fofn --progress -t 1 --reads 076533_180Genomes_11rx_hq.fasta -o np_EM_079517_076533_180Genomes_11rx_hq.vcf -b EM_079517_076533_180Genomes_11rx_hq_marginalign.sorted.bam -e EM_079517_076533_180Genomes_11rx_hq_np.sorted.bam -g ../refs/EM_079517.fasta -vv -w EM_079517:0-20000 --snp\n",
      "TODO: train model\n",
      "TODO: filter data\n",
      "EM_079517\t40\t.\tG\tT\t19.5\tPASS\tBaseCalledReadsWithVariant=1405;BaseCalledFrequency=0.431511;TotalReads=40;SupportingReads=28;SupportFraction=0.7;Template=1.531,-0.1774;Comp.P1=2.036,-nan;Comp.P2=0.8452,-nan;ReadCounts=1,17,2,0,20,0\n",
      "EM_079517\t213\t.\tG\tA\t34418.4\tPASS\tBaseCalledReadsWithVariant=2155;BaseCalledFrequency=0.626818;TotalReads=8641;SupportingReads=7914;SupportFraction=0.915866;Template=5.603,3.606;Comp.P1=3.068,-0.7354;Comp.P2=4.697,2.418;ReadCounts=2377,1943,66,215,1878,2162\n",
      "EM_079517\t800\t.\tC\tT\t8281.7\tPASS\tBaseCalledReadsWithVariant=437;BaseCalledFrequency=0.790235;TotalReads=1628;SupportingReads=1497;SupportFraction=0.919533;Template=6.475,1.724;Comp.P1=4.707,0.6099;Comp.P2=9.567,3.092;ReadCounts=386,428,17,65,411,321\n",
      "EM_079517\t1849\t.\tT\tC\t67337.7\tPASS\tBaseCalledReadsWithVariant=2990;BaseCalledFrequency=0.564577;TotalReads=15631;SupportingReads=11322;SupportFraction=0.72433;Template=8.136,-1.488;Comp.P1=4.174,-0.3215;Comp.P2=9.583,1.099;ReadCounts=4073,3743,73,126,3670,3946\n",
      "EM_079517\t2037\t.\tA\tT\t5490.1\tPASS\tBaseCalledReadsWithVariant=255;BaseCalledFrequency=0.478424;TotalReads=1248;SupportingReads=1047;SupportFraction=0.838942;Template=4.47,3.638;Comp.P1=3.232,1.736;Comp.P2=2.751,7.241;ReadCounts=350,274,43,69,231,281\n",
      "EM_079517\t2124\t.\tG\tA\t8052.1\tPASS\tBaseCalledReadsWithVariant=457;BaseCalledFrequency=0.701997;TotalReads=1468;SupportingReads=1214;SupportFraction=0.826975;Template=9.295,2.436;Comp.P1=4.189,2.912;Comp.P2=6.199,4.217;ReadCounts=398,336,52,94,284,304\n",
      "EM_079517\t2233\t.\tT\tC\t8100.7\tPASS\tBaseCalledReadsWithVariant=466;BaseCalledFrequency=0.681287;TotalReads=1562;SupportingReads=1267;SupportFraction=0.81114;Template=5.477,6.022;Comp.P1=3.649,3.57;Comp.P2=4.015,5.762;ReadCounts=414,366,58,97,310,317\n",
      "EM_079517\t4661\t.\tT\tC\t15923.6\tPASS\tBaseCalledReadsWithVariant=1048;BaseCalledFrequency=0.767204;TotalReads=3402;SupportingReads=3120;SupportFraction=0.917108;Template=3.847,3.734;Comp.P1=5.644,1.849;Comp.P2=4.746,6.281;ReadCounts=800,901,285,17,616,783\n",
      "EM_079517\t6283\t.\tC\tT\t904.4\tPASS\tBaseCalledReadsWithVariant=56;BaseCalledFrequency=0.643678;TotalReads=212;SupportingReads=197;SupportFraction=0.929245;Template=7.617,2.326;Comp.P1=4.871,1.87;Comp.P2=5.406,1.638;ReadCounts=57,49,6,5,43,52\n",
      "EM_079517\t6909\t.\tA\tT\t2785.1\tPASS\tBaseCalledReadsWithVariant=152;BaseCalledFrequency=0.915663;TotalReads=386;SupportingReads=368;SupportFraction=0.953368;Template=7.474,7.737;Comp.P1=4.441,5.048;Comp.P2=7.602,7.108;ReadCounts=115,78,24,6,54,109\n",
      "EM_079517\t7732\t.\tG\tA\t410.6\tPASS\tBaseCalledReadsWithVariant=35;BaseCalledFrequency=0.603448;TotalReads=142;SupportingReads=116;SupportFraction=0.816901;Template=1.808,2.264;Comp.P1=0.9987,2.703;Comp.P2=1.229,6.373;ReadCounts=39,32,8,5,24,34\n",
      "EM_079517\t8928\t.\tA\tC\t1416.3\tPASS\tBaseCalledReadsWithVariant=118;BaseCalledFrequency=0.531532;TotalReads=510;SupportingReads=454;SupportFraction=0.890196;Template=3.343,4.59;Comp.P1=1.642,3.421;Comp.P2=1.853,0.9535;ReadCounts=125,130,13,18,117,107\n",
      "EM_079517\t11204\t.\tT\tG\t7230.8\tPASS\tBaseCalledReadsWithVariant=422;BaseCalledFrequency=0.695222;TotalReads=1290;SupportingReads=1190;SupportFraction=0.922481;Template=2.974,8.129;Comp.P1=1.908,2.66;Comp.P2=4.101,6.818;ReadCounts=213,432,46,9,386,204\n",
      "EM_079517\t13390\t.\tA\tG\t7985.9\tPASS\tBaseCalledReadsWithVariant=650;BaseCalledFrequency=0.823828;TotalReads=1710;SupportingReads=1572;SupportFraction=0.919298;Template=4.096,5.307;Comp.P1=1.533,1.418;Comp.P2=6.264,3.942;ReadCounts=450,405,50,32,355,418\n",
      "EM_079517\t13856\t.\tA\tG\t11638.3\tPASS\tBaseCalledReadsWithVariant=607;BaseCalledFrequency=0.900593;TotalReads=1668;SupportingReads=1554;SupportFraction=0.931655;Template=4.423,8.105;Comp.P1=2.936,6.851;Comp.P2=6.729,9.335;ReadCounts=444,390,47,31,343,413\n",
      "EM_079517\t15313\t.\tG\tC\t79210.3\tPASS\tBaseCalledReadsWithVariant=3427;BaseCalledFrequency=0.760879;TotalReads=9604;SupportingReads=9060;SupportFraction=0.943357;Template=8.683,6.635;Comp.P1=3.75,5.183;Comp.P2=7.953,10.27;ReadCounts=2454,2348,226,118,2122,2336\n",
      "EM_079517\t15660\t.\tT\tC\t35551.8\tPASS\tBaseCalledReadsWithVariant=2437;BaseCalledFrequency=0.637291;TotalReads=8202;SupportingReads=6660;SupportFraction=0.811997;Template=7.592,0.66;Comp.P1=2.391,1.12;Comp.P2=8.234,1.248;ReadCounts=2085,2016,137,92,1879,1993\n",
      "EM_079517\t15933\t.\tA\tG\t1540.0\tPASS\tBaseCalledReadsWithVariant=151;BaseCalledFrequency=0.904192;TotalReads=336;SupportingReads=288;SupportFraction=0.857143;Template=6.802,4.259;Comp.P1=5.282,0.6668;Comp.P2=2.8,6.227;ReadCounts=65,103,2,7,101,58\n",
      "EM_079517\t15963\t.\tG\tA\t2080.3\tPASS\tBaseCalledReadsWithVariant=58;BaseCalledFrequency=0.591837;TotalReads=344;SupportingReads=321;SupportFraction=0.93314;Template=1.954,7.768;Comp.P1=0.5893,6.039;Comp.P2=4.801,9.985;ReadCounts=66,106,2,8,104,58\n",
      "EM_079517\t17142\t.\tT\tC\t13209.7\tPASS\tBaseCalledReadsWithVariant=727;BaseCalledFrequency=0.794536;TotalReads=2060;SupportingReads=1868;SupportFraction=0.906796;Template=9.609,5.607;Comp.P1=5.645,2.962;Comp.P2=5.625,6.243;ReadCounts=397,633,27,59,606,338\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=076533_180Genomes_11rx\n",
      "+ poretools_dir=076533_180Genomes_11rx\n",
      "+ sample_tag=076533_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "++ tempfile\n",
      "+ tmp=/tmp/fileejxMzZ\n",
      "+ poretools fastq --type 2D ../data/076533_180Genomes_11rx/pass\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/pass/NBPOR1137_076533_180Genomes_11rx_3815_1_ch393_file43_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/pass/NBPOR1137_076533_180Genomes_11rx_3815_1_ch385_file68_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ poretools fasta --type 2D ../data/076533_180Genomes_11rx/pass\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/pass/NBPOR1137_076533_180Genomes_11rx_3815_1_ch393_file43_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/pass/NBPOR1137_076533_180Genomes_11rx_3815_1_ch385_file68_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ '[' hq == hq ']'\n",
      "+ poretools fastq --type 2D --high-quality ../data/076533_180Genomes_11rx/fail\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch397_file198_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch3_file26_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch402_file180_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch395_file8_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch3_file3_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch403_file59_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch402_file102_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch38_file45_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch400_file91_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ poretools fasta --type 2D --high-quality ../data/076533_180Genomes_11rx/fail\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch397_file198_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch3_file26_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch402_file180_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch395_file8_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch3_file3_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch403_file59_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch402_file102_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch38_file45_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "WARNING:poretools:Cannot open file: ../data/076533_180Genomes_11rx/fail/NBPOR1137_076533_180Genomes_11rx_3815_1_ch400_file91_strand.fast5. Perhaps it is corrupt? Moving on.\n",
      "\n",
      "+ '[' '!' na == na ']'\n",
      "+ mv /tmp/fileejxMzZ 076533_180Genomes_11rx_hq.fastq\n",
      "+ mv /tmp/fileejxMzZ.fasta 076533_180Genomes_11rx_hq.fasta\n",
      "+ '[' '!' -e ../refs/EM_079517_mut30_2.fasta.amb ']'\n",
      "+ rm -rf jobTree_align_EM_079517_mut30_2_076533_180Genomes_11rx_hq\n",
      "+ marginAlign 076533_180Genomes_11rx_hq.fastq ../refs/EM_079517_mut30_2.fasta EM_079517_mut30_2_076533_180Genomes_11rx_hq.sam --jobTree jobTree_align_EM_079517_mut30_2_076533_180Genomes_11rx_hq --inputModel ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446201966.74 : Going to chain sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_076533_180Genomes_11rx_hq/jobs/gTD0/tmp_dD1MMF0Hxl/temp.sam\n",
      "Got message from job at time: 1446201966.74 : Going to realign sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_076533_180Genomes_11rx_hq/jobs/gTD1/tmp_7Wnj0B8UEv/temp.sam to create output sam file: EM_079517_mut30_2_076533_180Genomes_11rx_hq.sam     with match gamma 0.5 and gap gamma 0.0 and model ../models/input_10_04.hmm\n",
      "Got message from job at time: 1446202384.67 : Realigned sam file: /mnt/ebola/ebov/180genomes-validation/jobTree_align_EM_079517_mut30_2_076533_180Genomes_11rx_hq/jobs/gTD1/tmp_7Wnj0B8UEv/temp.sam to create output sam file: EM_079517_mut30_2_076533_180Genomes_11rx_hq.sam\n",
      "+ align_trim.py\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_076533_180Genomes_11rx_hq_marginalign.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_mut30_2_076533_180Genomes_11rx_hq_marginalign.sorted.bam\n",
      "+ rm 076533_180Genomes_11rx_hq.fasta.fast5.fofn\n",
      "rm: cannot remove ‘076533_180Genomes_11rx_hq.fasta.fast5.fofn’: No such file or directory\n",
      "+ nanopolish eventalign --reads 076533_180Genomes_11rx_hq.fasta -b EM_079517_mut30_2_076533_180Genomes_11rx_hq_marginalign.sorted.bam -g ../refs/EM_079517_mut30_2.fasta --sam\n",
      "+ samtools sort - EM_079517_mut30_2_076533_180Genomes_11rx_hq_np.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "+ samtools index EM_079517_mut30_2_076533_180Genomes_11rx_hq_np.sorted.bam\n",
      "+ nanopolish variants --models-fofn offset_models.fofn --progress -t 1 --reads 076533_180Genomes_11rx_hq.fasta -o np_EM_079517_mut30_2_076533_180Genomes_11rx_hq.vcf -b EM_079517_mut30_2_076533_180Genomes_11rx_hq_marginalign.sorted.bam -e EM_079517_mut30_2_076533_180Genomes_11rx_hq_np.sorted.bam -g ../refs/EM_079517_mut30_2.fasta -vv -w EM_079517:0-20000 --snp\n",
      "TODO: train model\n",
      "TODO: filter data\n",
      "EM_079517\t40\t.\tG\tT\t19.5\tPASS\tBaseCalledReadsWithVariant=1404;BaseCalledFrequency=0.431336;TotalReads=40;SupportingReads=28;SupportFraction=0.7;Template=1.531,-0.1774;Comp.P1=2.036,-nan;Comp.P2=0.8452,-nan;ReadCounts=1,17,2,0,20,0\n",
      "EM_079517\t213\t.\tG\tA\t34411.4\tPASS\tBaseCalledReadsWithVariant=2155;BaseCalledFrequency=0.627183;TotalReads=8637;SupportingReads=7911;SupportFraction=0.915943;Template=5.603,3.606;Comp.P1=3.063,-0.7138;Comp.P2=4.697,2.418;ReadCounts=2376,1942,65,214,1878,2162\n",
      "EM_079517\t672\t.\tT\tA\t12887.7\tPASS\tBaseCalledReadsWithVariant=415;BaseCalledFrequency=0.83501;TotalReads=1622;SupportingReads=1540;SupportFraction=0.949445;Template=10.09,4.778;Comp.P1=6.8,4.324;Comp.P2=11.79,5.438;ReadCounts=383,428,16,64,412,319\n",
      "EM_079517\t800\t.\tC\tT\t8265.8\tPASS\tBaseCalledReadsWithVariant=437;BaseCalledFrequency=0.788809;TotalReads=1626;SupportingReads=1495;SupportFraction=0.919434;Template=6.485,1.728;Comp.P1=4,0.6099;Comp.P2=9.567,3.092;ReadCounts=385,428,17,65,411,320\n",
      "EM_079517\t927\t.\tC\tT\t13906.3\tPASS\tBaseCalledReadsWithVariant=637;BaseCalledFrequency=0.90483;TotalReads=1630;SupportingReads=1472;SupportFraction=0.903067;Template=14.53,2.424;Comp.P1=5.884,2.678;Comp.P2=15.43,1.984;ReadCounts=384,431,18,61,413,323\n",
      "EM_079517\t1018\t.\tC\tA\t11945.4\tPASS\tBaseCalledReadsWithVariant=605;BaseCalledFrequency=0.866762;TotalReads=1532;SupportingReads=1419;SupportFraction=0.92624;Template=2.884,11.76;Comp.P1=2.261,4.732;Comp.P2=4.09,13.71;ReadCounts=362,404,14,49,390,313\n",
      "EM_079517\t1849\t.\tT\tC\t67344.8\tPASS\tBaseCalledReadsWithVariant=2990;BaseCalledFrequency=0.56447;TotalReads=15633;SupportingReads=11324;SupportFraction=0.724365;Template=8.135,-1.488;Comp.P1=4.174,-0.3215;Comp.P2=9.583,1.099;ReadCounts=4074,3743,73,126,3670,3947\n",
      "EM_079517\t2003\t.\tG\tC\t5508.3\tPASS\tBaseCalledReadsWithVariant=344;BaseCalledFrequency=0.744589;TotalReads=1100;SupportingReads=988;SupportFraction=0.898182;Template=8.231,2.56;Comp.P1=7.631,4.349;Comp.P2=4.686,3.328;ReadCounts=316,234,35,56,199,260\n",
      "EM_079517\t2037\t.\tA\tT\t5449.0\tPASS\tBaseCalledReadsWithVariant=253;BaseCalledFrequency=0.485605;TotalReads=1219;SupportingReads=1025;SupportFraction=0.840853;Template=4.589,3.734;Comp.P1=3.887,1.859;Comp.P2=2.731,7.105;ReadCounts=352,257,40,72,218,280\n",
      "EM_079517\t2116\t.\tC\tA\t18598.0\tPASS\tBaseCalledReadsWithVariant=393;BaseCalledFrequency=0.63285;TotalReads=1434;SupportingReads=1316;SupportFraction=0.917713;Template=19.51,6.405;Comp.P1=9.487,6.756;Comp.P2=14.87,12.33;ReadCounts=392,325,52,93,273,299\n",
      "EM_079517\t2124\t.\tG\tA\t18598.0\tPASS\tBaseCalledReadsWithVariant=465;BaseCalledFrequency=0.705615;TotalReads=1434;SupportingReads=1316;SupportFraction=0.917713;Template=19.51,6.405;Comp.P1=9.487,6.756;Comp.P2=14.87,12.33;ReadCounts=392,325,52,93,273,299\n",
      "EM_079517\t2233\t.\tT\tC\t8061.8\tPASS\tBaseCalledReadsWithVariant=464;BaseCalledFrequency=0.676385;TotalReads=1562;SupportingReads=1262;SupportFraction=0.807939;Template=5.436,6.024;Comp.P1=3.522,3.452;Comp.P2=4.02,5.756;ReadCounts=415,365,58,99,309,316\n",
      "EM_079517\t2287\t.\tT\tA\t4499.8\tPASS\tBaseCalledReadsWithVariant=299;BaseCalledFrequency=0.416435;TotalReads=1734;SupportingReads=1297;SupportFraction=0.747982;Template=3.567,-0.4913;Comp.P1=1.433,1.093;Comp.P2=1.361,6.845;ReadCounts=456,411,61,106,350,350\n",
      "EM_079517\t2418\t.\tC\tA\t6448.8\tPASS\tBaseCalledReadsWithVariant=409;BaseCalledFrequency=0.734291;TotalReads=1803;SupportingReads=1381;SupportFraction=0.765946;Template=5.224,0.2519;Comp.P1=2.646,1.505;Comp.P2=8.135,1.586;ReadCounts=475,427,65,109,362,365\n",
      "EM_079517\t2535\t.\tA\tC\t13919.3\tPASS\tBaseCalledReadsWithVariant=528;BaseCalledFrequency=0.701195;TotalReads=1830;SupportingReads=1737;SupportFraction=0.94918;Template=9.501,11;Comp.P1=4.324,7.321;Comp.P2=4.039,5.384;ReadCounts=482,433,66,110,367,372\n",
      "EM_079517\t2811\t.\tT\tC\t16106.7\tPASS\tBaseCalledReadsWithVariant=723;BaseCalledFrequency=0.844626;TotalReads=1882;SupportingReads=1729;SupportFraction=0.918704;Template=7.759,10.74;Comp.P1=3.695,9.466;Comp.P2=8.773,7.416;ReadCounts=496,445,66,114,379,382\n",
      "EM_079517\t4360\t.\tC\tA\t24517.2\tPASS\tBaseCalledReadsWithVariant=1019;BaseCalledFrequency=0.688049;TotalReads=3138;SupportingReads=2878;SupportFraction=0.917145;Template=9.747,9.617;Comp.P1=5.834,3.189;Comp.P2=6.601,5.539;ReadCounts=747,822,249,18,573,729\n",
      "EM_079517\t4550\t.\tT\tA\t24063.2\tPASS\tBaseCalledReadsWithVariant=1234;BaseCalledFrequency=0.824866;TotalReads=3266;SupportingReads=2941;SupportFraction=0.90049;Template=9.41,7.833;Comp.P1=3.87,5.881;Comp.P2=5.615,7.395;ReadCounts=772,861,266,16,595,756\n",
      "EM_079517\t4661\t.\tT\tC\t15934.0\tPASS\tBaseCalledReadsWithVariant=1048;BaseCalledFrequency=0.766082;TotalReads=3402;SupportingReads=3122;SupportFraction=0.917695;Template=3.85,3.734;Comp.P1=5.644,1.843;Comp.P2=4.744,6.292;ReadCounts=800,901,285,17,616,783\n",
      "EM_079517\t5042\t.\tC\tT\t17167.3\tPASS\tBaseCalledReadsWithVariant=1324;BaseCalledFrequency=0.842239;TotalReads=3512;SupportingReads=3054;SupportFraction=0.86959;Template=8.694,2.275;Comp.P1=2.968,1.581;Comp.P2=7.645,2.6;ReadCounts=827,928,293,13,635,816\n",
      "EM_079517\t6283\t.\tC\tT\t904.4\tPASS\tBaseCalledReadsWithVariant=56;BaseCalledFrequency=0.643678;TotalReads=212;SupportingReads=197;SupportFraction=0.929245;Template=7.617,2.326;Comp.P1=4.871,1.87;Comp.P2=5.406,1.638;ReadCounts=57,49,6,5,43,52\n",
      "EM_079517\t6619\t.\tC\tT\t2826.6\tPASS\tBaseCalledReadsWithVariant=148;BaseCalledFrequency=0.81768;TotalReads=386;SupportingReads=351;SupportFraction=0.909326;Template=11.31,3.666;Comp.P1=5.497,1.056;Comp.P2=10.28,4.972;ReadCounts=115,78,23,6,55,109\n",
      "EM_079517\t6895\t.\tT\tG\t2018.2\tPASS\tBaseCalledReadsWithVariant=109;BaseCalledFrequency=0.784173;TotalReads=386;SupportingReads=351;SupportFraction=0.909326;Template=9.579,2.35;Comp.P1=4.434,3.832;Comp.P2=6.236,2.452;ReadCounts=115,78,24,6,54,109\n",
      "EM_079517\t6909\t.\tA\tT\t2789.8\tPASS\tBaseCalledReadsWithVariant=153;BaseCalledFrequency=0.916168;TotalReads=386;SupportingReads=368;SupportFraction=0.953368;Template=7.555,7.731;Comp.P1=4.265,5.046;Comp.P2=7.602,7.109;ReadCounts=115,78,24,6,54,109\n",
      "EM_079517\t7732\t.\tG\tA\t410.6\tPASS\tBaseCalledReadsWithVariant=35;BaseCalledFrequency=0.603448;TotalReads=142;SupportingReads=116;SupportFraction=0.816901;Template=1.808,2.264;Comp.P1=0.9987,2.703;Comp.P2=1.229,6.373;ReadCounts=39,32,8,5,24,34\n",
      "EM_079517\t7956\t.\tT\tA\t931.1\tPASS\tBaseCalledReadsWithVariant=44;BaseCalledFrequency=0.814815;TotalReads=160;SupportingReads=144;SupportFraction=0.9;Template=8.292,6.152;Comp.P1=6.71,-0.4425;Comp.P2=5.59,3.626;ReadCounts=41,39,8,6,31,35\n",
      "EM_079517\t8625\t.\tG\tC\t2609.4\tPASS\tBaseCalledReadsWithVariant=187;BaseCalledFrequency=0.813043;TotalReads=496;SupportingReads=455;SupportFraction=0.917339;Template=3.723,7.946;Comp.P1=2.335,4.521;Comp.P2=3.711,6.005;ReadCounts=122,126,12,20,114,102\n",
      "EM_079517\t8778\t.\tG\tC\t1836.6\tPASS\tBaseCalledReadsWithVariant=79;BaseCalledFrequency=0.642276;TotalReads=502;SupportingReads=479;SupportFraction=0.954183;Template=2.089,3.525;Comp.P1=2.253,1.311;Comp.P2=5.296,4.473;ReadCounts=123,128,13,19,115,104\n",
      "EM_079517\t8928\t.\tA\tC\t1414.5\tPASS\tBaseCalledReadsWithVariant=118;BaseCalledFrequency=0.531532;TotalReads=510;SupportingReads=454;SupportFraction=0.890196;Template=3.329,4.59;Comp.P1=1.642,3.421;Comp.P2=1.853,0.9535;ReadCounts=125,130,13,18,117,107\n",
      "EM_079517\t9780\t.\tT\tG\t50137.6\tPASS\tBaseCalledReadsWithVariant=2189;BaseCalledFrequency=0.808943;TotalReads=6842;SupportingReads=6228;SupportFraction=0.91026;Template=13.7,1.997;Comp.P1=1.908,1.874;Comp.P2=10.98,3.22;ReadCounts=2002,1419,226,67,1193,1935\n",
      "EM_079517\t10096\t.\tT\tG\t33857.1\tPASS\tBaseCalledReadsWithVariant=798;BaseCalledFrequency=0.492593;TotalReads=4846;SupportingReads=4492;SupportFraction=0.92695;Template=11.34,2.411;Comp.P1=7,2.744;Comp.P2=12.4,2.743;ReadCounts=1451,972,180,34,792,1417\n",
      "EM_079517\t10407\t.\tG\tT\t19512.4\tPASS\tBaseCalledReadsWithVariant=1044;BaseCalledFrequency=0.681462;TotalReads=4940;SupportingReads=4432;SupportFraction=0.897166;Template=5.494,2.539;Comp.P1=2.873,1.841;Comp.P2=6.961,1.855;ReadCounts=1474,996,191,36,805,1438\n",
      "EM_079517\t10451\t.\tT\tA\t33280.7\tPASS\tBaseCalledReadsWithVariant=1620;BaseCalledFrequency=0.786026;TotalReads=4963;SupportingReads=4338;SupportFraction=0.874068;Template=0.9516,13.13;Comp.P1=0.06909,5.62;Comp.P2=3.411,10.9;ReadCounts=1479,1003,194,36,809,1442\n",
      "EM_079517\t10755\t.\tG\tA\t14230.0\tPASS\tBaseCalledReadsWithVariant=1667;BaseCalledFrequency=0.766085;TotalReads=4856;SupportingReads=3888;SupportFraction=0.800659;Template=4.061,5.334;Comp.P1=1.179,2.365;Comp.P2=0.9489,1.461;ReadCounts=1446,982,188,36,794,1410\n",
      "EM_079517\t11106\t.\tT\tA\t7383.1\tPASS\tBaseCalledReadsWithVariant=508;BaseCalledFrequency=0.900709;TotalReads=1256;SupportingReads=1091;SupportFraction=0.868631;Template=8.216,3.22;Comp.P1=3.816,0.8902;Comp.P2=10.42,1.1;ReadCounts=209,419,42,9,377,200\n",
      "EM_079517\t11204\t.\tT\tG\t7191.7\tPASS\tBaseCalledReadsWithVariant=423;BaseCalledFrequency=0.699174;TotalReads=1286;SupportingReads=1181;SupportFraction=0.918351;Template=2.981,8.092;Comp.P1=1.798,2.761;Comp.P2=4.098,6.842;ReadCounts=213,430,46,9,384,204\n",
      "EM_079517\t11215\t.\tG\tC\t13387.4\tPASS\tBaseCalledReadsWithVariant=542;BaseCalledFrequency=0.882736;TotalReads=1288;SupportingReads=1215;SupportFraction=0.943323;Template=12.49,10.64;Comp.P1=5.273,6.065;Comp.P2=8.26,13.05;ReadCounts=213,431,46,9,385,204\n",
      "EM_079517\t12568\t.\tT\tA\t7783.7\tPASS\tBaseCalledReadsWithVariant=298;BaseCalledFrequency=0.657837;TotalReads=1583;SupportingReads=1479;SupportFraction=0.934302;Template=5.101,7.162;Comp.P1=2.769,0.4506;Comp.P2=2.141,5.488;ReadCounts=412,379,46,27,334,385\n",
      "EM_079517\t13390\t.\tA\tG\t7985.9\tPASS\tBaseCalledReadsWithVariant=650;BaseCalledFrequency=0.823828;TotalReads=1710;SupportingReads=1572;SupportFraction=0.919298;Template=4.096,5.307;Comp.P1=1.533,1.418;Comp.P2=6.264,3.942;ReadCounts=450,405,50,32,355,418\n",
      "EM_079517\t13856\t.\tA\tG\t11638.3\tPASS\tBaseCalledReadsWithVariant=607;BaseCalledFrequency=0.900593;TotalReads=1668;SupportingReads=1554;SupportFraction=0.931655;Template=4.423,8.105;Comp.P1=2.936,6.851;Comp.P2=6.729,9.335;ReadCounts=444,390,47,31,343,413\n",
      "EM_079517\t15313\t.\tG\tC\t79207.0\tPASS\tBaseCalledReadsWithVariant=3427;BaseCalledFrequency=0.761048;TotalReads=9602;SupportingReads=9059;SupportFraction=0.943449;Template=8.685,6.635;Comp.P1=3.75,5.183;Comp.P2=7.953,10.27;ReadCounts=2453,2348,226,118,2122,2335\n",
      "EM_079517\t15652\t.\tC\tG\t74805.6\tPASS\tBaseCalledReadsWithVariant=2942;BaseCalledFrequency=0.835321;TotalReads=8138;SupportingReads=7538;SupportFraction=0.926272;Template=16.35,4.154;Comp.P1=3.988,4.478;Comp.P2=13.46,3.324;ReadCounts=2070,1999,130,91,1869,1979\n",
      "EM_079517\t15660\t.\tT\tC\t74805.6\tPASS\tBaseCalledReadsWithVariant=2417;BaseCalledFrequency=0.63505;TotalReads=8138;SupportingReads=7538;SupportFraction=0.926272;Template=16.35,4.154;Comp.P1=3.988,4.478;Comp.P2=13.46,3.324;ReadCounts=2070,1999,130,91,1869,1979\n",
      "EM_079517\t15802\t.\tC\tA\t1374.0\tPASS\tBaseCalledReadsWithVariant=130;BaseCalledFrequency=0.860927;TotalReads=316;SupportingReads=270;SupportFraction=0.85443;Template=4.108,2.095;Comp.P1=1.528,0.9031;Comp.P2=5.714,6.704;ReadCounts=63,95,2,7,93,56\n",
      "EM_079517\t15933\t.\tA\tG\t1540.0\tPASS\tBaseCalledReadsWithVariant=151;BaseCalledFrequency=0.904192;TotalReads=336;SupportingReads=288;SupportFraction=0.857143;Template=6.802,4.259;Comp.P1=5.282,0.6668;Comp.P2=2.8,6.227;ReadCounts=65,103,2,7,101,58\n",
      "EM_079517\t15963\t.\tG\tA\t2083.3\tPASS\tBaseCalledReadsWithVariant=58;BaseCalledFrequency=0.591837;TotalReads=344;SupportingReads=321;SupportFraction=0.93314;Template=1.954,7.796;Comp.P1=0.5893,6.039;Comp.P2=4.801,9.985;ReadCounts=66,106,2,8,104,58\n",
      "EM_079517\t16119\t.\tC\tA\t1426.4\tPASS\tBaseCalledReadsWithVariant=132;BaseCalledFrequency=0.804878;TotalReads=350;SupportingReads=311;SupportFraction=0.888571;Template=3.333,3.635;Comp.P1=4.221,0.09814;Comp.P2=4.067,6.274;ReadCounts=67,108,2,8,106,59\n",
      "EM_079517\t16352\t.\tG\tA\t1389.4\tPASS\tBaseCalledReadsWithVariant=96;BaseCalledFrequency=0.564706;TotalReads=365;SupportingReads=314;SupportFraction=0.860274;Template=2.218,2.417;Comp.P1=2.357,2.158;Comp.P2=5.37,5.723;ReadCounts=67,116,4,8,111,59\n",
      "EM_079517\t17142\t.\tT\tC\t13209.7\tPASS\tBaseCalledReadsWithVariant=727;BaseCalledFrequency=0.794536;TotalReads=2060;SupportingReads=1868;SupportFraction=0.906796;Template=9.609,5.607;Comp.P1=5.645,2.962;Comp.P2=5.625,6.243;ReadCounts=397,633,27,59,606,338\n",
      "EM_079517\t17864\t.\tT\tA\t11002.2\tPASS\tBaseCalledReadsWithVariant=675;BaseCalledFrequency=0.842697;TotalReads=2182;SupportingReads=1831;SupportFraction=0.839138;Template=7.768,1.885;Comp.P1=1.656,2.394;Comp.P2=8.769,1.847;ReadCounts=418,673,31,60,642,358\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "ROOTDIR=\"/mnt/ebola/ebov\"\n",
    "cd $ROOTDIR\n",
    "export PATH=`pwd`/bin/bwa:$PATH\n",
    "export PATH=$PATH:`pwd`/scripts\n",
    "export PATH=$PATH:`pwd`/bin/nanopolish-6mer\n",
    "export PATH=$PATH:`pwd`/bin/samtools\n",
    "export PATH=$PATH:`pwd`/bin/marginAlign\n",
    "\n",
    "mkdir -p 180genomes-validation\n",
    "cd 180genomes-validation\n",
    "\n",
    "echo \"../models/6mer/ont_complement.pop1.model\" > offset_models.fofn\n",
    "echo \"../models/6mer/ont_complement.pop2.model\" >> offset_models.fofn\n",
    "echo \"../models/6mer/ont_template.model\" >> offset_models.fofn\n",
    "\n",
    "makecommands.py ../metadata/metadata.db 180_Genomes align_defaultkmer_margin.sh | parallel -j16"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/ebola/ebov/analysis-var6mer-aln5mer-reverse\n"
     ]
    }
   ],
   "source": [
    "## After the raw files are produced, show statistics.\n",
    "\n",
    "%cd 180genomes-validation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "fh = open(\"validation_files.txt\", \"w\")\n",
    "fh.write(\"\"\"076534_180Genomes_11rx\tnp-var6mer-aln5mer\t../refs/EM_079517_EM_076534.mutations.txt\tnp_EM_079517_076534_180Genomes_11rx_hq.vcf\n",
    "076533_180Genomes_11rx\tnp-var6mer-aln5mer\t../refs/EM_079517_EM_076533.mutations.txt\tnp_EM_079517_076533_180Genomes_11rx_hq.vcf\n",
    "076383_180Genomes_11rx\tnp-var6mer-aln5mer\t../refs/EM_079517_EM_076383.mutations.txt\tnp_EM_079517_076383_180Genomes_11rx_hq.vcf\n",
    "078416_180Genomes_11rx\tnp-var6mer-aln5mer\t../refs/EM_079517_EM_078416.mutations.txt\tnp_EM_079517_078416_180Genomes_11rx_hq.vcf\n",
    "076769_180Genomes_19Rx\tnp-var6mer-aln5mer\t../refs/EM_079517_EM_076769.mutations.txt\tnp_EM_079517_076769_180Genomes_19Rx_hq.vcf\n",
    "\"\"\")\n",
    "fh.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "sample                  tag                 total_calls  mutations  TP    FP   FN   TPR\r\n",
      "076534_180Genomes_11rx  np-var6mer-aln5mer  18           17         14.0  4.0  3.0  0.823529411765\r\n",
      "076533_180Genomes_11rx  np-var6mer-aln5mer  25           19         19.0  6.0  0.0  1.0\r\n",
      "076383_180Genomes_11rx  np-var6mer-aln5mer  19           17         16.0  3.0  1.0  0.941176470588\r\n",
      "078416_180Genomes_11rx  np-var6mer-aln5mer  23           18         17.0  6.0  1.0  0.944444444444\r\n",
      "076769_180Genomes_19Rx  np-var6mer-aln5mer  20           19         19.0  1.0  0.0  1.0\r\n"
     ]
    }
   ],
   "source": [
    "!../scripts/intersection_vcf_stats.py validation_files.txt | cut -f 1,2,5,6,7,8,9,10 | column -t "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!../scripts/intersection_vcf_interrogate.py validation_files.txt > validation_stats.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "tag\tpos\tbasecalledfreq\tsupportfraction\tsupportingreads\tqual\tstate\r\n",
      "np-var6mer-aln5mer\t213\t0.676856\t0.913721\t879\t4667.5\tReal\r\n",
      "np-var6mer-aln5mer\t800\t0.866972\t0.935662\t509\t3170.2\tReal\r\n",
      "np-var6mer-aln5mer\t1849\t0.668085\t0.745399\t486\t3407.0\tReal\r\n",
      "np-var6mer-aln5mer\t2124\t0.757256\t0.895619\t695\t5721.8\tReal\r\n",
      "np-var6mer-aln5mer\t6283\t0.75\t0.933333\t56\t278.5\tReal\r\n",
      "np-var6mer-aln5mer\t6909\t0.731183\t0.955224\t256\t2162.3\tReal\r\n",
      "np-var6mer-aln5mer\t8928\t0.654135\t0.879147\t371\t1415.6\tReal\r\n",
      "np-var6mer-aln5mer\t11204\t0.824675\t0.943218\t598\t4686.3\tReal\r\n",
      "np-var6mer-aln5mer\t12208\t0.881481\t0.867069\t574\t4027.2\tReal\r\n"
     ]
    }
   ],
   "source": [
    "!head -10 validation_stats.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The rpy2.ipython extension is already loaded. To reload it, use:\n",
      "  %reload_ext rpy2.ipython\n"
     ]
    }
   ],
   "source": [
    "%load_ext rpy2.ipython"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAeAAAAHgCAMAAABKCk6nAAAC3FBMVEUAAAAAv8QBAQEDAwMEBAQF\nBQUGBgYHBwcICAgKCgoLCwsMDAwNDQ0ODg4PDw8QEBASEhITExMUFBQVFRUXFxcYGBgZGRkaGhoc\nHBwdHR0eHh4fHx8gICAhISEiIiIjIyMkJCQlJSUmJiYnJycoKCgpKSkqKiorKyssLCwtLS0uLi4v\nLy8wMDAxMTEyMjIzMzM0NDQ1NTU2NjY3Nzc4ODg5OTk6Ojo7Ozs8PDw9PT0+Pj4/Pz9AQEBBQUFC\nQkJDQ0NERERFRUVGRkZHR0dISEhJSUlKSkpLS0tMTExNTU1OTk5PT09QUFBRUVFSUlJTU1NUVFRV\nVVVWVlZXV1dYWFhZWVlaWlpbW1tcXFxdXV1eXl5fX19gYGBhYWFiYmJkZGRlZWVmZmZnZ2doaGhp\naWlqampsbGxtbW1ubm5vb29wcHBzc3N0dHR1dXV2dnZ3d3d4eHh5eXl6enp7e3t8fHx9fX1+fn5/\nf3+AgICBgYGCgoKDg4OFhYWGhoaHh4eIiIiJiYmKioqLi4uMjIyNjY2Ojo6Pj4+QkJCRkZGTk5OU\nlJSVlZWWlpaXl5eYmJiZmZmampqbm5ucnJydnZ2enp6fn5+goKChoaGioqKjo6OkpKSlpaWmpqan\np6eoqKipqamqqqqrq6usrKytra2urq6vr6+wsLCxsbGysrKzs7O0tLS1tbW2tra4uLi5ubm6urq7\nu7u8vLy9vb2+vr6/v7/AwMDBwcHCwsLDw8PExMTFxcXGxsbHx8fIyMjJycnKysrLy8vMzMzOzs7P\nz8/Q0NDR0dHT09PU1NTV1dXW1tbX19fY2NjZ2dna2trb29vc3Nzd3d3e3t7f39/h4eHi4uLj4+Pk\n5OTl5eXm5ubn5+fo6Ojp6enq6urr6+vs7Ozt7e3u7u7v7+/w8PDx8fHy8vLz8/P09PT19fX29vb3\n9/f4dm34+Pj5+fn6pJ76+vr7+/v8/Pz9/f3+/v7///+sM5rCAAATDUlEQVR4nO3dj38U9Z3HcXpw\n2sOAcHetd54HHthWwIom/FAQBS3q4Y+e5OwpUO8QTg7Pgq09UTmESk2qvcSAEQKGdiMiFeJJiKRg\ngLAQQ0ikJgX5kewm2YT8anLZ7H7/gZud3Z0Ns7Mz35nJZ2ZnHu/X41F2Mvv97meTJzsBeWw6giFX\nN8LuJ4BoA7DLA7DLA7DLA7DLA7DLA7DLA7DLA7DLA7DLA7DLA7DLA7DL4wb2ouGO0lUKwPZF6SoF\nYPuidJUCsH1Rukq5Afj56d/bm+q+DavFm8rrJk36D8ueEGeUrlIuAH7/9qpfLkp1Zwy45CHLng5/\nlK5SjgHetOjBaW+uXHTXrVuFDzI93hdX/X5WZub2DY/c+NRab9XBTffOmfRvs79/uPLBqdP3CCfL\nvd7o/avFjW9+b+78/QprbI3SVco5wHec2D5/5ULv7puFD15e7p3+ye43va/9eMOdVdmLMrOKNy3w\n/nu2d/GvfrbEWzRDOCksit6/WtxYuNG7YY7CGlujdJVyDvBy7+/uXfkzr/eGl+9deeg7++Z4S3/4\no/lLN6zwPvPY8fe+s2ml96c/9y7fvHjGww/fLZwUFkXvXy1uFB7g8xvka+yO0lXKOcArI8A/8O6e\nEPno3kW/9C57zfvSkg3PeXN/5P1wchzvhdXefS8JJ4Wi968WN65a682fo7DG1ihdpRwGPGfGdwsi\nH/1i3DFvwcx5S25e85y3KnvKdwvjeEfuz5q7LYoXvT8KXH7P97M+Ulhja5SuUo4BFlu5ye5nMJxR\nukoB2L4oXaWcBeyuKF2lAGxflK5S3MBXLmp32cexKLmBFkPbjO3qDVg4rEv1q0bpKgVgymEAFgMw\nYQCmHAZgMQATBmDKYQAWAzBhAKYcBmAxABMGYMphABYDMGEAphwGYDEAEwZgymEAFgMwYamAg3u2\n5V8Ivp6XdwTAxoelMXD9fnZxi+/DxAkAGyiNgS/5WWBrfcEuT3sEt62tzd+kna+VY1FywTZD2wKG\ndvW1Wzisu1Plzsu2AjN2If/s+dOsplg43Llx48aKEEdhnkVJMWPbDO4azmF/imdkWKetwOGyQvF3\n2EBu7AQu0QpJwCl2pfEl+nRJiLHDVayxGMCphzkYeG9OXt77PbsKi1oBnHqYg4GTArBCAOYKwIQB\n2MwwAHMFYMIAbGYYgLkCMGEANjMMwFwBmDAAmxkGYK4ATBiAzQwDMFcAJgzAZoYBmCsAEwZgM8MA\nzBWACQOwmWEA5grAhAHYzDAAcwVgwgBsZhiAuQIwYQA2MwzAXAGYMACbGQZgrgBMGIDNDAMwVwAm\nDMBmhgGYKwATBmAzwwDMFYAJA7CZYa4CbtbO18KxKLlgwNC2NkO7+jqGcZgEnGJXT5fKQ9r9U3Zk\nAVghVwFzXMdwiZblqEs0x1cBwLIALAZgwgBsZhiAuQIwYQA2MwzAXAGYMACbGQZgrgBMGIDNDAMw\nVwAmDMBmhgGYKwATBmAzwwDMFYAJA7CZYQDmCsCEAdjMMABzBWDCAGxmGIC5AjBhADYzDMBcAZgw\nAJsZBmCuAEwYgM0MAzBXACYMwGaGAZgrABMGYDPDAMwVgAkDsJlhAOYKwISlAg7u2ZZ/Ibxv284u\nAKce5mDg+v3s4pZGDzt5AMCphzkY+JKfBbYeqmS+QgCnHuZgYMYu5J8tPcP63xYOPbm5uZ8PkhUO\n0T027TAJ2MiwK7YCh8sKL7Ny4RX8rvCB/2Lk1aldSxvHouSCAUPb2g3t6r8yjMMk4BS7erpUHrLJ\nVuDTJSHGGkpYNb4Hqwxz8CV6b05e3vvh0p3F3QBOPczBwEkBWCEAcwVgwgBsZhiAuQIwYQA2MwzA\nXAGYMACbGQZgrgBMGIDNDAMwVwAmDMBmhgGYKwATBmAzwwDMFYAJA7CZYQDmCsCEAdjMMABzBWDC\nAGxmGIC5AjBhADYzDMBcAZgwAJsZBmCuAEwYgM0MAzBXACYMwGaGAZgrABMGYDPDAMwVgAkDsJlh\nAOYKwIQB2MwwAHMFYML4gVu0a23jWJRcsMPQtoChXf2dwzhMAk6xq7db5SGbKV2l8Ao2M8xVr2CO\nrwKAZQFYDMCEAdjMMABzBWDCAGxmGIC5AjBhADYzDMBcAZgwAJsZBmCuAEwYgM0MAzBXACYMwGaG\nAZgrABMGYDPDAMwVgAkDsJlhAOYKwIQB2MwwAHMFYMIAbGYYgLkCMGEANjMMwFwBmDAAmxkGYK4A\nTBiAzQwDMFcAJgzAZoYBmCsAEwZgM8MAzBWACUsNXFHLgq/n5R0BcOphDgYOFb5ay3wfJk4AWCEH\nA4dDZbWsvmCXpx3AqYc5GJixg7Xs/GlWUywclns8nlM92vX2cSxKLmRsW7+hXYPGtinvkoBT7AoO\nqDxkwH5goYFc4Zd6r9fbGNCu4wrHouQGOw1t6zK0a6Db9LA/JZdiV1+vykP67Qc+XMUai2Mf4hIt\npQCcYle6X6J7dhUWtQJYPswlwLIALAVgnQGYMADrHwZgnQGYMADrHwZgnQGYMADrHwZgnQGYMADr\nHwZgnQGYMADrHwZgnQGYMADrH0YIfHFd61W3ANZdegOfGPGHq24BrLu0BW7MvHbS/949YmbDHddO\n+ES4bV6YMecsgHWXtsD//ReVm18SXrnblp/P+kfhdtW05n/JBLDu0hb4/JPX/c2vBVjvvBkTHxBu\n77l+8t99axDAektb4F//3Jf9V94RNU/N/frWB4Tb5Xe3vveKaV8AGxhGA3zitmu+/U7vbbfsv2ni\n3RO6b7vl4ryMeYeHG7hryK8ATjWM8E/Rw99Q4GDwxqBQIAPAqsMcCzxy5IiRkbIBrDrMscCMLUi9\nEMBSDgZWCcBSNMBJj0kBXDZzciQAqw5zMPCNa2vqhACsOszBwN/uSbkQwFIOBn5jY8r/NgZgKQcD\nzx49ZhK+B2sNswz46+tmz5r4bDixZtcbJoHrogFYdZh1wFmMhSZUDyOwSgCWshT4ys3+vqdm3vOH\nKwvvm3/cNHBWNACrDrMOOGPunIxfsPyfsKr76/eyop+aBj569GilZ24JgFWHWfoKbr8h9Nz9S5c+\nfGHF80+sHZ5LtH8KgFWHWQrMZl341RvsXMGLRaxgzfAAn1T+16QW7VrbOBYlF+wwtC1gaFd/p+lh\nCsApdvV2qzxkMx/wk57uf17wyLGKBx5bM+Wt4fgePH3UakVgjt/meAXLSru/Bx+NVBdWWghgKQcD\nMzZ4Oai8EMBSDgb2LR41etRiH4BVhzkY+NFlPuZb8jiAVYfRAP+fPArgjDbhl5YxAFYdRgP8DXkU\nwP9wQPjlAP6xQX2Yg4E/GLt03dKxuwGsOszBwKwhZ21Og+JCAEs5GLhvawXbkdMPYNVhDgZ+dno1\nq7zzGQCrDrMM+NRDwvmHTol35+wYDuBxkctzw1gAqw5zMPDEY8IvlRMBrDrMauCSZ56evTdnx1d3\nNYhHHY/Pn3fmTv+JkVd+/6R4Qgfwb8av2vz8+CIAqw6zHHhe+PgTOa9MbWTi0YZ17ODDL3s2Z5a+\n8r54Qgcwq1//7LovFBcCWMoy4OqFwvmF1SXr2B9/mHPrXZ8x8WhZGeu75djyxz7+ybxW8YQe4NQB\nWMoy4I4JzezShM6S9RHgovqpfeKR8AoufyiUeX9f5n1MPAHgoTkJmO259bap+6KKwh+yXlsnHnU8\ntmDeGfb0f7IFmwCcnKOAhyEA6x8GYJ0BGMBKARjACgEYwFcnfcGlMzTARAFYKwAnArAsAIsBmDAA\nawXgRACWBWAxABMGYK0AnAjAsgAsBmDCAKwVgBMBWFZaA1fUsvC+bTvjP/4dwNIZVwCHCl+tZY0e\ndvIAgF0JHA6V1bJDlcxXKHxQ7/V6GwPadVzhWJTcYKehbV2Gdg1069wgAUtnFIBT7O3rVXlgv63A\njB2sZaVnWP/bwmG5x+M51aNdbx/HouRCxrb1G9o1qHebBCydUQBOsTc4oPLAAfuBy4VX8LuxD3GJ\nls644hItAjeUsGp8D3YxcLh0Z3E3gF0KLAvA0hkA6wzAhAFYKwAnArAsAIsBmDAAawXgRACWBWAx\nABMGYK0AnAjAsgAsBmDCAKwVgBMBWBaAxQBMGIC1AnAiAMsCsBiACQOwVgBOBGBZABYDMGEA1grA\niQAsC8BiACYMwFoBOBGAZQFYDMCEAVgrCVjBFcBcAZgwAGsF4EQAlgVgMQATBmCtAJwIwLIcBezX\nrqWNY1FywXZD2wKGdvXzfCJD4wJOsbe3W+WBmyhdpfiBL2nX5OdYlNxAi6Ftxnb1BnRu4AJOsbdL\n7auWbq9gjqsZLtGyHHWJ5vhaAFgWgMUATBiAtQJwIgDLArAYgAkDsFYATgRgWQAWAzBhANYKwIkA\nLAvAYgAmDMBaATgRgGUBWAzAhAFYKwAnArAsAIsBmDAAawXgRACWBWAxABMGYK3UXAHMFYAJA7BW\nAE4EYFkAFgMwYQDWCsCJACwLwGIAJgzAWgE4EYBlAVgMwIQBWCs3Awdfz8s7EjsGsAuBfR8mjgHs\nQuD6gl2e9ghuW1ubv0k7XyvHouSCbYa2BQzt6mvXuYELOMXe7k6VB75sP/D506ymWLjduXHjxooQ\nR2GeRUkxY9t07pL+XVfnGC5gKdlTVHuOnfYDCw3kxg4cf4mWgHWO0Qd89d50v0QfrmKNxbFjALsQ\nuGdXYVFr7BjALgQeGoABDGAAKwZgwgCsFYATARjASgGYMABrBeBEAAawUgAmDMBaATgRgAGslMXA\naiwKUABOBGAAKwVgwgAM4GgABjCAAawYgAkDMICjARjAAAawYgAmDMAAjgZgAAMYwIoBmDAAAzia\nI4DV/lFfDVgNSh/w1Y8AYDEAEwZgAEcDMIABDGDFAEwYP/Bl7Zr8HIuSG2g1tE1hl4SosFwfcGKf\nUeDI3u5Oled/idJVih/Yp52/lWNRcsGAoW1tyackRIXl+oAT+4wCR/b2dKk8/zT4MUpDwyVaZ5G9\njrpEc3zJAQxgpQBMGIABHA3AAAYwgBUDMGHmgb+hEp/U8AGr+egDNl9kOIDFAEwYgAEcDcAABjCA\nFQMwYQAGcDQAAxjAAFYMwIQBGMDRAAxgAANYMQATBmAARzMEnPzVVTDX+/shnhqwQlYDa39WlK5S\nAKYKwAC2IgBTBWAAWxGAqQIwgK0IwFQBGMBWBGCqAAxgK1IFDu/btrMrdgxgnTkBuNHDTh6IHQNY\nZ04APlTJfIWxYwDrzAnApWdY/9vCrSc3N/fzwRTpBFZ7gFQjuOP6gpPTyualerJX7AcuF17B7wq3\n/ouRt4Zq16Lwg284MvhTdtoN7ern+XFBwzVM9afsNNkP3FDCqjW/Bw/J9vcHc9QbsHBYuv+Df7h0\nZ3F37BjABkp34KEB2EAAFgMwYQCmHAZgMQATBmDKYQAWAzBhAKYcBmAxABMGYMphABYDMGEAphwG\nYDEAEwZgymEAFgMwYQCmHOY24IuXDX0ZPvvS0LYmQ7uOnLZwWNUptXspXaW4gQnLb7RwWHG1hcP2\nV1g4TDkAUwZgMQATlg7AH1vz/+IXrfwrC4cd/8LCYcqlAzAiDMAuD8Auz15g6e2Ln79X0GbVsMEP\ntrxLPoyxitqhQ23LXuD42xcvvReu+8iqYWf2sC8+ph4WKny1duhQ27IXOP72xYoqFu7WXD1Mwxp+\nEzr+KfWwcKisduhQ27IXOP72xdKPtheR/10pPiy0JXdDB/Uwxg7WDh1qW/YCx9++WPa70NfvWDXs\ncHn43HbqYXHg+FDbshc4/vbFLz9hLflWDfv0GPPnUQ+LAw99g6Yt2fyn6MjbF5vzhdst75y3aljX\nji3vnKMeJgKLn1niDZq2hL8HuzwAuzwAuzzXAY8Msheub1Y4fWK6Dc/G/twIfL3CX6lHBlv2W/9k\n0iDnAAdXXD/+NXY0i0X+t+Nfnx47qz5+w3ZPGvO4j9XMXT9lwYgbF/zZ35YLR6zgpm9mCff99uZx\nK/qE04emx9fVzX7jhpsO2v35WJRzgD2T/3jy2q/iwKP+x7dmWjh20zj209Zl2axm7LLTkVfw2M7I\n0flrKvzLlrMvx1c2TN8avUTH1tWNfr17zUy7Px+LchDwhONhf38ceCpjA+POxm5yljLm+/PBmmv6\nWAxYOOo9x7peyGbrVzFWXREFjq2rGxNkNZPt/nwsyjnAg1unfWt9TwS4UgB+RDhz+2exm7XrhJvR\nl2smsjiwcBT8rzvuezCbPbtZ3C0Cx9bVTWKsDsDp1tlGdi7zraPCd1JP9BUc/Mv62E3OMuGVOSoo\nviqjwMJR8e2tbEc2e/l54XdEUewVHF0XwQVw2rU5q6lx2tYzo061zBWAR+T7X5wait18NaasbckT\n0cvuyEAM+K05Pc2Zi9gX4481ZL4pnBaAY+sAnJZ1Pjp63IqB8KrrppQIwA/+U8aMM/Eb9sGkjEeb\no8CLM7qiwO0Lxs36+K+L2Pa/H/PjfuF05E/R0XUATv+Ea++QG5Q6ALs8ALs8hwIj3gDs8gDs8gDs\n8gDs8gDs8gDs8gDs8v4fw3m7uI2cexUAAAAASUVORK5CYII=\n"
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "%%R\n",
    "\n",
    "library(ggplot2)\n",
    "\n",
    "stats=read.table(\"validation_stats.txt\", header=T)\n",
    "ggplot(stats, aes(x=supportfraction, fill=state)) + geom_histogram() + facet_wrap(~tag)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Remove low depth positions from plot (depth >= 30)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAeAAAAHgCAMAAABKCk6nAAAC3FBMVEUAAAAAv8QBAQEDAwMEBAQF\nBQUGBgYHBwcICAgKCgoLCwsMDAwNDQ0ODg4PDw8QEBASEhITExMUFBQVFRUXFxcYGBgZGRkaGhoc\nHBwdHR0eHh4fHx8gICAhISEiIiIjIyMkJCQlJSUmJiYnJycoKCgpKSkqKiorKyssLCwtLS0uLi4v\nLy8wMDAxMTEyMjIzMzM0NDQ1NTU2NjY3Nzc4ODg5OTk6Ojo7Ozs8PDw9PT0+Pj4/Pz9AQEBBQUFC\nQkJDQ0NERERFRUVGRkZHR0dISEhJSUlKSkpLS0tMTExNTU1OTk5PT09QUFBRUVFSUlJTU1NUVFRV\nVVVWVlZXV1dYWFhZWVlaWlpbW1tcXFxdXV1eXl5fX19gYGBhYWFiYmJkZGRlZWVmZmZnZ2doaGhp\naWlqampsbGxtbW1ubm5vb29wcHBzc3N0dHR1dXV2dnZ3d3d4eHh5eXl6enp7e3t8fHx9fX1+fn5/\nf3+AgICBgYGCgoKDg4OFhYWGhoaHh4eIiIiJiYmKioqLi4uMjIyNjY2Ojo6Pj4+QkJCRkZGTk5OU\nlJSVlZWWlpaXl5eYmJiZmZmampqbm5ucnJydnZ2enp6fn5+goKChoaGioqKjo6OkpKSlpaWmpqan\np6eoqKipqamqqqqrq6usrKytra2urq6vr6+wsLCxsbGysrKzs7O0tLS1tbW2tra4uLi5ubm6urq7\nu7u8vLy9vb2+vr6/v7/AwMDBwcHCwsLDw8PExMTFxcXGxsbHx8fIyMjJycnKysrLy8vMzMzOzs7P\nz8/Q0NDR0dHT09PU1NTV1dXW1tbX19fY2NjZ2dna2trb29vc3Nzd3d3e3t7f39/h4eHi4uLj4+Pk\n5OTl5eXm5ubn5+fo6Ojp6enq6urr6+vs7Ozt7e3u7u7v7+/w8PDx8fHy8vLz8/P09PT19fX29vb3\n9/f4dm34+Pj5+fn6pJ76+vr7+/v8/Pz9/f3+/v7///+sM5rCAAARUUlEQVR4nO3ci59U5XnAcVKo\ntgoIbRNba8GCSQSM6AKiIApaL8VLKtRUhdgiVIo1YmIqXooQiRCTgohcDaaLeEHECxARXW6jiMsS\n427ADJe9wa4LZre7O/P+Az07c3bcnZ0ze96Hfc7snPx+n4+cM++8L++c/XpmuTj2MBTqeuT6BZBu\nAIc8gEMewCEP4JAHcMgDOOQBHPIADnkAhzyAQx7AIQ/gkOcbOEJdnaZrKoBzl6ZrKoBzl6ZrKoBz\nl6ZrqjAA3z/82695PTd3VuJQdPagQf8R2AvymaZrqhAA//KS3T+d6PWkC7zuxsBejv80XVPlDfD8\niTcMe3rGxCsuWuY8KCiMPDTzvcsLClbNvfm8O2dHdm+Zf/WYQf82+jvbi24YOvwVZ3BrJJJ8flZi\n4dPfHjv+jQxzcpqma6r8Ab50z6rxM66PrL/AefDItMjwN9c/HXn8+3Mv2z15YsGItfMnRP59cmTS\nz350d2T1SGfQmZR8flZi4fJ5kbljMszJaZquqfIHeFpkw9UzfhSJnPvI1TPe/ebGMZFN3/3e+Clz\np0fuuXXXim/OnxH54Y8j0xZMGnnTTVc6g86k5POzEgudn+CDc9Pn5DpN11T5AzyjBfgfIusHtDy6\neuJPI1Mfjzx899z7Iou+F3l5cCveA7MiGx92Bp2Sz89KLJw5O7JkTIY5OU3TNVWeAY8Z+a2lLY9+\n0m9nZOmocXdf8OB9kd2Th3xreSve+9eOGLsyiZd8Pgm89arvjHg1w5ycpumaKm+AE82Yn+tX0JVp\nuqYCOHdpuqbKL+BwpemaCuDcpemayjdwbdRPlb5mday8WbjwSLlwYX2VcKH0Eqvq00c0XVMBbBvA\nkgDWCmDbAJYEsFYA2wawJIC1Atg2gCUBrBXAtgEsCWCtALYNYEkAawWwbQBLAlgrgG0DWBLAWgFs\nG8CS8h34D615rwNYFMCdBLBtAEsCWCuAbQNYEsBaAWwbwJIA1gpg2wCWBLBW/oHL/VTta1aGdTHh\nwooq4cLGGuHCjJeYAvZeV9OYNnBU0zWVf+DDfqr0NatjFc3ChUfLhQsbqoULM15iCth7XXVD2kB3\nu4Pl718+4i1aK4BtA1gSwFoBbBvAkgDWCmDbAJYEsFYA2wawJIC1Ati2tpf4hw55rwNYFMCdBLBt\nAEsCWCuAbQNYEsBaAWwbwJIA1gpg2wCWBLBWANsGsCSAtQLYNoAlAawVwLYBLAlgrQC2DWBJAGsF\nsG0ASwJYK4BtA1gSwFoBbBvAkgDWCmDbAJYEsFYA2wawJIC1Atg2gCUBrBXAtgEsCWCtALYNYEkA\nawWwbQBLAlgrgG0DWBLAWgFsW2iAtxWbpicXL34f4PaFBDi2/LFiU/7yVwMAu4UEOB7bXGwOLH2x\n8ITz4OPt27d/WuOnk75mdawuJlxYWydc2CR9qW3XdQTOsq4pbaAyp8DGbCk2hz4x+9Y6p+9v2LBh\n30k/felrVoZ1ceFC8Y7N9V2wYUdg73X1zWkD1bkHdmpc5D7kLdotJG/RCeDtu03ZWoDbFybgUy8u\nX10FcPtCA5wWwG4ASwJYK4BtA1gSwFoBbBvAkgDWCmDbAJYEsFYA2wawJIC1Atg2gCUBrBXAtgEs\nCWCtALYNYEkAawWwbQBLAlgrgG0DWBLAWgFsG8CSANYKYNsAlgSwVgDbBrAkgLUC2DaAJQGsFcC2\nASwJYK0Atg1gSQBrBbBtAEsCWCuAbQNYEsBaAWwbwJIA1gpg2wCWBLBWANsGsCSAtQLYNoAlAawV\nwLYBLAlgrQC2DWBJAGvlH7jST8d9zcqwLiZcWFUtXNjo74I61vYSOwJ7r6ttTBs4pumaijvYttDe\nwdZXbxPAWgFsG8CSANYKYNsAlgSwVgDbBrAkgLUC2DaAJQGsFcC2ZQf2pgZYFMCdBLBtAEsCWCuA\nbQNYEsBaAWwbwJIA1gpg2wCWBLBWANsGsCSAtQLYNoAlAawVwLYBLAlgrQC2DWBJAGsFsG0ASwJY\nK4BtA1gSwFoBbBvAkgDWCmDbAJYEsFYA2wawJIC1Atg2gCUBrBXAtgEsCWCtALYNYEkAawWwbQBL\nAlgrgG0DWBLAWnkDbys28Y0rX6gDuH0hAY4tf6zYlBWaD98BuH0hAY7HNhebd4tM+XKA26cKHJ1T\n1e6oB2zMlmKzab9p+Llz+sK8efO2xfwU9zUrQ0a6ULpjXLywzXk24E43/CLDF31Pj9+2OyoDb3Xu\n4Oed09rq6uqKo3467mtWxyqbhQvLq4QLG6Qvte26bMAd1jWkDRxp//UuKzhz0FtX9hhVeumZA950\njseu7z3mU2Xg0nVmL9+D09J6i/7vPy9a8LBz566cdmjEPzrHmcOO/UuBMnB80wtrTwLcPi3gQ3ec\n/de/cGAj40YOvM45XnXO4L/9erMicFoAu2kB/+LH5ZP/MtJj351jP7/oOuc47cqqFY+eti/A1mkB\n77n4jG88++XFF75x/sArB5y8+MLouN7jtnc1cF2bHwHOXB7/Prip6bwmp+O9Ac5SHgP37NmjZ0uT\nAc5SHgMbM8F7IsBueQ2cJYDdtIA7/AQawJtHDW4J4CzlNfB5s/eVOAGcpbwG/sYpz4kAu+U18FPz\nPP9sDGC3vAYefVafQXwPzl6AwJ+fPfrygffGv5rz4lOnCVySDOAsBQk8wpjYgL1dCJwlgN0CBq69\noKL+zlFX/bb2+mvG7zpt4BHJAM5SkMC9x47p/ROz5Adm97UHXjOrf3jawDt27CgqHLsO4CwFfAef\nODd237VTptz0++n33z67a96iK4YAnKWAgc3lv//ZU+bg0odWm6UPdg3wh/xtUraCBr6j8OQ/T7h5\n57brbn1wyDNd8T14eK9ZAGcpr38fvKOlknimiQC75TWwMc1HmjJPBNgtr4HLJ/U6q9ekcoCzlNfA\nt0wtN+V33wZwlrSA/y89DeDe1c4PlX0AzpIW8NfS0wD++5bPMbzDXzZkK6+BX+o7Zc6UvusBzlJe\nA5vShbMXlmacCLBbXgPXL9tm1ixsADhLeQ187/C9puiyewDOUoDAH93ojN/4UeLphWu6Arhfy9tz\naV+As5TXwAN3Oj8UDQQ4S8EDr7vnrtGvLVzz2RWlibOa28aP239ZxZ6ete/dkRiwAP5V/5kL7u+/\nGuAs5QB4XHzX7QsfHVpmEmdz55gtNz1SuKBg06O/TAxYAJsDT9w75+OMEwF2CxB47/XO+PV7180x\nv/vuwouu+LVJnE3dbOov3Dnt1td/MK4qMWAD7B3AbgEC1ww4Zg4P+GLdEy3Aqw8MrU+cOXfw1htj\nBdfWF1xjEgMAZyofgM0rF108dGNS0flF1uNzEmc1t04Yt9/c9Z9mwnyAvcsL4C4IYNsAlgQwwJkD\nGODMAQxw5vhfOEgCWCuAbQNYEsBaAWwbwJIA1gpg2wCWBLBW/oHL/VTta1aGdTHhwooq4cLGGuHC\ntpeYDTh9XU1j2sBRTddU/oEP+6nS16yOVTQLFx4tFy5sqBYubHuJ2YDT11U3pI9ouqbiLdq20L5F\nW1+9TQBrBbBtAEsCWCuAbQNYEsBaAWwbwJIA1gpg2wCWBLBWANsGsCSAtQLYNoAlAawVwLYBLAlg\nrQC2DWBJAGsFsG0ASwJYK4BtA1gSwFoBbBvAkgDWCmDbAJYEsFYA2wawJIC1Atg2gCUBrBXAvstA\nB7DfANYKYN8BHAUYYI8A1gpg3wEcBRhgjwDWCmDfARwFGGCPANYKYN8BHAUYYI8A1gpg3wEcBRhg\njwDWCmDfhRC46cnFi993zwEOIXD5y1+dAxxC4ANLXyw80XKMRCJlx/1U52tWx2pjwoU1tcKFTdYv\nNUWXYShD6cvrmtIGKnIPfOgTs2+tc9xaWFj40Sk/Nfia1bH6uHDhl/XChTHrl5pNM0PpyxtiaQPH\ncw/s1LjIPeEt2hI4fXl3fIvevtuUrXXPAQ4h8KkXl6+ucs8BDiFw2wAGOAowwB4BrBXAvgM4CjDA\nHgGsFcC+AzgKMMAeAawVwL4DOAowwB4BrBXAvgM4CjDAHgGsFcC+AzgKMMAe5QQ4m0rWIZE0wKIA\n7iSAAU4GMMBRgAH2CGCtAAY4GcAARwEG2COAtQIY4GQAAxwFGGCPANYKYICTAQxwFGCAPQJYK4AB\nTgYwwFGAAfaouwD7MwfYOoC1AhjgZAADHAUYYI8A1gpggJMBDHAUYIA9Algr/8CVfjrua1aGdTHh\nwqpq4cLG2kp/PqkVUmB3eW1j2is4pumaijvY5w0Y/jvY15cNYIAzBrBWAAOcDGCAowAD7BHAWgEM\ncDKAAY4CHEbgr7XW8qC7AGfzST70DXzaufsDLArgTgIY4GQAAxwFGGCPANYKYICTAQxwFGCAPQJY\nK4ABTgYwwFGAAfYIYK0ABjgZwABHAQbYI4C1AhjgZAADHAX4jw2445e5/fRUXQTs/eUFuLMAlubu\nD7AogDsJYPXc/QEWBXAnAayeuz/AogDuJIDVc/cHWBTAnQSweu7+AIsCuJMAVs/dH2BRAHcSwOq5\n+wMsCuBOAlg9d3+ARQHcSVmB4xtXvlDnngMszd2/OwKXFZoP33HPAZbm7t8dgd8tMuXL3XOApbn7\nd0fgTftNw8+dY+GiRYs+aPYoJZbhudRFdjY9Zrx+eqsyfHm9Xks8lmm2Rq2XGE97sbW5B97q3MHP\nO8eKxK3ipxO+ZnWsOiZcWFktXNhYI1wovcSaxrSBo7kHLl1n9nb6Pbhd3eU/m+085y1alvQSu+Nb\ndHzTC2tPuucAu4UJuG0AuwEsCWCtALYNYEkAawWwbQBLAlgrgG0DWBLAWgFsG8CSANYKYNsAlgSw\nVgDbBrAkgLUC2LY/buCjwqsvfUe48PAR4cL3SoQLpZdY8l76iKZrKt/Auh19OugdV5QEvOH+lQFv\nmAzgoAI42AAOtOqXgt7xrUMBb3jwrYA3TNZNgEkrgEMewCEv98CpjzB+sGJpdZAbNr/03POBbOi0\nrbjtzkGWe+DWjzAeXhEveTXIDfe/Yj5+PYgNTWz5Y8Vtdw6y3AO3foRx224TP9np7C7csPRXsV1v\nB7Ghicc2F7fdOchyD9z6EcZNr65afSTIDWPPLZpbE8SGTluK2+4cZLkHbv0I4+YNsc+fDXLD7Vvj\nB1cFsaFpBW7dOchyD9z6EcbfvGkqlwS54ds7TcXiIDY0rcBtP60ZVLkHTnyE8dgS5/jcs4H86VLr\nhnVrnnv2YBAbmgRw4hK/+rRmUOUemFQDOOQBHPJCB9yzyTxwzrEMw3uG5+DV5L4wAp+T4bfTPZsq\n3wj+xXSD8ge4afo5/R83O0aYln/W/OtdfS8/0How6wf1ua3c7Bv7xJAJPc6b8Cd/s9U5M0vP/7MR\nznP/e0G/6fXO8LvDW+eVjH7q3PO35Pp6Aip/gAsH/+7DMz9rBe71P+UPDou7h7K+b1dNnWz29Z36\nScsd3PeLlrNDZ2yrmDrN/KZ/UenwZcm3aHdeyVlPnnxwVK6vJ6DyCHjArnhFQyvwUGMa+33qHhZO\nMab8T5v3nVFvXGDn7MuDpu6ByeaJmcbs3ZYEdueV9Gky+wbn+noCKn+Am5cN+/oTp1qAixzgm52R\nS37tHmbPcQ5nHdk30LQCO2dN/3XpNTdMNvcuSKxOALvzSgYZUwJwd+vTMnOw4JkdznfSwuQd3PQX\nB9zDwqnOndmrKXFXJoGds7WXVJk1k80j9zv/Rqx27+DkvBZcgLtdC0YcLRu2bH+vjyrHOsA9llQ8\nNDTmHj7rs7n67tuTb7s9j7vAz4w5daxgovm4/87SgqedYQfYnQdwt+yLW87qN70xPvPsIesc4Bv+\nqffI/a0H89Kg3rccSwJP6l2XBD4xod/lr//VarPq7/p8v8EZbvlVdHIewN0/5723zYG8AzjkARzy\n8hSY/AZwyAM45AEc8gAOeQCHPIBDHsAh7/8BS7MjLhijFP0AAAAASUVORK5CYII=\n"
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "%%R\n",
    "\n",
    "library(ggplot2)\n",
    "\n",
    "stats=read.table(\"validation_stats.txt\", header=T)\n",
    "ggplot(subset(stats, supportingreads>50), aes(x=supportfraction, fill=state)) + geom_histogram(breaks=c()) + facet_wrap(~tag)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "## Filter the files as per the study (depth > 30, support fraction > 0.70%, primer sequences masked)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "+ ref_prefix=EM_079517\n",
      "+ sample=076534_180Genomes_11rx\n",
      "+ poretools_dir=076534_180Genomes_11rx\n",
      "+ sample_tag=076534_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ vcftagprimersites.py all np_EM_079517_076534_180Genomes_11rx_hq.vcf\n",
      "+ vcffilter.py 076534_180Genomes_11rx_hq_EM_079517_np_primer.tagged.vcf\n",
      "Filtering Record(CHROM=EM_079517, POS=16622, REF=G, ALT=[A])\n",
      "Filtering Record(CHROM=EM_079517, POS=16626, REF=T, ALT=[A])\n",
      "Filtering Record(CHROM=EM_079517, POS=16629, REF=A, ALT=[G])\n",
      "Filtering Record(CHROM=EM_079517, POS=16631, REF=A, ALT=[T])\n",
      "Output 14 records\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=076534_180Genomes_11rx\n",
      "+ poretools_dir=076534_180Genomes_11rx\n",
      "+ sample_tag=076534_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ vcftagprimersites.py all np_EM_079517_mut30_2_076534_180Genomes_11rx_hq.vcf\n",
      "+ vcffilter.py 076534_180Genomes_11rx_hq_EM_079517_mut30_2_np_primer.tagged.vcf\n",
      "Filtering Record(CHROM=EM_079517, POS=16622, REF=G, ALT=[A])\n",
      "Filtering Record(CHROM=EM_079517, POS=16626, REF=T, ALT=[A])\n",
      "Filtering Record(CHROM=EM_079517, POS=16629, REF=A, ALT=[G])\n",
      "Filtering Record(CHROM=EM_079517, POS=16631, REF=A, ALT=[T])\n",
      "Output 38 records\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=076533_180Genomes_11rx\n",
      "+ poretools_dir=076533_180Genomes_11rx\n",
      "+ sample_tag=076533_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ vcftagprimersites.py all np_EM_079517_076533_180Genomes_11rx_hq.vcf\n",
      "+ vcffilter.py 076533_180Genomes_11rx_hq_EM_079517_np_primer.tagged.vcf\n",
      "Filtering Record(CHROM=EM_079517, POS=27, REF=A, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=29, REF=A, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=30, REF=A, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=31, REF=A, ALT=[C])\n",
      "Filtering Record(CHROM=EM_079517, POS=32, REF=A, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=5755, REF=C, ALT=[G])\n",
      "Output 19 records\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=076533_180Genomes_11rx\n",
      "+ poretools_dir=076533_180Genomes_11rx\n",
      "+ sample_tag=076533_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ vcftagprimersites.py all np_EM_079517_mut30_2_076533_180Genomes_11rx_hq.vcf\n",
      "+ vcffilter.py 076533_180Genomes_11rx_hq_EM_079517_mut30_2_np_primer.tagged.vcf\n",
      "Filtering Record(CHROM=EM_079517, POS=27, REF=A, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=29, REF=A, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=30, REF=A, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=31, REF=A, ALT=[C])\n",
      "Filtering Record(CHROM=EM_079517, POS=32, REF=A, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=5755, REF=C, ALT=[G])\n",
      "Output 49 records\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=076383_180Genomes_11rx\n",
      "+ poretools_dir=076383_180Genomes_11rx\n",
      "+ sample_tag=076383_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ vcftagprimersites.py all np_EM_079517_076383_180Genomes_11rx_hq.vcf\n",
      "+ vcffilter.py 076383_180Genomes_11rx_hq_EM_079517_np_primer.tagged.vcf\n",
      "Filtering Record(CHROM=EM_079517, POS=40, REF=G, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=5745, REF=A, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=18581, REF=C, ALT=[G])\n",
      "Output 16 records\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=076383_180Genomes_11rx\n",
      "+ poretools_dir=076383_180Genomes_11rx\n",
      "+ sample_tag=076383_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ vcftagprimersites.py all np_EM_079517_mut30_2_076383_180Genomes_11rx_hq.vcf\n",
      "+ vcffilter.py 076383_180Genomes_11rx_hq_EM_079517_mut30_2_np_primer.tagged.vcf\n",
      "Filtering Record(CHROM=EM_079517, POS=40, REF=G, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=5745, REF=A, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=18581, REF=C, ALT=[G])\n",
      "Output 46 records\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=078416_180Genomes_11rx\n",
      "+ poretools_dir=078416_180Genomes_11rx\n",
      "+ sample_tag=078416_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ vcftagprimersites.py all np_EM_079517_078416_180Genomes_11rx_hq.vcf\n",
      "+ vcffilter.py 078416_180Genomes_11rx_hq_EM_079517_np_primer.tagged.vcf\n",
      "Filtering Record(CHROM=EM_079517, POS=2282, REF=G, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=5748, REF=C, ALT=[G])\n",
      "Filtering Record(CHROM=EM_079517, POS=5755, REF=C, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=18479, REF=T, ALT=[C])\n",
      "Filtering Record(CHROM=EM_079517, POS=18481, REF=A, ALT=[C])\n",
      "Filtering Record(CHROM=EM_079517, POS=18588, REF=A, ALT=[G])\n",
      "Filtering Record(CHROM=EM_079517, POS=18591, REF=C, ALT=[G])\n",
      "Filtering Record(CHROM=EM_079517, POS=18597, REF=T, ALT=[A])\n",
      "Output 17 records\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=078416_180Genomes_11rx\n",
      "+ poretools_dir=078416_180Genomes_11rx\n",
      "+ sample_tag=078416_180Genomes_11rx_hq\n",
      "+ second_batch=na\n",
      "+ vcftagprimersites.py all np_EM_079517_mut30_2_078416_180Genomes_11rx_hq.vcf\n",
      "+ vcffilter.py 078416_180Genomes_11rx_hq_EM_079517_mut30_2_np_primer.tagged.vcf\n",
      "Filtering Record(CHROM=EM_079517, POS=5748, REF=C, ALT=[G])\n",
      "Filtering Record(CHROM=EM_079517, POS=5755, REF=C, ALT=[T])\n",
      "Filtering Record(CHROM=EM_079517, POS=18479, REF=T, ALT=[C])\n",
      "Filtering Record(CHROM=EM_079517, POS=18481, REF=A, ALT=[C])\n",
      "Filtering Record(CHROM=EM_079517, POS=18588, REF=A, ALT=[G])\n",
      "Filtering Record(CHROM=EM_079517, POS=18591, REF=C, ALT=[G])\n",
      "Filtering Record(CHROM=EM_079517, POS=18597, REF=T, ALT=[A])\n",
      "Output 48 records\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=076769_180Genomes_19Rx\n",
      "+ poretools_dir=076769_180Genomes_19Rx\n",
      "+ sample_tag=076769_180Genomes_19Rx_hq\n",
      "+ second_batch=na\n",
      "+ vcftagprimersites.py all np_EM_079517_076769_180Genomes_19Rx_hq.vcf\n",
      "+ vcffilter.py 076769_180Genomes_19Rx_hq_EM_079517_np_primer.tagged.vcf\n",
      "Filtering Record(CHROM=EM_079517, POS=3447, REF=T, ALT=[C])\n",
      "Filtering Record(CHROM=EM_079517, POS=18588, REF=A, ALT=[G])\n",
      "Filtering Record(CHROM=EM_079517, POS=18597, REF=T, ALT=[A])\n",
      "Output 18 records\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=076769_180Genomes_19Rx\n",
      "+ poretools_dir=076769_180Genomes_19Rx\n",
      "+ sample_tag=076769_180Genomes_19Rx_hq\n",
      "+ second_batch=na\n",
      "+ vcftagprimersites.py all np_EM_079517_mut30_2_076769_180Genomes_19Rx_hq.vcf\n",
      "+ vcffilter.py 076769_180Genomes_19Rx_hq_EM_079517_mut30_2_np_primer.tagged.vcf\n",
      "Filtering Record(CHROM=EM_079517, POS=3447, REF=T, ALT=[C])\n",
      "Filtering Record(CHROM=EM_079517, POS=18588, REF=A, ALT=[G])\n",
      "Filtering Record(CHROM=EM_079517, POS=18597, REF=T, ALT=[A])\n",
      "Output 49 records\n"
     ]
    }
   ],
   "source": [
    "!makecommands.py ../metadata/metadata.db 180_Genomes consensus | parallel -j16"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "fh = open(\"validation_files_filtered.txt\", \"w\")\n",
    "fh.write(\"\"\"076534_180Genomes_11rx\tnp-6mer-5meralign\t../refs/EM_079517_EM_076534.mutations.txt\t076534_180Genomes_11rx_hq_EM_079517_np_primer.filtered075_30.vcf\n",
    "076533_180Genomes_11rx\tnp-6mer-5meralign\t../refs/EM_079517_EM_076533.mutations.txt\t076533_180Genomes_11rx_hq_EM_079517_np_primer.filtered075_30.vcf\n",
    "076383_180Genomes_11rx\tnp-6mer-5meralign\t../refs/EM_079517_EM_076383.mutations.txt\t076383_180Genomes_11rx_hq_EM_079517_np_primer.filtered075_30.vcf\n",
    "078416_180Genomes_11rx\tnp-6mer-5meralign\t../refs/EM_079517_EM_078416.mutations.txt\t078416_180Genomes_11rx_hq_EM_079517_np_primer.filtered075_30.vcf\n",
    "076769_180Genomes_19Rx\tnp-6mer-5meralign\t../refs/EM_079517_EM_076769.mutations.txt\t076769_180Genomes_19Rx_hq_EM_079517_np_primer.filtered075_30.vcf\"\"\")\n",
    "fh.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "sample                  tag                total_calls  mutations  TP    FP   FN   TPR\r\n",
      "076534_180Genomes_11rx  np-6mer-5meralign  14           17         14.0  0.0  3.0  0.823529411765\r\n",
      "076533_180Genomes_11rx  np-6mer-5meralign  19           19         19.0  0.0  0.0  1.0\r\n",
      "076383_180Genomes_11rx  np-6mer-5meralign  16           17         16.0  0.0  1.0  0.941176470588\r\n",
      "078416_180Genomes_11rx  np-6mer-5meralign  17           18         17.0  0.0  1.0  0.944444444444\r\n",
      "076769_180Genomes_19Rx  np-6mer-5meralign  18           19         18.0  0.0  1.0  0.947368421053\r\n"
     ]
    }
   ],
   "source": [
    "!../scripts/intersection_vcf_stats.py validation_files_filtered.txt | cut -f 1,2,5,6,7,8,9,10 | column -t "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Get the 180 genome set"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "from Bio import Entrez\n",
    "Entrez.email = \"n.j.loman@bham.ac.uk\"     # Always tell NCBI who you are\n",
    "handle = Entrez.esearch(db=\"nucleotide\", term=\"carroll and hiscox and ebola\", retmax=200)\n",
    "record = Entrez.read(handle)\n",
    "handle = Entrez.efetch(db=\"nucleotide\", id=record[\"IdList\"], rettype=\"fasta\", retmode=\"text\")\n",
    "fh = open(\"180_Genomes_references.fasta\", \"w\")\n",
    "fh.write(handle.read())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!echo \"\\n\"  >> 180_Genomes_references.fasta \n",
    "!cat 180_Genomes_references.fasta  180_Genomes.fasta > validate_tree.fasta\n",
    "!sed --in-place '/^$/d' validate_tree.fasta\n",
    "!muscle3.8.31_i86linux64 -in validate_tree.fasta > validate_tree_aligned.fasta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!sed --in-place 's/>.*H.sapiens-wt\\//>/' validate_tree_aligned.fasta\n",
    "!sed --in-place 's/>EM_079517_/>MinION_/' validate_tree_aligned.fasta\n",
    "!sed --in-place 's/,//' validate_tree_aligned.fasta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!grep \">\" validate_tree_aligned.fasta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!rm *validate_tree_aligned.raxml\n",
    "!raxmlHPC-PTHREADS-SSE3 -T 32 -f a -x 12345 -p 12345 -# 100 -m GTRGAMMA -s validate_tree_aligned.fasta -n validate_tree_aligned.raxml"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat /mnt/ebola/ebov/analysis/RAxML_bipartitions.validate_tree_aligned.raxml\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!pip install ete2\n",
    "\n",
    "from ete2 import Tree\n",
    "t = Tree()\n",
    "t.populate(20)\n",
    "t.render(file_name=\"test.png\", w=500)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
