{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Run alignment and variant calling\n",
    "\n",
    "The parameters we use are:\n",
    "    * nanopolish: 5-mer alignment, 6-mer variant calling, reverse HMM fix"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "ROOTDIR=\"/mnt/ebola/ebov\"\n",
    "import os\n",
    "os.environ['PATH'] = os.getenv('PATH') + ':' + ROOTDIR + '/scripts' + ':' + ROOTDIR + '/bin/standard-RAxML' + ':' + '/bin/nanopore-scripts'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/ebola/ebov\n"
     ]
    }
   ],
   "source": [
    "%cd $ROOTDIR"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "+ ref_prefix=EM_079517\n",
      "+ sample=FCH_2015_209B\n",
      "+ poretools_dir=FCH_2015_209B\n",
      "+ sample_tag=FCH_2015_209B_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH_2015_209B_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_FCH_2015_209B_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 251 sequences (753447 bp)...\n",
      "[M::mem_process_seqs] Processed 251 reads in 5.692 CPU sec, 5.692 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH_2015_209B_hq.fasta\n",
      "[main] Real time: 6.361 sec; CPU: 6.152 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH_2015_209B_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH_2015_209B_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FCH_2015_209B\n",
      "+ poretools_dir=FCH_2015_209B\n",
      "+ sample_tag=FCH_2015_209B_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH_2015_209B_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_FCH_2015_209B_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 251 sequences (753447 bp)...\n",
      "[M::mem_process_seqs] Processed 251 reads in 5.803 CPU sec, 5.749 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH_2015_209B_hq.fasta\n",
      "[main] Real time: 6.386 sec; CPU: 6.236 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH_2015_209B_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=015986\n",
      "+ poretools_dir=015986\n",
      "+ sample_tag=015986_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015986_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_015986_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2809 sequences (4656253 bp)...\n",
      "[M::mem_process_seqs] Processed 2809 reads in 17.937 CPU sec, 17.348 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015986_hq.fasta\n",
      "[main] Real time: 20.537 sec; CPU: 20.135 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015986_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015986_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=015982\n",
      "+ poretools_dir=015982\n",
      "+ sample_tag=015982_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015982_hq.fasta\n",
      "+ samtools sort - EM_079517_015982_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2319 sequences (3873101 bp)...\n",
      "[M::mem_process_seqs] Processed 2319 reads in 17.213 CPU sec, 16.794 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015982_hq.fasta\n",
      "[main] Real time: 19.452 sec; CPU: 19.363 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015982_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015982_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=015982\n",
      "+ poretools_dir=015982\n",
      "+ sample_tag=015982_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015982_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_015982_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2319 sequences (3873101 bp)...\n",
      "[M::mem_process_seqs] Processed 2319 reads in 16.350 CPU sec, 15.952 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015982_hq.fasta\n",
      "[main] Real time: 18.635 sec; CPU: 18.360 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015982_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015982_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=015986\n",
      "+ poretools_dir=015986\n",
      "+ sample_tag=015986_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015986_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_015986_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2809 sequences (4656253 bp)...\n",
      "[M::mem_process_seqs] Processed 2809 reads in 17.298 CPU sec, 16.764 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015986_hq.fasta\n",
      "[main] Real time: 19.821 sec; CPU: 19.420 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015986_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015986_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=013962\n",
      "+ poretools_dir=013962\n",
      "+ sample_tag=013962_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 013962_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_013962_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2227 sequences (3716642 bp)...\n",
      "[M::mem_process_seqs] Processed 2227 reads in 17.631 CPU sec, 17.438 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 013962_hq.fasta\n",
      "[main] Real time: 20.129 sec; CPU: 19.622 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_013962_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_013962_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FCH_2015_216C\n",
      "+ poretools_dir=FCH_2015_216C\n",
      "+ sample_tag=FCH_2015_216C_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH_2015_216C_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_FCH_2015_216C_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 1928 sequences (3531962 bp)...\n",
      "[M::mem_process_seqs] Processed 1928 reads in 16.992 CPU sec, 16.992 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH_2015_216C_hq.fasta\n",
      "[main] Real time: 19.551 sec; CPU: 18.924 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH_2015_216C_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH_2015_216C_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=015972\n",
      "+ poretools_dir=015972\n",
      "+ sample_tag=015972_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015972_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_015972_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2414 sequences (4054921 bp)...\n",
      "[M::mem_process_seqs] Processed 2414 reads in 20.683 CPU sec, 20.492 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015972_hq.fasta\n",
      "[main] Real time: 22.824 sec; CPU: 22.405 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015972_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015972_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=015972\n",
      "+ poretools_dir=015972\n",
      "+ sample_tag=015972_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015972_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_015972_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2414 sequences (4054921 bp)...\n",
      "[M::mem_process_seqs] Processed 2414 reads in 22.196 CPU sec, 21.667 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015972_hq.fasta\n",
      "[main] Real time: 24.293 sec; CPU: 24.092 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015972_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015972_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=013962\n",
      "+ poretools_dir=013962\n",
      "+ sample_tag=013962_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 013962_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_013962_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2227 sequences (3716642 bp)...\n",
      "[M::mem_process_seqs] Processed 2227 reads in 20.672 CPU sec, 20.007 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 013962_hq.fasta\n",
      "[main] Real time: 22.849 sec; CPU: 22.870 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_013962_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_013962_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FCH_2015_216C\n",
      "+ poretools_dir=FCH_2015_216C\n",
      "+ sample_tag=FCH_2015_216C_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH_2015_216C_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FCH_2015_216C_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 1928 sequences (3531962 bp)...\n",
      "[M::mem_process_seqs] Processed 1928 reads in 23.157 CPU sec, 23.032 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH_2015_216C_hq.fasta\n",
      "[main] Real time: 25.446 sec; CPU: 24.998 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH_2015_216C_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH_2015_216C_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=IPD_2015_6505\n",
      "+ poretools_dir=IPD_2015_6505\n",
      "+ sample_tag=IPD_2015_6505_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta IPD_2015_6505_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_IPD_2015_6505_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3189 sequences (5314143 bp)...\n",
      "[M::mem_process_seqs] Processed 3189 reads in 20.112 CPU sec, 19.758 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta IPD_2015_6505_hq.fasta\n",
      "[main] Real time: 22.902 sec; CPU: 22.332 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_IPD_2015_6505_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=004674\n",
      "+ poretools_dir=004674\n",
      "+ sample_tag=004674_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 004674_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_004674_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3280 sequences (5227960 bp)...\n",
      "[M::mem_process_seqs] Processed 3280 reads in 27.388 CPU sec, 26.861 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 004674_hq.fasta\n",
      "[main] Real time: 30.529 sec; CPU: 29.780 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_004674_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_004674_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=013857\n",
      "+ poretools_dir=013857\n",
      "+ sample_tag=013857_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 013857_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_013857_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3632 sequences (6154585 bp)...\n",
      "[M::mem_process_seqs] Processed 3632 reads in 24.572 CPU sec, 23.784 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 013857_hq.fasta\n",
      "[main] Real time: 28.911 sec; CPU: 27.604 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_013857_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_013857_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=015980\n",
      "+ poretools_dir=015980\n",
      "+ sample_tag=015980_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015980_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_015980_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3045 sequences (5106288 bp)...\n",
      "[M::mem_process_seqs] Processed 3045 reads in 25.228 CPU sec, 24.960 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015980_hq.fasta\n",
      "[main] Real time: 28.725 sec; CPU: 27.668 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015980_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015980_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=004674\n",
      "+ poretools_dir=004674\n",
      "+ sample_tag=004674_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 004674_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_004674_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3280 sequences (5227960 bp)...\n",
      "[M::mem_process_seqs] Processed 3280 reads in 26.751 CPU sec, 26.205 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 004674_hq.fasta\n",
      "[main] Real time: 30.251 sec; CPU: 29.174 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_004674_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_004674_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=IPD_2015_6505\n",
      "+ poretools_dir=IPD_2015_6505\n",
      "+ sample_tag=IPD_2015_6505_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta IPD_2015_6505_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_IPD_2015_6505_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3189 sequences (5314143 bp)...\n",
      "[M::mem_process_seqs] Processed 3189 reads in 23.488 CPU sec, 23.314 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta IPD_2015_6505_hq.fasta\n",
      "[main] Real time: 27.280 sec; CPU: 25.615 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_IPD_2015_6505_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_IPD_2015_6505_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=013857\n",
      "+ poretools_dir=013857\n",
      "+ sample_tag=013857_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 013857_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_013857_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3632 sequences (6154585 bp)...\n",
      "[M::mem_process_seqs] Processed 3632 reads in 27.309 CPU sec, 26.744 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 013857_hq.fasta\n",
      "[main] Real time: 30.619 sec; CPU: 30.258 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_013857_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_013857_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=015980\n",
      "+ poretools_dir=015980\n",
      "+ sample_tag=015980_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015980_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_015980_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3045 sequences (5106288 bp)...\n",
      "[M::mem_process_seqs] Processed 3045 reads in 26.109 CPU sec, 25.951 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015980_hq.fasta\n",
      "[main] Real time: 29.396 sec; CPU: 28.690 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015980_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015980_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=013731\n",
      "+ poretools_dir=013731\n",
      "+ sample_tag=013731_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 013731_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_013731_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 4182 sequences (6447646 bp)...\n",
      "[M::mem_process_seqs] Processed 4182 reads in 29.188 CPU sec, 29.014 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 013731_hq.fasta\n",
      "[main] Real time: 32.239 sec; CPU: 31.472 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_013731_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_013731_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=013731\n",
      "+ poretools_dir=013731\n",
      "+ sample_tag=013731_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 013731_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_013731_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4182 sequences (6447646 bp)...\n",
      "[M::mem_process_seqs] Processed 4182 reads in 31.002 CPU sec, 31.023 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 013731_hq.fasta\n",
      "[main] Real time: 34.155 sec; CPU: 33.103 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_013731_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_013731_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016263\n",
      "+ poretools_dir=016263\n",
      "+ sample_tag=016263_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016263_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_016263_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3841 sequences (5811205 bp)...\n",
      "[M::mem_process_seqs] Processed 3841 reads in 26.318 CPU sec, 26.326 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016263_hq.fasta\n",
      "[main] Real time: 29.281 sec; CPU: 28.287 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016263_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016263_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016414\n",
      "+ poretools_dir=016414\n",
      "+ sample_tag=016414_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016414_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016414_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3445 sequences (5331469 bp)...\n",
      "[M::mem_process_seqs] Processed 3445 reads in 23.054 CPU sec, 23.057 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016414_hq.fasta\n",
      "[main] Real time: 23.876 sec; CPU: 23.112 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016414_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016414_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016414\n",
      "+ poretools_dir=016414\n",
      "+ sample_tag=016414_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016414_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016414_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3445 sequences (5331469 bp)...\n",
      "[M::mem_process_seqs] Processed 3445 reads in 22.005 CPU sec, 22.007 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016414_hq.fasta\n",
      "[main] Real time: 23.179 sec; CPU: 22.105 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016414_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016414_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=014370\n",
      "+ poretools_dir=014370\n",
      "+ sample_tag=014370_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014370_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_014370_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 8107 sequences (8526115 bp)...\n",
      "[M::mem_process_seqs] Processed 8107 reads in 31.250 CPU sec, 30.803 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014370_hq.fasta\n",
      "[main] Real time: 34.599 sec; CPU: 33.400 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014370_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014370_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=014370\n",
      "+ poretools_dir=014370\n",
      "+ sample_tag=014370_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014370_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_014370_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 8107 sequences (8526115 bp)...\n",
      "[M::mem_process_seqs] Processed 8107 reads in 31.943 CPU sec, 31.677 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014370_hq.fasta\n",
      "[main] Real time: 35.409 sec; CPU: 34.026 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014370_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014370_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016263\n",
      "+ poretools_dir=016263\n",
      "+ sample_tag=016263_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016263_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016263_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3841 sequences (5811205 bp)...\n",
      "[M::mem_process_seqs] Processed 3841 reads in 29.415 CPU sec, 29.415 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016263_hq.fasta\n",
      "[main] Real time: 32.248 sec; CPU: 31.171 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016263_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016263_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016267\n",
      "+ poretools_dir=016267\n",
      "+ sample_tag=016267_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016267_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016267_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4105 sequences (6178580 bp)...\n",
      "[M::mem_process_seqs] Processed 4105 reads in 30.480 CPU sec, 30.468 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016267_hq.fasta\n",
      "[main] Real time: 33.079 sec; CPU: 32.182 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016267_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016267_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016267\n",
      "+ poretools_dir=016267\n",
      "+ sample_tag=016267_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016267_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016267_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4105 sequences (6178580 bp)...\n",
      "[M::mem_process_seqs] Processed 4105 reads in 31.613 CPU sec, 31.603 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016267_hq.fasta\n",
      "[main] Real time: 34.128 sec; CPU: 33.172 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016267_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016267_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016483\n",
      "+ poretools_dir=016483\n",
      "+ sample_tag=016483_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016483_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016483_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2755 sequences (4251908 bp)...\n",
      "[M::mem_process_seqs] Processed 2755 reads in 17.842 CPU sec, 17.860 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016483_hq.fasta\n",
      "[main] Real time: 18.815 sec; CPU: 18.138 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016483_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016483_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016238\n",
      "+ poretools_dir=016238\n",
      "+ sample_tag=016238_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016238_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_016238_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4435 sequences (7299471 bp)...\n",
      "[M::mem_process_seqs] Processed 4435 reads in 35.919 CPU sec, 35.926 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016238_hq.fasta\n",
      "[main] Real time: 39.354 sec; CPU: 38.133 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016238_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016238_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016483\n",
      "+ poretools_dir=016483\n",
      "+ sample_tag=016483_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016483_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016483_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2755 sequences (4251908 bp)...\n",
      "[M::mem_process_seqs] Processed 2755 reads in 18.757 CPU sec, 18.760 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016483_hq.fasta\n",
      "[main] Real time: 19.882 sec; CPU: 19.044 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016483_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016483_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016238\n",
      "+ poretools_dir=016238\n",
      "+ sample_tag=016238_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016238_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_016238_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4435 sequences (7299471 bp)...\n",
      "[M::mem_process_seqs] Processed 4435 reads in 37.461 CPU sec, 37.281 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016238_hq.fasta\n",
      "[main] Real time: 40.993 sec; CPU: 39.839 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016238_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016238_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=014060\n",
      "+ poretools_dir=014060\n",
      "+ sample_tag=014060_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014060_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_014060_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3108 sequences (4774531 bp)...\n",
      "[M::mem_process_seqs] Processed 3108 reads in 18.963 CPU sec, 18.963 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014060_hq.fasta\n",
      "[main] Real time: 19.700 sec; CPU: 19.069 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014060_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014060_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=015815\n",
      "+ poretools_dir=015815\n",
      "+ sample_tag=015815_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015815_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_015815_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 6258 sequences (10002024 bp)...\n",
      "[M::process] read 712 sequences (1082398 bp)...\n",
      "[M::mem_process_seqs] Processed 6258 reads in 39.038 CPU sec, 37.798 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 712 reads in 4.483 CPU sec, 4.430 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015815_hq.fasta\n",
      "[main] Real time: 46.547 sec; CPU: 47.546 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015815_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015815_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=013671\n",
      "+ poretools_dir=013671\n",
      "+ sample_tag=013671_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 013671_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_013671_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5895 sequences (9968647 bp)...\n",
      "[M::mem_process_seqs] Processed 5895 reads in 41.360 CPU sec, 40.029 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 013671_hq.fasta\n",
      "[main] Real time: 46.362 sec; CPU: 45.812 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_013671_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_013671_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=014098\n",
      "+ poretools_dir=014098\n",
      "+ sample_tag=014098_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014098_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_014098_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5769 sequences (9569170 bp)...\n",
      "[M::mem_process_seqs] Processed 5769 reads in 44.081 CPU sec, 43.281 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014098_hq.fasta\n",
      "[main] Real time: 48.538 sec; CPU: 48.164 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014098_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014098_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=013671\n",
      "+ poretools_dir=013671\n",
      "+ sample_tag=013671_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 013671_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_013671_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5895 sequences (9968647 bp)...\n",
      "[M::mem_process_seqs] Processed 5895 reads in 44.651 CPU sec, 43.464 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 013671_hq.fasta\n",
      "[main] Real time: 49.246 sec; CPU: 48.981 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_013671_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_013671_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016531\n",
      "+ poretools_dir=016531\n",
      "+ sample_tag=016531_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016531_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016531_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3716 sequences (5652141 bp)...\n",
      "[M::mem_process_seqs] Processed 3716 reads in 25.139 CPU sec, 25.139 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016531_hq.fasta\n",
      "[main] Real time: 26.023 sec; CPU: 25.237 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016531_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016531_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016456\n",
      "+ poretools_dir=016456\n",
      "+ sample_tag=016456_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016456_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016456_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4561 sequences (7192750 bp)...\n",
      "[M::mem_process_seqs] Processed 4561 reads in 27.014 CPU sec, 27.019 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016456_hq.fasta\n",
      "[main] Real time: 28.314 sec; CPU: 27.234 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016456_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016456_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016515\n",
      "+ poretools_dir=016515\n",
      "+ sample_tag=016515_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016515_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016515_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4185 sequences (6547147 bp)...\n",
      "[M::mem_process_seqs] Processed 4185 reads in 26.092 CPU sec, 26.121 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016515_hq.fasta\n",
      "[main] Real time: 27.090 sec; CPU: 26.162 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016515_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016515_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=015802\n",
      "+ poretools_dir=015802\n",
      "+ sample_tag=015802_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015802_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_015802_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 6426 sequences (10001686 bp)...\n",
      "[M::process] read 413 sequences (615970 bp)...\n",
      "[M::mem_process_seqs] Processed 6426 reads in 43.072 CPU sec, 42.836 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 413 reads in 2.915 CPU sec, 2.882 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015802_hq.fasta\n",
      "[main] Real time: 49.329 sec; CPU: 49.478 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015802_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015802_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=014060\n",
      "+ poretools_dir=014060\n",
      "+ sample_tag=014060_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014060_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_014060_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "[M::process] read 3108 sequences (4774531 bp)...\n",
      "[M::mem_process_seqs] Processed 3108 reads in 20.726 CPU sec, 20.763 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014060_hq.fasta\n",
      "[main] Real time: 21.443 sec; CPU: 20.760 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014060_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014060_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=015815\n",
      "+ poretools_dir=015815\n",
      "+ sample_tag=015815_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015815_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_015815_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 6258 sequences (10002024 bp)...\n",
      "[M::process] read 712 sequences (1082398 bp)...\n",
      "[M::mem_process_seqs] Processed 6258 reads in 41.004 CPU sec, 39.775 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 712 reads in 6.387 CPU sec, 6.328 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015815_hq.fasta\n",
      "[main] Real time: 50.375 sec; CPU: 51.507 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015815_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015815_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=015802\n",
      "+ poretools_dir=015802\n",
      "+ sample_tag=015802_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015802_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_015802_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 6426 sequences (10001686 bp)...\n",
      "[M::process] read 413 sequences (615970 bp)...\n",
      "[M::mem_process_seqs] Processed 6426 reads in 43.946 CPU sec, 43.431 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 413 reads in 2.790 CPU sec, 2.766 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015802_hq.fasta\n",
      "[main] Real time: 49.890 sec; CPU: 50.275 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015802_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015802_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016515\n",
      "+ poretools_dir=016515\n",
      "+ sample_tag=016515_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016515_hq.fasta\n",
      "+ samtools sort - EM_079517_016515_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4185 sequences (6547147 bp)...\n",
      "[M::mem_process_seqs] Processed 4185 reads in 29.028 CPU sec, 29.028 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016515_hq.fasta\n",
      "[main] Real time: 30.171 sec; CPU: 29.229 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016515_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016515_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016456\n",
      "+ poretools_dir=016456\n",
      "+ sample_tag=016456_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016456_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016456_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4561 sequences (7192750 bp)...\n",
      "[M::mem_process_seqs] Processed 4561 reads in 29.846 CPU sec, 29.846 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016456_hq.fasta\n",
      "[main] Real time: 31.141 sec; CPU: 30.072 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016456_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016456_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016531\n",
      "+ poretools_dir=016531\n",
      "+ sample_tag=016531_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016531_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_016531_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3716 sequences (5652141 bp)...\n",
      "[M::mem_process_seqs] Processed 3716 reads in 25.237 CPU sec, 25.244 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016531_hq.fasta\n",
      "[main] Real time: 26.142 sec; CPU: 25.308 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016531_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016531_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016854\n",
      "+ poretools_dir=016854\n",
      "+ sample_tag=016854_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016854_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 804 sequences (1166492 bp)...\n",
      "[M::mem_process_seqs] Processed 804 reads in 5.920 CPU sec, 5.919 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016854_hq.fasta\n",
      "[main] Real time: 6.100 sec; CPU: 5.936 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016854_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016854_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=014100\n",
      "+ poretools_dir=014100\n",
      "+ sample_tag=014100_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014100_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_014100_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 6064 sequences (10001638 bp)...\n",
      "[M::process] read 337 sequences (510273 bp)...\n",
      "[M::mem_process_seqs] Processed 6064 reads in 47.690 CPU sec, 46.957 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 337 reads in 3.639 CPU sec, 3.630 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014100_hq.fasta\n",
      "[main] Real time: 54.720 sec; CPU: 55.345 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014100_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014100_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=014100\n",
      "+ poretools_dir=014100\n",
      "+ sample_tag=014100_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014100_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_014100_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 6064 sequences (10001638 bp)...\n",
      "[M::process] read 337 sequences (510273 bp)...\n",
      "[M::mem_process_seqs] Processed 6064 reads in 46.227 CPU sec, 45.886 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 337 reads in 3.552 CPU sec, 3.547 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014100_hq.fasta\n",
      "[main] Real time: 53.465 sec; CPU: 53.747 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014100_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014100_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016617\n",
      "+ poretools_dir=016617\n",
      "+ sample_tag=016617_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016617_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016617_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3277 sequences (5478386 bp)...\n",
      "[M::mem_process_seqs] Processed 3277 reads in 23.230 CPU sec, 23.232 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016617_hq.fasta\n",
      "[main] Real time: 24.240 sec; CPU: 23.372 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016617_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "fasta ../refs/EM_079517.fasta --bam EM_079517_016617_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016617\n",
      "+ poretools_dir=016617\n",
      "+ sample_tag=016617_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016617_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016617_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3277 sequences (5478386 bp)...\n",
      "[M::mem_process_seqs] Processed 3277 reads in 22.072 CPU sec, 22.100 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016617_hq.fasta\n",
      "[main] Real time: 22.997 sec; CPU: 22.118 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016617_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016617_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016854\n",
      "+ poretools_dir=016854\n",
      "+ sample_tag=016854_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016854_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016854_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 804 sequences (1166492 bp)...\n",
      "[M::mem_process_seqs] Processed 804 reads in 6.544 CPU sec, 6.545 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016854_hq.fasta\n",
      "[main] Real time: 6.722 sec; CPU: 6.559 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016854_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016854_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=014098\n",
      "+ poretools_dir=014098\n",
      "+ sample_tag=014098_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014098_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_014098_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5769 sequences (9569170 bp)...\n",
      "[M::mem_process_seqs] Processed 5769 reads in 55.622 CPU sec, 54.561 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014098_hq.fasta\n",
      "[main] Real time: 60.167 sec; CPU: 59.802 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014098_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014098_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016449\n",
      "+ poretools_dir=016449\n",
      "+ sample_tag=016449_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016449_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016449_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5138 sequences (8058697 bp)...\n",
      "[M::mem_process_seqs] Processed 5138 reads in 35.281 CPU sec, 35.285 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016449_hq.fasta\n",
      "[main] Real time: 36.637 sec; CPU: 35.616 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016449_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016449_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016278\n",
      "+ poretools_dir=016278\n",
      "+ sample_tag=016278_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016278_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016278_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3646 sequences (6065185 bp)...\n",
      "[M::mem_process_seqs] Processed 3646 reads in 24.842 CPU sec, 24.847 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016278_hq.fasta\n",
      "[main] Real time: 25.879 sec; CPU: 24.932 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016278_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016278_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016449\n",
      "+ poretools_dir=016449\n",
      "+ sample_tag=016449_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016449_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016449_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 5138 sequences (8058697 bp)...\n",
      "[M::mem_process_seqs] Processed 5138 reads in 36.044 CPU sec, 36.050 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016449_hq.fasta\n",
      "[main] Real time: 37.046 sec; CPU: 36.103 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016449_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016449_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016278\n",
      "+ poretools_dir=016278\n",
      "+ sample_tag=016278_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016278_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016278_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3646 sequences (6065185 bp)...\n",
      "[M::mem_process_seqs] Processed 3646 reads in 26.255 CPU sec, 26.267 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016278_hq.fasta\n",
      "[main] Real time: 27.305 sec; CPU: 26.440 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016278_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016278_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=013795\n",
      "+ poretools_dir=013795\n",
      "+ sample_tag=013795_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 013795_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_013795_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3722 sequences (5576475 bp)...\n",
      "[M::mem_process_seqs] Processed 3722 reads in 24.856 CPU sec, 24.866 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 013795_hq.fasta\n",
      "[main] Real time: 25.817 sec; CPU: 24.929 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_013795_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_013795_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=014465\n",
      "+ poretools_dir=014465\n",
      "+ sample_tag=014465_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014465_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_014465_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2101 sequences (3335145 bp)...\n",
      "[M::mem_process_seqs] Processed 2101 reads in 12.536 CPU sec, 12.539 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014465_hq.fasta\n",
      "[main] Real time: 13.010 sec; CPU: 12.563 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014465_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014465_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=013795\n",
      "+ poretools_dir=013795\n",
      "+ sample_tag=013795_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_013795_hq_bwa.sorted\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 013795_hq.fasta\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3722 sequences (5576475 bp)...\n",
      "[M::mem_process_seqs] Processed 3722 reads in 27.857 CPU sec, 27.864 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 013795_hq.fasta\n",
      "[main] Real time: 28.834 sec; CPU: 27.910 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_013795_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_013795_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=014102\n",
      "+ poretools_dir=014102\n",
      "+ sample_tag=014102_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014102_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_014102_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3577 sequences (6053658 bp)...\n",
      "[M::mem_process_seqs] Processed 3577 reads in 25.862 CPU sec, 25.866 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014102_hq.fasta\n",
      "[main] Real time: 26.696 sec; CPU: 25.963 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014102_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014102_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=015984\n",
      "+ poretools_dir=015984\n",
      "+ sample_tag=015984_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015984_hq.fasta\n",
      "+ samtools sort - EM_079517_015984_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 952 sequences (1468763 bp)...\n",
      "[M::mem_process_seqs] Processed 952 reads in 5.855 CPU sec, 5.856 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 015984_hq.fasta\n",
      "[main] Real time: 6.084 sec; CPU: 5.884 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015984_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_015984_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=014465\n",
      "+ poretools_dir=014465\n",
      "+ sample_tag=014465_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014465_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_014465_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2101 sequences (3335145 bp)...\n",
      "[M::mem_process_seqs] Processed 2101 reads in 14.313 CPU sec, 14.324 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014465_hq.fasta\n",
      "[main] Real time: 14.758 sec; CPU: 14.368 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014465_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014465_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016236\n",
      "+ poretools_dir=016236\n",
      "+ sample_tag=016236_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016236_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016236_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 4627 sequences (7838046 bp)...\n",
      "[M::mem_process_seqs] Processed 4627 reads in 35.456 CPU sec, 35.471 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016236_hq.fasta\n",
      "[main] Real time: 36.492 sec; CPU: 35.589 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016236_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016236_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=014102\n",
      "+ poretools_dir=014102\n",
      "+ sample_tag=014102_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014102_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_014102_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3577 sequences (6053658 bp)...\n",
      "[M::mem_process_seqs] Processed 3577 reads in 27.053 CPU sec, 27.060 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014102_hq.fasta\n",
      "[main] Real time: 27.924 sec; CPU: 27.121 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014102_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014102_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=015984\n",
      "+ poretools_dir=015984\n",
      "+ sample_tag=015984_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015984_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_015984_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 952 sequences (1468763 bp)...\n",
      "[M::mem_process_seqs] Processed 952 reads in 7.680 CPU sec, 7.684 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 015984_hq.fasta\n",
      "[main] Real time: 7.899 sec; CPU: 7.692 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015984_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_015984_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016642\n",
      "+ poretools_dir=016642\n",
      "+ sample_tag=016642_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016642_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016642_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4379 sequences (7465101 bp)...\n",
      "[M::mem_process_seqs] Processed 4379 reads in 34.377 CPU sec, 34.387 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016642_hq.fasta\n",
      "[main] Real time: 35.360 sec; CPU: 34.496 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016642_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016642_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016464\n",
      "+ poretools_dir=016464\n",
      "+ sample_tag=016464_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016464_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016464_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3996 sequences (5266850 bp)...\n",
      "[M::mem_process_seqs] Processed 3996 reads in 19.229 CPU sec, 19.229 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016464_hq.fasta\n",
      "[main] Real time: 20.049 sec; CPU: 19.266 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016464_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016464_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016852\n",
      "+ poretools_dir=016852\n",
      "+ sample_tag=016852_hq\n",
      "+ second_batch=016852R\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016852_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_016852_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4507 sequences (4413203 bp)...\n",
      "[M::mem_process_seqs] Processed 4507 reads in 14.051 CPU sec, 14.051 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016852_hq.fasta\n",
      "[main] Real time: 14.674 sec; CPU: 14.128 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016852_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016852_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016464\n",
      "+ poretools_dir=016464\n",
      "+ sample_tag=016464_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016464_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016464_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3996 sequences (5266850 bp)...\n",
      "[M::mem_process_seqs] Processed 3996 reads in 22.840 CPU sec, 22.856 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016464_hq.fasta\n",
      "[main] Real time: 23.765 sec; CPU: 22.917 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016464_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016464_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016236\n",
      "+ poretools_dir=016236\n",
      "+ sample_tag=016236_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016236_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016236_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4627 sequences (7838046 bp)...\n",
      "[M::mem_process_seqs] Processed 4627 reads in 39.429 CPU sec, 39.507 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016236_hq.fasta\n",
      "[main] Real time: 40.772 sec; CPU: 39.569 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016236_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016236_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016642\n",
      "+ poretools_dir=016642\n",
      "+ sample_tag=016642_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016642_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016642_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 4379 sequences (7465101 bp)...\n",
      "[M::mem_process_seqs] Processed 4379 reads in 35.898 CPU sec, 35.891 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016642_hq.fasta\n",
      "[main] Real time: 37.066 sec; CPU: 36.010 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016642_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016642_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016852\n",
      "+ poretools_dir=016852\n",
      "+ sample_tag=016852_hq\n",
      "+ second_batch=016852R\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016852_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016852_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4507 sequences (4413203 bp)...\n",
      "[M::mem_process_seqs] Processed 4507 reads in 14.165 CPU sec, 14.166 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016852_hq.fasta\n",
      "[main] Real time: 14.706 sec; CPU: 14.205 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016852_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016852_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016743\n",
      "+ poretools_dir=016743\n",
      "+ sample_tag=016743_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016743_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016743_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3079 sequences (5134485 bp)...\n",
      "[M::mem_process_seqs] Processed 3079 reads in 26.727 CPU sec, 26.726 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016743_hq.fasta\n",
      "[main] Real time: 27.632 sec; CPU: 26.807 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016743_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016743_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016743\n",
      "+ poretools_dir=016743\n",
      "+ sample_tag=016743_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016743_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016743_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3079 sequences (5134485 bp)...\n",
      "[M::mem_process_seqs] Processed 3079 reads in 25.215 CPU sec, 25.219 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016743_hq.fasta\n",
      "[main] Real time: 26.063 sec; CPU: 25.258 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016743_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016743_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017174\n",
      "+ poretools_dir=017174\n",
      "+ sample_tag=017174_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017174_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017174_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2924 sequences (2888188 bp)...\n",
      "[M::mem_process_seqs] Processed 2924 reads in 9.416 CPU sec, 9.419 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017174_hq.fasta\n",
      "[main] Real time: 10.057 sec; CPU: 9.480 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017174_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017174_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016293\n",
      "+ poretools_dir=016293\n",
      "+ sample_tag=016293_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016293_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016293_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4794 sequences (7517531 bp)...\n",
      "[M::mem_process_seqs] Processed 4794 reads in 37.915 CPU sec, 37.917 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016293_hq.fasta\n",
      "[main] Real time: 39.383 sec; CPU: 38.206 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016293_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016293_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016653\n",
      "+ poretools_dir=016653\n",
      "+ sample_tag=016653_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016653_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_016653_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3790 sequences (5100313 bp)...\n",
      "[M::mem_process_seqs] Processed 3790 reads in 20.222 CPU sec, 20.223 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016653_hq.fasta\n",
      "[main] Real time: 21.165 sec; CPU: 20.357 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016653_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016653_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017174\n",
      "+ poretools_dir=017174\n",
      "+ sample_tag=017174_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017174_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_017174_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2924 sequences (2888188 bp)...\n",
      "[M::mem_process_seqs] Processed 2924 reads in 12.814 CPU sec, 12.816 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017174_hq.fasta\n",
      "[main] Real time: 13.375 sec; CPU: 12.847 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017174_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017174_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016653\n",
      "+ poretools_dir=016653\n",
      "+ sample_tag=016653_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016653_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016653_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3790 sequences (5100313 bp)...\n",
      "[M::mem_process_seqs] Processed 3790 reads in 21.050 CPU sec, 21.055 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016653_hq.fasta\n",
      "[main] Real time: 21.882 sec; CPU: 21.093 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016653_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016653_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016293\n",
      "+ poretools_dir=016293\n",
      "+ sample_tag=016293_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016293_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_016293_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4794 sequences (7517531 bp)...\n",
      "[M::mem_process_seqs] Processed 4794 reads in 41.051 CPU sec, 41.060 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016293_hq.fasta\n",
      "[main] Real time: 42.517 sec; CPU: 41.336 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016293_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016293_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016602R\n",
      "+ poretools_dir=016602R\n",
      "+ sample_tag=016602R_hq\n",
      "+ second_batch=016602\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016602R_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_016602R_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 8450 sequences (8754997 bp)...\n",
      "[M::mem_process_seqs] Processed 8450 reads in 27.901 CPU sec, 27.906 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016602R_hq.fasta\n",
      "[main] Real time: 29.118 sec; CPU: 27.975 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016602R_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016602R_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=014261\n",
      "+ poretools_dir=014261\n",
      "+ sample_tag=014261_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014261_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_014261_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3309 sequences (3385893 bp)...\n",
      "[M::mem_process_seqs] Processed 3309 reads in 10.605 CPU sec, 10.606 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 014261_hq.fasta\n",
      "[main] Real time: 11.047 sec; CPU: 10.639 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014261_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_014261_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017018\n",
      "+ poretools_dir=017018\n",
      "+ sample_tag=017018_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017018_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017018_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3448 sequences (5610188 bp)...\n",
      "[M::mem_process_seqs] Processed 3448 reads in 23.667 CPU sec, 23.670 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017018_hq.fasta\n",
      "[main] Real time: 24.521 sec; CPU: 23.711 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017018_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017018_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017500\n",
      "+ poretools_dir=017500\n",
      "+ sample_tag=017500_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017500_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017500_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 1115 sequences (1920933 bp)...\n",
      "[M::mem_process_seqs] Processed 1115 reads in 10.108 CPU sec, 10.121 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017500_hq.fasta\n",
      "[main] Real time: 10.391 sec; CPU: 10.127 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017500_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017500_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016602R\n",
      "+ poretools_dir=016602R\n",
      "+ sample_tag=016602R_hq\n",
      "+ second_batch=016602\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016602R_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_016602R_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 8450 sequences (8754997 bp)...\n",
      "[M::mem_process_seqs] Processed 8450 reads in 31.186 CPU sec, 31.188 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016602R_hq.fasta\n",
      "[main] Real time: 32.178 sec; CPU: 31.265 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016602R_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016602R_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017135\n",
      "+ poretools_dir=017135\n",
      "+ sample_tag=017135_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017135_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017135_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2721 sequences (4563393 bp)...\n",
      "[M::mem_process_seqs] Processed 2721 reads in 21.930 CPU sec, 21.939 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017135_hq.fasta\n",
      "[main] Real time: 22.744 sec; CPU: 22.028 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017135_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017135_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=013576\n",
      "+ poretools_dir=013576\n",
      "+ sample_tag=013576_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 013576_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_013576_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5806 sequences (9629878 bp)...\n",
      "[M::mem_process_seqs] Processed 5806 reads in 43.692 CPU sec, 43.718 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 013576_hq.fasta\n",
      "[main] Real time: 45.080 sec; CPU: 43.841 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_013576_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_013576_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=014261\n",
      "+ poretools_dir=014261\n",
      "+ sample_tag=014261_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014261_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_014261_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "[M::process] read 3309 sequences (3385893 bp)...\n",
      "[M::mem_process_seqs] Processed 3309 reads in 12.297 CPU sec, 12.309 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 014261_hq.fasta\n",
      "[main] Real time: 12.844 sec; CPU: 12.339 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014261_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_014261_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017500\n",
      "+ poretools_dir=017500\n",
      "+ sample_tag=017500_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017500_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017500_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 1115 sequences (1920933 bp)...\n",
      "[M::mem_process_seqs] Processed 1115 reads in 9.581 CPU sec, 9.591 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017500_hq.fasta\n",
      "[main] Real time: 9.943 sec; CPU: 9.599 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017500_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017500_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=013576\n",
      "+ poretools_dir=013576\n",
      "+ sample_tag=013576_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 013576_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_013576_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 5806 sequences (9629878 bp)...\n",
      "[M::mem_process_seqs] Processed 5806 reads in 44.106 CPU sec, 44.121 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 013576_hq.fasta\n",
      "[main] Real time: 45.401 sec; CPU: 44.224 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_013576_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_013576_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=016800\n",
      "+ poretools_dir=016800\n",
      "+ sample_tag=016800_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016800_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_016800_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4880 sequences (8078682 bp)...\n",
      "[M::mem_process_seqs] Processed 4880 reads in 35.330 CPU sec, 35.334 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 016800_hq.fasta\n",
      "[main] Real time: 36.703 sec; CPU: 35.467 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016800_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_016800_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017018\n",
      "+ poretools_dir=017018\n",
      "+ sample_tag=017018_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017018_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017018_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3448 sequences (5610188 bp)...\n",
      "[M::mem_process_seqs] Processed 3448 reads in 27.765 CPU sec, 27.768 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017018_hq.fasta\n",
      "[main] Real time: 28.740 sec; CPU: 27.864 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017018_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017018_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=016800\n",
      "+ poretools_dir=016800\n",
      "+ sample_tag=016800_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016800_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_016800_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4880 sequences (8078682 bp)...\n",
      "[M::mem_process_seqs] Processed 4880 reads in 35.152 CPU sec, 35.167 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 016800_hq.fasta\n",
      "[main] Real time: 36.448 sec; CPU: 35.204 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016800_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_016800_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017021\n",
      "+ poretools_dir=017021\n",
      "+ sample_tag=017021_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017021_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017021_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3979 sequences (6278478 bp)...\n",
      "[M::mem_process_seqs] Processed 3979 reads in 30.459 CPU sec, 30.467 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017021_hq.fasta\n",
      "[main] Real time: 31.554 sec; CPU: 30.553 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017021_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017021_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017135\n",
      "+ poretools_dir=017135\n",
      "+ sample_tag=017135_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017135_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_017135_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2721 sequences (4563393 bp)...\n",
      "[M::mem_process_seqs] Processed 2721 reads in 25.269 CPU sec, 25.279 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017135_hq.fasta\n",
      "[main] Real time: 25.948 sec; CPU: 25.307 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017135_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017135_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017240\n",
      "+ poretools_dir=017240\n",
      "+ sample_tag=017240_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017240_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017240_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2726 sequences (4282098 bp)...\n",
      "[M::mem_process_seqs] Processed 2726 reads in 17.362 CPU sec, 17.363 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017240_hq.fasta\n",
      "[main] Real time: 18.118 sec; CPU: 17.456 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017240_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017240_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017021\n",
      "+ poretools_dir=017021\n",
      "+ sample_tag=017021_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017021_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017021_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3979 sequences (6278478 bp)...\n",
      "[M::mem_process_seqs] Processed 3979 reads in 28.811 CPU sec, 28.822 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017021_hq.fasta\n",
      "[main] Real time: 29.789 sec; CPU: 28.864 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017021_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017021_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017057\n",
      "+ poretools_dir=017057\n",
      "+ sample_tag=017057_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017057_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_017057_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4015 sequences (6067319 bp)...\n",
      "[M::mem_process_seqs] Processed 4015 reads in 28.975 CPU sec, 28.992 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017057_hq.fasta\n",
      "[main] Real time: 29.915 sec; CPU: 29.024 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017057_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017057_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017091\n",
      "+ poretools_dir=017091\n",
      "+ sample_tag=017091_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017091_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017091_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4102 sequences (6522531 bp)...\n",
      "[M::mem_process_seqs] Processed 4102 reads in 28.320 CPU sec, 28.323 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017091_hq.fasta\n",
      "[main] Real time: 29.391 sec; CPU: 28.403 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017091_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017091_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017240\n",
      "+ poretools_dir=017240\n",
      "+ sample_tag=017240_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017240_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017240_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2726 sequences (4282098 bp)...\n",
      "[M::mem_process_seqs] Processed 2726 reads in 17.668 CPU sec, 17.668 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017240_hq.fasta\n",
      "[main] Real time: 18.430 sec; CPU: 17.760 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017240_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017574\n",
      "+ poretools_dir=017574\n",
      "+ sample_tag=017574_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017574_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017574_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2162 sequences (3570549 bp)...\n",
      "[M::mem_process_seqs] Processed 2162 reads in 13.700 CPU sec, 13.700 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017574_hq.fasta\n",
      "[main] Real time: 14.225 sec; CPU: 13.729 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017574_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017574_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017057\n",
      "+ poretools_dir=017057\n",
      "+ sample_tag=017057_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017057_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017057_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4015 sequences (6067319 bp)...\n",
      "[M::mem_process_seqs] Processed 4015 reads in 31.630 CPU sec, 31.632 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017057_hq.fasta\n",
      "[main] Real time: 32.797 sec; CPU: 31.756 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017057_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017057_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017091\n",
      "+ poretools_dir=017091\n",
      "+ sample_tag=017091_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017091_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017091_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4102 sequences (6522531 bp)...\n",
      "[M::mem_process_seqs] Processed 4102 reads in 30.561 CPU sec, 30.561 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017091_hq.fasta\n",
      "[main] Real time: 31.675 sec; CPU: 30.660 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017091_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017091_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017790\n",
      "+ poretools_dir=017790\n",
      "+ sample_tag=017790_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017790_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017790_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 946 sequences (1434540 bp)...\n",
      "[M::mem_process_seqs] Processed 946 reads in 6.563 CPU sec, 6.578 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017790_hq.fasta\n",
      "[main] Real time: 6.858 sec; CPU: 6.578 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017790_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017790_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017574\n",
      "+ poretools_dir=017574\n",
      "+ sample_tag=017574_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017574_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017574_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2162 sequences (3570549 bp)...\n",
      "[M::mem_process_seqs] Processed 2162 reads in 16.540 CPU sec, 16.541 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017574_hq.fasta\n",
      "[main] Real time: 17.063 sec; CPU: 16.588 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017574_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017574_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017666\n",
      "+ poretools_dir=017666\n",
      "+ sample_tag=017666_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017666_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_017666_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2303 sequences (3948703 bp)...\n",
      "[M::mem_process_seqs] Processed 2303 reads in 14.141 CPU sec, 14.141 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017666_hq.fasta\n",
      "[main] Real time: 14.757 sec; CPU: 14.212 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017666_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017666_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017790\n",
      "+ poretools_dir=017790\n",
      "+ sample_tag=017790_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017790_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017790_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 946 sequences (1434540 bp)...\n",
      "[M::mem_process_seqs] Processed 946 reads in 7.600 CPU sec, 7.601 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017790_hq.fasta\n",
      "[main] Real time: 7.858 sec; CPU: 7.616 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017790_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017790_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017664\n",
      "+ poretools_dir=017664\n",
      "+ sample_tag=017664_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017664_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_017664_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2715 sequences (4432415 bp)...\n",
      "[M::mem_process_seqs] Processed 2715 reads in 18.001 CPU sec, 18.001 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017664_hq.fasta\n",
      "[main] Real time: 18.593 sec; CPU: 18.034 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017664_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017664_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017499\n",
      "+ poretools_dir=017499\n",
      "+ sample_tag=017499_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017499_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017499_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3025 sequences (5405070 bp)...\n",
      "[M::mem_process_seqs] Processed 3025 reads in 20.145 CPU sec, 20.147 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017499_hq.fasta\n",
      "[main] Real time: 20.975 sec; CPU: 20.194 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017499_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017499_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017590\n",
      "+ poretools_dir=017590\n",
      "+ sample_tag=017590_hq\n",
      "+ second_batch=017590_2\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017590_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_017590_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2197 sequences (3449621 bp)...\n",
      "[M::mem_process_seqs] Processed 2197 reads in 18.175 CPU sec, 18.183 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017590_hq.fasta\n",
      "[main] Real time: 18.725 sec; CPU: 18.202 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017590_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017590_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017590\n",
      "+ poretools_dir=017590\n",
      "+ sample_tag=017590_hq\n",
      "+ second_batch=017590_2\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017590_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017590_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2197 sequences (3449621 bp)...\n",
      "[M::mem_process_seqs] Processed 2197 reads in 17.581 CPU sec, 17.589 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017590_hq.fasta\n",
      "[main] Real time: 18.166 sec; CPU: 17.609 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017590_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017590_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017666\n",
      "+ poretools_dir=017666\n",
      "+ sample_tag=017666_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017666_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017666_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2303 sequences (3948703 bp)...\n",
      "[M::mem_process_seqs] Processed 2303 reads in 18.294 CPU sec, 18.300 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017666_hq.fasta\n",
      "[main] Real time: 19.040 sec; CPU: 18.364 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017666_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=KG90\n",
      "+ poretools_dir=KG90\n",
      "+ sample_tag=KG90_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG90_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_KG90_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 1440 sequences (2080970 bp)...\n",
      "[M::mem_process_seqs] Processed 1440 reads in 9.936 CPU sec, 9.939 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG90_hq.fasta\n",
      "[main] Real time: 10.326 sec; CPU: 9.970 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG90_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG90_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017664\n",
      "+ poretools_dir=017664\n",
      "+ sample_tag=017664_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017664_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_017664_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2715 sequences (4432415 bp)...\n",
      "[M::mem_process_seqs] Processed 2715 reads in 18.983 CPU sec, 18.985 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017664_hq.fasta\n",
      "[main] Real time: 19.616 sec; CPU: 19.028 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017664_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017664_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=KG90\n",
      "+ poretools_dir=KG90\n",
      "+ sample_tag=KG90_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG90_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_KG90_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 1440 sequences (2080970 bp)...\n",
      "[M::mem_process_seqs] Processed 1440 reads in 10.238 CPU sec, 10.260 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG90_hq.fasta\n",
      "[main] Real time: 10.652 sec; CPU: 10.254 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG90_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG90_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017498\n",
      "+ poretools_dir=017498\n",
      "+ sample_tag=017498_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017498_hq.fasta\n",
      "+ samtools sort - EM_079517_017498_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3136 sequences (5321659 bp)...\n",
      "[M::mem_process_seqs] Processed 3136 reads in 22.131 CPU sec, 22.143 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017498_hq.fasta\n",
      "[main] Real time: 22.950 sec; CPU: 22.202 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017498_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017498_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=018035\n",
      "+ poretools_dir=018035\n",
      "+ sample_tag=018035_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 018035_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_018035_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2148 sequences (2265395 bp)...\n",
      "[M::mem_process_seqs] Processed 2148 reads in 8.166 CPU sec, 8.166 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 018035_hq.fasta\n",
      "[main] Real time: 8.610 sec; CPU: 8.206 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_018035_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_018035_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017498\n",
      "+ poretools_dir=017498\n",
      "+ sample_tag=017498_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017498_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017498_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3136 sequences (5321659 bp)...\n",
      "[M::mem_process_seqs] Processed 3136 reads in 24.163 CPU sec, 24.170 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017498_hq.fasta\n",
      "[main] Real time: 25.161 sec; CPU: 24.228 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017498_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017498_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=018035\n",
      "+ poretools_dir=018035\n",
      "+ sample_tag=018035_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 018035_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_018035_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2148 sequences (2265395 bp)...\n",
      "[M::mem_process_seqs] Processed 2148 reads in 11.065 CPU sec, 11.065 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 018035_hq.fasta\n",
      "[main] Real time: 11.494 sec; CPU: 11.130 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_018035_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_018035_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017734\n",
      "+ poretools_dir=017734\n",
      "+ sample_tag=017734_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017734_hq.fasta\n",
      "+ samtools sort - EM_079517_017734_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2596 sequences (4482787 bp)...\n",
      "[M::mem_process_seqs] Processed 2596 reads in 19.033 CPU sec, 19.043 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017734_hq.fasta\n",
      "[main] Real time: 19.941 sec; CPU: 19.106 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017734_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017734_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017499\n",
      "+ poretools_dir=017499\n",
      "+ sample_tag=017499_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017499_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017499_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3025 sequences (5405070 bp)...\n",
      "[M::mem_process_seqs] Processed 3025 reads in 26.971 CPU sec, 26.976 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017499_hq.fasta\n",
      "[main] Real time: 27.893 sec; CPU: 27.055 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017499_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017499_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017734\n",
      "+ poretools_dir=017734\n",
      "+ sample_tag=017734_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017734_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017734_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2596 sequences (4482787 bp)...\n",
      "[M::mem_process_seqs] Processed 2596 reads in 20.841 CPU sec, 20.842 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017734_hq.fasta\n",
      "[main] Real time: 21.573 sec; CPU: 20.893 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017734_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017734_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017934\n",
      "+ poretools_dir=017934\n",
      "+ sample_tag=017934_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017934_hq.fasta\n",
      "+ samtools sort - EM_079517_017934_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2758 sequences (2999462 bp)...\n",
      "[M::mem_process_seqs] Processed 2758 reads in 12.097 CPU sec, 12.105 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017934_hq.fasta\n",
      "[main] Real time: 12.668 sec; CPU: 12.148 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017934_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017934_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017934\n",
      "+ poretools_dir=017934\n",
      "+ sample_tag=017934_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017934_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017934_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2758 sequences (2999462 bp)...\n",
      "[M::mem_process_seqs] Processed 2758 reads in 12.612 CPU sec, 12.611 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017934_hq.fasta\n",
      "[main] Real time: 13.101 sec; CPU: 12.644 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017934_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017934_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=022093\n",
      "+ poretools_dir=022093\n",
      "+ sample_tag=022093_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022093_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_022093_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 1789 sequences (1829983 bp)...\n",
      "[M::mem_process_seqs] Processed 1789 reads in 5.825 CPU sec, 5.852 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022093_hq.fasta\n",
      "[main] Real time: 6.234 sec; CPU: 5.850 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022093_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022093_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=022093\n",
      "+ poretools_dir=022093\n",
      "+ sample_tag=022093_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022093_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_022093_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 1789 sequences (1829983 bp)...\n",
      "[M::mem_process_seqs] Processed 1789 reads in 6.172 CPU sec, 6.172 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022093_hq.fasta\n",
      "[main] Real time: 6.589 sec; CPU: 6.191 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022093_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022093_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017865\n",
      "+ poretools_dir=017865\n",
      "+ sample_tag=017865_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017865_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017865_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2925 sequences (4878216 bp)...\n",
      "[M::mem_process_seqs] Processed 2925 reads in 21.625 CPU sec, 21.670 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017865_hq.fasta\n",
      "[main] Real time: 22.954 sec; CPU: 21.764 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017865_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017865_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017238\n",
      "+ poretools_dir=017238\n",
      "+ sample_tag=017238_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017238_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017238_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4706 sequences (7891040 bp)...\n",
      "[M::mem_process_seqs] Processed 4706 reads in 34.700 CPU sec, 34.701 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017238_hq.fasta\n",
      "[main] Real time: 36.340 sec; CPU: 34.847 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017238_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017238_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=KG12\n",
      "+ poretools_dir=KG12\n",
      "+ sample_tag=KG12_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG12_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_KG12_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2903 sequences (4890563 bp)...\n",
      "[M::mem_process_seqs] Processed 2903 reads in 19.006 CPU sec, 19.011 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG12_hq.fasta\n",
      "[main] Real time: 20.433 sec; CPU: 19.072 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG12_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG12_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017865\n",
      "+ poretools_dir=017865\n",
      "+ sample_tag=017865_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017865_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_017865_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2925 sequences (4878216 bp)...\n",
      "[M::mem_process_seqs] Processed 2925 reads in 23.880 CPU sec, 23.883 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017865_hq.fasta\n",
      "[main] Real time: 25.250 sec; CPU: 24.031 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017865_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017865_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=022059\n",
      "+ poretools_dir=022059\n",
      "+ sample_tag=022059_hq\n",
      "+ second_batch=022059_rpt\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022059_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_022059_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3700 sequences (3753700 bp)...\n",
      "[M::mem_process_seqs] Processed 3700 reads in 11.706 CPU sec, 11.706 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022059_hq.fasta\n",
      "[main] Real time: 12.314 sec; CPU: 11.739 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022059_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022059_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=KG12\n",
      "+ poretools_dir=KG12\n",
      "+ sample_tag=KG12_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG12_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_KG12_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2903 sequences (4890563 bp)...\n",
      "[M::mem_process_seqs] Processed 2903 reads in 22.593 CPU sec, 22.615 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG12_hq.fasta\n",
      "[main] Real time: 23.496 sec; CPU: 22.675 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG12_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG12_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=8559\n",
      "+ poretools_dir=8559\n",
      "+ sample_tag=8559_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 8559_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_8559_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3096 sequences (4839206 bp)...\n",
      "[M::mem_process_seqs] Processed 3096 reads in 19.507 CPU sec, 19.508 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 8559_hq.fasta\n",
      "[main] Real time: 20.290 sec; CPU: 19.565 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_8559_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_8559_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=8811\n",
      "+ poretools_dir=8811\n",
      "+ sample_tag=8811_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 8811_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_8811_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2820 sequences (4611371 bp)...\n",
      "[M::mem_process_seqs] Processed 2820 reads in 18.186 CPU sec, 18.206 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 8811_hq.fasta\n",
      "[main] Real time: 18.808 sec; CPU: 18.222 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_8811_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_8811_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=022059\n",
      "+ poretools_dir=022059\n",
      "+ sample_tag=022059_hq\n",
      "+ second_batch=022059_rpt\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022059_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_022059_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3700 sequences (3753700 bp)...\n",
      "[M::mem_process_seqs] Processed 3700 reads in 13.007 CPU sec, 13.035 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022059_hq.fasta\n",
      "[main] Real time: 13.694 sec; CPU: 13.043 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022059_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022059_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=018216\n",
      "+ poretools_dir=018216\n",
      "+ sample_tag=018216_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 018216_hq.fasta\n",
      "+ samtools sort - EM_079517_018216_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2678 sequences (4579991 bp)...\n",
      "[M::mem_process_seqs] Processed 2678 reads in 18.912 CPU sec, 18.930 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 018216_hq.fasta\n",
      "[main] Real time: 19.569 sec; CPU: 18.992 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_018216_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_018216_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017796\n",
      "+ poretools_dir=017796\n",
      "+ sample_tag=017796_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017796_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017796_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3346 sequences (5397790 bp)...\n",
      "[M::mem_process_seqs] Processed 3346 reads in 23.820 CPU sec, 23.825 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017796_hq.fasta\n",
      "[main] Real time: 24.779 sec; CPU: 23.876 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017796_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017796_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017796\n",
      "+ poretools_dir=017796\n",
      "+ sample_tag=017796_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017796_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017796_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3346 sequences (5397790 bp)...\n",
      "[M::mem_process_seqs] Processed 3346 reads in 25.373 CPU sec, 25.377 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017796_hq.fasta\n",
      "[main] Real time: 26.323 sec; CPU: 25.424 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017796_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017796_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=20236\n",
      "+ poretools_dir=20236\n",
      "+ sample_tag=20236_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 20236_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_20236_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3597 sequences (3845724 bp)...\n",
      "[M::mem_process_seqs] Processed 3597 reads in 12.949 CPU sec, 12.949 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 20236_hq.fasta\n",
      "[main] Real time: 13.789 sec; CPU: 12.988 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_20236_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_20236_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=20236\n",
      "+ poretools_dir=20236\n",
      "+ sample_tag=20236_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 20236_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_20236_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3597 sequences (3845724 bp)...\n",
      "[M::mem_process_seqs] Processed 3597 reads in 14.663 CPU sec, 14.663 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 20236_hq.fasta\n",
      "[main] Real time: 15.374 sec; CPU: 14.721 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_20236_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_20236_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=018205\n",
      "+ poretools_dir=018205\n",
      "+ sample_tag=018205_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 018205_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_018205_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2844 sequences (4820343 bp)...\n",
      "[M::mem_process_seqs] Processed 2844 reads in 21.778 CPU sec, 21.795 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 018205_hq.fasta\n",
      "[main] Real time: 22.732 sec; CPU: 21.849 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_018205_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_018205_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=018242\n",
      "+ poretools_dir=018242\n",
      "+ sample_tag=018242_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 018242_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_018242_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2845 sequences (3939807 bp)...\n",
      "[M::mem_process_seqs] Processed 2845 reads in 17.615 CPU sec, 17.643 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 018242_hq.fasta\n",
      "[main] Real time: 18.345 sec; CPU: 17.651 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_018242_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_018242_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017238\n",
      "+ poretools_dir=017238\n",
      "+ sample_tag=017238_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017238_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017238_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4706 sequences (7891040 bp)...\n",
      "[M::mem_process_seqs] Processed 4706 reads in 40.555 CPU sec, 40.559 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017238_hq.fasta\n",
      "[main] Real time: 42.264 sec; CPU: 40.705 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017238_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=018205\n",
      "+ poretools_dir=018205\n",
      "+ sample_tag=018205_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 018205_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_018205_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2844 sequences (4820343 bp)...\n",
      "[M::mem_process_seqs] Processed 2844 reads in 22.309 CPU sec, 22.316 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 018205_hq.fasta\n",
      "[main] Real time: 23.296 sec; CPU: 22.352 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_018205_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017788\n",
      "+ poretools_dir=017788\n",
      "+ sample_tag=017788_hq\n",
      "+ second_batch=017788_2\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017788_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017788_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5112 sequences (8390236 bp)...\n",
      "[M::mem_process_seqs] Processed 5112 reads in 28.550 CPU sec, 28.554 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017788_hq.fasta\n",
      "[main] Real time: 30.619 sec; CPU: 28.753 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017788_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017788_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=8811\n",
      "+ poretools_dir=8811\n",
      "+ sample_tag=8811_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 8811_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_8811_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2820 sequences (4611371 bp)...\n",
      "[M::mem_process_seqs] Processed 2820 reads in 23.516 CPU sec, 23.523 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 8811_hq.fasta\n",
      "[main] Real time: 24.224 sec; CPU: 23.578 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_8811_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_8811_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=KG35_rpt\n",
      "+ poretools_dir=KG35_rpt\n",
      "+ sample_tag=KG35_rpt_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG35_rpt_hq.fasta\n",
      "+ samtools sort - EM_079517_KG35_rpt_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2787 sequences (4361257 bp)...\n",
      "[M::mem_process_seqs] Processed 2787 reads in 21.172 CPU sec, 21.187 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG35_rpt_hq.fasta\n",
      "[main] Real time: 21.961 sec; CPU: 21.235 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG35_rpt_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG35_rpt_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=KG35_rpt\n",
      "+ poretools_dir=KG35_rpt\n",
      "+ sample_tag=KG35_rpt_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG35_rpt_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_KG35_rpt_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2787 sequences (4361257 bp)...\n",
      "[M::mem_process_seqs] Processed 2787 reads in 21.762 CPU sec, 21.771 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG35_rpt_hq.fasta\n",
      "[main] Real time: 22.507 sec; CPU: 21.799 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG35_rpt_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=018242\n",
      "+ poretools_dir=018242\n",
      "+ sample_tag=018242_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 018242_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_018242_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2845 sequences (3939807 bp)...\n",
      "[M::mem_process_seqs] Processed 2845 reads in 21.119 CPU sec, 21.129 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 018242_hq.fasta\n",
      "[main] Real time: 22.002 sec; CPU: 21.246 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_018242_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_018242_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017788\n",
      "+ poretools_dir=017788\n",
      "+ sample_tag=017788_hq\n",
      "+ second_batch=017788_2\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017788_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017788_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 5112 sequences (8390236 bp)...\n",
      "[M::mem_process_seqs] Processed 5112 reads in 30.451 CPU sec, 30.475 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017788_hq.fasta\n",
      "[main] Real time: 31.866 sec; CPU: 30.539 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017788_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017788_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=8559\n",
      "+ poretools_dir=8559\n",
      "+ sample_tag=8559_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 8559_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_8559_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3096 sequences (4839206 bp)...\n",
      "[M::mem_process_seqs] Processed 3096 reads in 24.953 CPU sec, 24.954 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 8559_hq.fasta\n",
      "[main] Real time: 25.701 sec; CPU: 25.030 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_8559_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_8559_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=018216\n",
      "+ poretools_dir=018216\n",
      "+ sample_tag=018216_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 018216_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_018216_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2678 sequences (4579991 bp)...\n",
      "[M::mem_process_seqs] Processed 2678 reads in 24.844 CPU sec, 24.850 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 018216_hq.fasta\n",
      "[main] Real time: 25.568 sec; CPU: 24.880 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_018216_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_018216_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=KG88b\n",
      "+ poretools_dir=KG88b\n",
      "+ sample_tag=KG88b_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG88b_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_KG88b_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2797 sequences (4668163 bp)...\n",
      "[M::mem_process_seqs] Processed 2797 reads in 23.990 CPU sec, 23.991 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG88b_hq.fasta\n",
      "[main] Real time: 24.845 sec; CPU: 24.035 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG88b_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG88b_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017608\n",
      "+ poretools_dir=017608\n",
      "+ sample_tag=017608_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017608_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017608_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3457 sequences (3494391 bp)...\n",
      "[M::mem_process_seqs] Processed 3457 reads in 12.651 CPU sec, 12.652 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017608_hq.fasta\n",
      "[main] Real time: 13.681 sec; CPU: 12.692 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017608_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017608_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017608\n",
      "+ poretools_dir=017608\n",
      "+ sample_tag=017608_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017608_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017608_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3457 sequences (3494391 bp)...\n",
      "[M::mem_process_seqs] Processed 3457 reads in 13.367 CPU sec, 13.367 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017608_hq.fasta\n",
      "[main] Real time: 14.599 sec; CPU: 13.498 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017608_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017608_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=29656b\n",
      "+ poretools_dir=29656b\n",
      "+ sample_tag=29656b_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools sort - EM_079517_29656b_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 29656b_hq.fasta\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3123 sequences (3378625 bp)...\n",
      "[M::mem_process_seqs] Processed 3123 reads in 13.039 CPU sec, 13.039 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 29656b_hq.fasta\n",
      "[main] Real time: 13.866 sec; CPU: 13.107 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_29656b_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_29656b_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC497\n",
      "+ poretools_dir=REDC497\n",
      "+ sample_tag=REDC497_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC497_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_REDC497_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2783 sequences (2731259 bp)...\n",
      "[M::mem_process_seqs] Processed 2783 reads in 11.178 CPU sec, 11.178 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC497_hq.fasta\n",
      "[main] Real time: 11.647 sec; CPU: 11.232 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC497_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC497_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC469b\n",
      "+ poretools_dir=REDC469b\n",
      "+ sample_tag=REDC469b_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC469b_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_REDC469b_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3220 sequences (3058654 bp)...\n",
      "[M::mem_process_seqs] Processed 3220 reads in 9.785 CPU sec, 9.792 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC469b_hq.fasta\n",
      "[main] Real time: 10.345 sec; CPU: 9.815 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC469b_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC469b_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC470\n",
      "+ poretools_dir=REDC470\n",
      "+ sample_tag=REDC470_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC470_hq.fasta\n",
      "+ samtools sort - EM_079517_REDC470_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2692 sequences (2663278 bp)...\n",
      "[M::mem_process_seqs] Processed 2692 reads in 8.553 CPU sec, 8.567 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC470_hq.fasta\n",
      "[main] Real time: 9.095 sec; CPU: 8.615 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC470_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC470_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC497\n",
      "+ poretools_dir=REDC497\n",
      "+ sample_tag=REDC497_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC497_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_REDC497_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2783 sequences (2731259 bp)...\n",
      "[M::mem_process_seqs] Processed 2783 reads in 10.988 CPU sec, 10.988 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC497_hq.fasta\n",
      "[main] Real time: 11.530 sec; CPU: 11.027 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC497_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC497_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=KG45\n",
      "+ poretools_dir=KG45\n",
      "+ sample_tag=KG45_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG45_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_KG45_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "[M::process] read 3195 sequences (6217790 bp)...\n",
      "[M::mem_process_seqs] Processed 3195 reads in 32.934 CPU sec, 32.934 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG45_hq.fasta\n",
      "[main] Real time: 34.128 sec; CPU: 33.102 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG45_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG45_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC502\n",
      "+ poretools_dir=REDC502\n",
      "+ sample_tag=REDC502_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC502_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_REDC502_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 1751 sequences (2928472 bp)...\n",
      "[M::mem_process_seqs] Processed 1751 reads in 12.460 CPU sec, 12.469 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC502_hq.fasta\n",
      "[main] Real time: 12.872 sec; CPU: 12.484 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC502_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC502_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=29656b\n",
      "+ poretools_dir=29656b\n",
      "+ sample_tag=29656b_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 29656b_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_29656b_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3123 sequences (3378625 bp)...\n",
      "[M::mem_process_seqs] Processed 3123 reads in 15.199 CPU sec, 15.224 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 29656b_hq.fasta\n",
      "[main] Real time: 15.846 sec; CPU: 15.228 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_29656b_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_29656b_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC469b\n",
      "+ poretools_dir=REDC469b\n",
      "+ sample_tag=REDC469b_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC469b_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_REDC469b_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3220 sequences (3058654 bp)...\n",
      "[M::mem_process_seqs] Processed 3220 reads in 11.468 CPU sec, 11.470 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC469b_hq.fasta\n",
      "[main] Real time: 12.157 sec; CPU: 11.541 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC469b_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC469b_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC502\n",
      "+ poretools_dir=REDC502\n",
      "+ sample_tag=REDC502_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC502_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_REDC502_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 1751 sequences (2928472 bp)...\n",
      "[M::mem_process_seqs] Processed 1751 reads in 13.780 CPU sec, 13.781 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC502_hq.fasta\n",
      "[main] Real time: 14.342 sec; CPU: 13.824 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC502_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC502_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC470\n",
      "+ poretools_dir=REDC470\n",
      "+ sample_tag=REDC470_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC470_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_REDC470_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "+ samtools view -bS -\n",
      "[M::process] read 2692 sequences (2663278 bp)...\n",
      "[M::mem_process_seqs] Processed 2692 reads in 10.823 CPU sec, 10.837 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC470_hq.fasta\n",
      "[main] Real time: 11.296 sec; CPU: 10.843 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC470_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC470_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=KG88b\n",
      "+ poretools_dir=KG88b\n",
      "+ sample_tag=KG88b_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG88b_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_KG88b_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2797 sequences (4668163 bp)...\n",
      "[M::mem_process_seqs] Processed 2797 reads in 28.277 CPU sec, 28.281 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG88b_hq.fasta\n",
      "[main] Real time: 29.236 sec; CPU: 28.330 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG88b_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG88b_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=022275\n",
      "+ poretools_dir=022275\n",
      "+ sample_tag=022275_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022275_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_022275_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3714 sequences (6008314 bp)...\n",
      "[M::mem_process_seqs] Processed 3714 reads in 25.774 CPU sec, 25.776 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022275_hq.fasta\n",
      "[main] Real time: 26.787 sec; CPU: 25.901 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022275_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022275_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=KG45\n",
      "+ poretools_dir=KG45\n",
      "+ sample_tag=KG45_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG45_hq.fasta\n",
      "+ samtools sort - EM_079517_KG45_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3195 sequences (6217790 bp)...\n",
      "[M::mem_process_seqs] Processed 3195 reads in 37.976 CPU sec, 37.976 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG45_hq.fasta\n",
      "[main] Real time: 39.839 sec; CPU: 38.168 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG45_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG45_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017586\n",
      "+ poretools_dir=017586\n",
      "+ sample_tag=017586_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017586_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_017586_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3061 sequences (3625434 bp)...\n",
      "[M::mem_process_seqs] Processed 3061 reads in 13.616 CPU sec, 13.642 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017586_hq.fasta\n",
      "[main] Real time: 14.413 sec; CPU: 13.692 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017586_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017586_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=022275\n",
      "+ poretools_dir=022275\n",
      "+ sample_tag=022275_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022275_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_022275_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3714 sequences (6008314 bp)...\n",
      "[M::mem_process_seqs] Processed 3714 reads in 26.912 CPU sec, 26.918 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022275_hq.fasta\n",
      "[main] Real time: 28.571 sec; CPU: 27.188 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022275_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022275_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017609\n",
      "+ poretools_dir=017609\n",
      "+ sample_tag=017609_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017609_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017609_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 4145 sequences (4149402 bp)...\n",
      "[M::mem_process_seqs] Processed 4145 reads in 17.421 CPU sec, 17.435 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017609_hq.fasta\n",
      "[main] Real time: 18.221 sec; CPU: 17.486 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017609_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017609_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017609\n",
      "+ poretools_dir=017609\n",
      "+ sample_tag=017609_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017609_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017609_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4145 sequences (4149402 bp)...\n",
      "[M::mem_process_seqs] Processed 4145 reads in 16.711 CPU sec, 16.721 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017609_hq.fasta\n",
      "[main] Real time: 17.536 sec; CPU: 16.763 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017609_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017609_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=KG80\n",
      "+ poretools_dir=KG80\n",
      "+ sample_tag=KG80_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG80_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_KG80_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "+ samtools view -bS -\n",
      "[M::process] read 3401 sequences (5650051 bp)...\n",
      "[M::mem_process_seqs] Processed 3401 reads in 25.626 CPU sec, 25.637 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG80_hq.fasta\n",
      "[main] Real time: 26.675 sec; CPU: 25.704 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG80_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG80_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017586\n",
      "+ poretools_dir=017586\n",
      "+ sample_tag=017586_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017586_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017586_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3061 sequences (3625434 bp)...\n",
      "[M::mem_process_seqs] Processed 3061 reads in 14.913 CPU sec, 14.928 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017586_hq.fasta\n",
      "[main] Real time: 15.460 sec; CPU: 14.947 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017586_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017586_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=022057\n",
      "+ poretools_dir=022057\n",
      "+ sample_tag=022057_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022057_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_022057_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3422 sequences (5677756 bp)...\n",
      "[M::mem_process_seqs] Processed 3422 reads in 25.107 CPU sec, 25.131 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022057_hq.fasta\n",
      "[main] Real time: 26.206 sec; CPU: 25.177 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022057_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022057_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017592\n",
      "+ poretools_dir=017592\n",
      "+ sample_tag=017592_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017592_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_017592_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4142 sequences (4267623 bp)...\n",
      "[M::mem_process_seqs] Processed 4142 reads in 15.517 CPU sec, 15.550 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017592_hq.fasta\n",
      "[main] Real time: 16.355 sec; CPU: 15.608 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017592_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017592_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC534\n",
      "+ poretools_dir=REDC534\n",
      "+ sample_tag=REDC534_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC534_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_REDC534_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 513 sequences (774641 bp)...\n",
      "[M::mem_process_seqs] Processed 513 reads in 3.483 CPU sec, 3.483 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC534_hq.fasta\n",
      "[main] Real time: 3.604 sec; CPU: 3.491 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC534_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC534_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC534\n",
      "+ poretools_dir=REDC534\n",
      "+ sample_tag=REDC534_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC534_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_REDC534_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 513 sequences (774641 bp)...\n",
      "[M::mem_process_seqs] Processed 513 reads in 4.059 CPU sec, 4.062 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC534_hq.fasta\n",
      "[main] Real time: 4.199 sec; CPU: 4.070 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC534_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC534_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC402\n",
      "+ poretools_dir=REDC402\n",
      "+ sample_tag=REDC402_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC402_hq.fasta\n",
      "+ samtools sort - EM_079517_REDC402_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "+ samtools view -bS -\n",
      "[M::process] read 3289 sequences (5779093 bp)...\n",
      "[M::mem_process_seqs] Processed 3289 reads in 26.452 CPU sec, 26.456 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC402_hq.fasta\n",
      "[main] Real time: 27.424 sec; CPU: 26.527 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC402_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC402_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC402\n",
      "+ poretools_dir=REDC402\n",
      "+ sample_tag=REDC402_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_REDC402_hq_bwa.sorted\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC402_hq.fasta\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3289 sequences (5779093 bp)...\n",
      "[M::mem_process_seqs] Processed 3289 reads in 24.433 CPU sec, 24.437 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC402_hq.fasta\n",
      "[main] Real time: 25.386 sec; CPU: 24.480 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC402_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC402_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=KG91\n",
      "+ poretools_dir=KG91\n",
      "+ sample_tag=KG91_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG91_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_KG91_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4164 sequences (6921539 bp)...\n",
      "[M::mem_process_seqs] Processed 4164 reads in 30.918 CPU sec, 30.924 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG91_hq.fasta\n",
      "[main] Real time: 31.957 sec; CPU: 31.027 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG91_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "fasta ../refs/EM_079517.fasta --bam EM_079517_KG91_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=KG87\n",
      "+ poretools_dir=KG87\n",
      "+ sample_tag=KG87_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG87_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_KG87_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4789 sequences (8113949 bp)...\n",
      "[M::mem_process_seqs] Processed 4789 reads in 37.108 CPU sec, 37.112 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG87_hq.fasta\n",
      "[main] Real time: 38.424 sec; CPU: 37.274 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG87_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG87_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017592\n",
      "+ poretools_dir=017592\n",
      "+ sample_tag=017592_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017592_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017592_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4142 sequences (4267623 bp)...\n",
      "[M::mem_process_seqs] Processed 4142 reads in 17.525 CPU sec, 17.550 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017592_hq.fasta\n",
      "[main] Real time: 18.232 sec; CPU: 17.584 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017592_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017592_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017787\n",
      "+ poretools_dir=017787\n",
      "+ sample_tag=017787_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017787_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017787_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2872 sequences (3009412 bp)...\n",
      "[M::mem_process_seqs] Processed 2872 reads in 9.432 CPU sec, 9.432 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017787_hq.fasta\n",
      "[main] Real time: 9.917 sec; CPU: 9.460 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017787_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017787_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=KG80\n",
      "+ poretools_dir=KG80\n",
      "+ sample_tag=KG80_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG80_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_KG80_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3401 sequences (5650051 bp)...\n",
      "[M::mem_process_seqs] Processed 3401 reads in 34.285 CPU sec, 34.286 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG80_hq.fasta\n",
      "[main] Real time: 35.376 sec; CPU: 34.502 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG80_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG80_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC483\n",
      "+ poretools_dir=REDC483\n",
      "+ sample_tag=REDC483_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC483_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_REDC483_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3249 sequences (5447142 bp)...\n",
      "[M::mem_process_seqs] Processed 3249 reads in 21.646 CPU sec, 21.657 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC483_hq.fasta\n",
      "[main] Real time: 22.468 sec; CPU: 21.706 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC483_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC483_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=KG87\n",
      "+ poretools_dir=KG87\n",
      "+ sample_tag=KG87_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG87_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_KG87_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4789 sequences (8113949 bp)...\n",
      "[M::mem_process_seqs] Processed 4789 reads in 38.281 CPU sec, 38.311 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG87_hq.fasta\n",
      "[main] Real time: 39.608 sec; CPU: 38.360 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG87_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG87_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC627\n",
      "+ poretools_dir=REDC627\n",
      "+ sample_tag=REDC627_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC627_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_REDC627_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 1083 sequences (1457313 bp)...\n",
      "[M::mem_process_seqs] Processed 1083 reads in 5.451 CPU sec, 5.451 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC627_hq.fasta\n",
      "[main] Real time: 5.649 sec; CPU: 5.471 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC627_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC627_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017883b\n",
      "+ poretools_dir=017883b\n",
      "+ sample_tag=017883b_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017883b_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017883b_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2410 sequences (2415913 bp)...\n",
      "[M::mem_process_seqs] Processed 2410 reads in 8.512 CPU sec, 8.521 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017883b_hq.fasta\n",
      "[main] Real time: 8.873 sec; CPU: 8.532 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017883b_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017883b_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=022017\n",
      "+ poretools_dir=022017\n",
      "+ sample_tag=022017_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022017_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_022017_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 1395 sequences (2184231 bp)...\n",
      "[M::mem_process_seqs] Processed 1395 reads in 11.176 CPU sec, 11.183 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022017_hq.fasta\n",
      "[main] Real time: 11.529 sec; CPU: 11.197 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022017_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022017_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=022057\n",
      "+ poretools_dir=022057\n",
      "+ sample_tag=022057_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022057_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_022057_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3422 sequences (5677756 bp)...\n",
      "[M::mem_process_seqs] Processed 3422 reads in 31.772 CPU sec, 31.774 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022057_hq.fasta\n",
      "[main] Real time: 32.570 sec; CPU: 31.815 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022057_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022057_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=KG91\n",
      "+ poretools_dir=KG91\n",
      "+ sample_tag=KG91_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG91_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_KG91_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4164 sequences (6921539 bp)...\n",
      "[M::mem_process_seqs] Processed 4164 reads in 34.710 CPU sec, 34.722 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG91_hq.fasta\n",
      "[main] Real time: 35.720 sec; CPU: 34.796 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG91_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG91_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC627\n",
      "+ poretools_dir=REDC627\n",
      "+ sample_tag=REDC627_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC627_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_REDC627_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 1083 sequences (1457313 bp)...\n",
      "[M::mem_process_seqs] Processed 1083 reads in 7.676 CPU sec, 7.679 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC627_hq.fasta\n",
      "[main] Real time: 7.892 sec; CPU: 7.693 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC627_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC627_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FCH549\n",
      "+ poretools_dir=FCH549\n",
      "+ sample_tag=FCH549_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH549_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_FCH549_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 242 sequences (397197 bp)...\n",
      "[M::mem_process_seqs] Processed 242 reads in 1.432 CPU sec, 1.432 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH549_hq.fasta\n",
      "[main] Real time: 1.501 sec; CPU: 1.437 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH549_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH549_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC483\n",
      "+ poretools_dir=REDC483\n",
      "+ sample_tag=REDC483_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC483_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_REDC483_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3249 sequences (5447142 bp)...\n",
      "[M::mem_process_seqs] Processed 3249 reads in 25.376 CPU sec, 25.381 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC483_hq.fasta\n",
      "[main] Real time: 26.194 sec; CPU: 25.494 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC483_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC483_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017787\n",
      "+ poretools_dir=017787\n",
      "+ sample_tag=017787_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017787_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017787_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2872 sequences (3009412 bp)...\n",
      "[M::mem_process_seqs] Processed 2872 reads in 12.656 CPU sec, 12.664 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017787_hq.fasta\n",
      "[main] Real time: 13.145 sec; CPU: 12.688 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017787_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017787_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FCH549\n",
      "+ poretools_dir=FCH549\n",
      "+ sample_tag=FCH549_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH549_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_FCH549_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 242 sequences (397197 bp)...\n",
      "[M::mem_process_seqs] Processed 242 reads in 2.494 CPU sec, 2.494 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH549_hq.fasta\n",
      "[main] Real time: 2.552 sec; CPU: 2.500 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH549_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH549_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017883b\n",
      "+ poretools_dir=017883b\n",
      "+ sample_tag=017883b_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017883b_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017883b_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2410 sequences (2415913 bp)...\n",
      "[M::mem_process_seqs] Processed 2410 reads in 11.762 CPU sec, 11.769 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017883b_hq.fasta\n",
      "[main] Real time: 12.264 sec; CPU: 11.806 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017883b_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017883b_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=022017\n",
      "+ poretools_dir=022017\n",
      "+ sample_tag=022017_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022017_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_022017_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 1395 sequences (2184231 bp)...\n",
      "[M::mem_process_seqs] Processed 1395 reads in 13.493 CPU sec, 13.497 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022017_hq.fasta\n",
      "[main] Real time: 13.924 sec; CPU: 13.517 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022017_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022017_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017589\n",
      "+ poretools_dir=017589\n",
      "+ sample_tag=017589_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017589_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017589_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3319 sequences (4093288 bp)...\n",
      "[M::mem_process_seqs] Processed 3319 reads in 14.305 CPU sec, 14.307 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017589_hq.fasta\n",
      "[main] Real time: 14.974 sec; CPU: 14.336 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017589_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017589_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=022390\n",
      "+ poretools_dir=022390\n",
      "+ sample_tag=022390_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022390_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_022390_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3552 sequences (5899421 bp)...\n",
      "[M::mem_process_seqs] Processed 3552 reads in 25.116 CPU sec, 25.118 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022390_hq.fasta\n",
      "[main] Real time: 26.098 sec; CPU: 25.193 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022390_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022390_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=017595\n",
      "+ poretools_dir=017595\n",
      "+ sample_tag=017595_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017595_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_017595_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3738 sequences (3786969 bp)...\n",
      "[M::mem_process_seqs] Processed 3738 reads in 16.754 CPU sec, 16.767 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 017595_hq.fasta\n",
      "[main] Real time: 17.272 sec; CPU: 16.787 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017595_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_017595_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC574\n",
      "+ poretools_dir=REDC574\n",
      "+ sample_tag=REDC574_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC574_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_REDC574_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 1141 sequences (1900300 bp)...\n",
      "[M::mem_process_seqs] Processed 1141 reads in 8.916 CPU sec, 8.924 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC574_hq.fasta\n",
      "[main] Real time: 9.300 sec; CPU: 8.935 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC574_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC574_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC574\n",
      "+ poretools_dir=REDC574\n",
      "+ sample_tag=REDC574_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC574_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "[M::process] read 1141 sequences (1900300 bp)...\n",
      "+ samtools sort - EM_079517_mut30_2_REDC574_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::mem_process_seqs] Processed 1141 reads in 9.289 CPU sec, 9.295 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC574_hq.fasta\n",
      "[main] Real time: 9.660 sec; CPU: 9.306 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC574_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC574_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017595\n",
      "+ poretools_dir=017595\n",
      "+ sample_tag=017595_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017595_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017595_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3738 sequences (3786969 bp)...\n",
      "[M::mem_process_seqs] Processed 3738 reads in 17.740 CPU sec, 17.743 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017595_hq.fasta\n",
      "[main] Real time: 18.438 sec; CPU: 17.801 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017595_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017595_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=022390\n",
      "+ poretools_dir=022390\n",
      "+ sample_tag=022390_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022390_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_022390_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3552 sequences (5899421 bp)...\n",
      "[M::mem_process_seqs] Processed 3552 reads in 28.962 CPU sec, 28.964 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022390_hq.fasta\n",
      "[main] Real time: 29.992 sec; CPU: 29.078 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022390_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022390_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=017589\n",
      "+ poretools_dir=017589\n",
      "+ sample_tag=017589_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017589_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_017589_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3319 sequences (4093288 bp)...\n",
      "[M::mem_process_seqs] Processed 3319 reads in 18.876 CPU sec, 18.877 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 017589_hq.fasta\n",
      "[main] Real time: 19.511 sec; CPU: 18.952 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017589_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_017589_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FORE1415\n",
      "+ poretools_dir=FORE1415\n",
      "+ sample_tag=FORE1415_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE1415_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FORE1415_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3693 sequences (3793200 bp)...\n",
      "[M::mem_process_seqs] Processed 3693 reads in 12.641 CPU sec, 12.641 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE1415_hq.fasta\n",
      "[main] Real time: 13.333 sec; CPU: 12.677 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE1415_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE1415_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FORE1415\n",
      "+ poretools_dir=FORE1415\n",
      "+ sample_tag=FORE1415_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE1415_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_FORE1415_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3693 sequences (3793200 bp)...\n",
      "[M::mem_process_seqs] Processed 3693 reads in 14.196 CPU sec, 14.197 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE1415_hq.fasta\n",
      "[main] Real time: 14.890 sec; CPU: 14.239 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE1415_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE1415_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=20248\n",
      "+ poretools_dir=20248\n",
      "+ sample_tag=20248_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 20248_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_20248_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4088 sequences (6054706 bp)...\n",
      "[M::mem_process_seqs] Processed 4088 reads in 29.487 CPU sec, 29.497 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 20248_hq.fasta\n",
      "[main] Real time: 30.445 sec; CPU: 29.612 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_20248_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=20248\n",
      "+ poretools_dir=20248\n",
      "+ sample_tag=20248_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 20248_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_20248_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 4088 sequences (6054706 bp)...\n",
      "[M::mem_process_seqs] Processed 4088 reads in 30.572 CPU sec, 30.575 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 20248_hq.fasta\n",
      "[main] Real time: 31.460 sec; CPU: 30.729 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_20248_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_20248_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=1160\n",
      "+ poretools_dir=1160\n",
      "+ sample_tag=1160_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 1160_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_1160_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "+ samtools view -bS -\n",
      "[M::process] read 3913 sequences (3790169 bp)...\n",
      "[M::mem_process_seqs] Processed 3913 reads in 14.587 CPU sec, 14.586 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 1160_hq.fasta\n",
      "[main] Real time: 15.245 sec; CPU: 14.618 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_1160_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_1160_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=1160\n",
      "+ poretools_dir=1160\n",
      "+ sample_tag=1160_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 1160_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_1160_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3913 sequences (3790169 bp)...\n",
      "[M::mem_process_seqs] Processed 3913 reads in 15.477 CPU sec, 15.477 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 1160_hq.fasta\n",
      "[main] Real time: 16.108 sec; CPU: 15.511 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_1160_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_1160_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=022019\n",
      "+ poretools_dir=022019\n",
      "+ sample_tag=022019_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022019_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_022019_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3064 sequences (5138631 bp)...\n",
      "[M::mem_process_seqs] Processed 3064 reads in 24.260 CPU sec, 24.280 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 022019_hq.fasta\n",
      "[main] Real time: 25.680 sec; CPU: 24.316 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022019_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_022019_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=022019\n",
      "+ poretools_dir=022019\n",
      "+ sample_tag=022019_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022019_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_022019_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3064 sequences (5138631 bp)...\n",
      "[M::mem_process_seqs] Processed 3064 reads in 25.926 CPU sec, 25.939 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 022019_hq.fasta\n",
      "[main] Real time: 27.270 sec; CPU: 26.054 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022019_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_022019_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FORE1789\n",
      "+ poretools_dir=FORE1789\n",
      "+ sample_tag=FORE1789_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE1789_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_FORE1789_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3924 sequences (3856528 bp)...\n",
      "[M::mem_process_seqs] Processed 3924 reads in 13.353 CPU sec, 13.353 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE1789_hq.fasta\n",
      "[main] Real time: 14.325 sec; CPU: 13.464 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE1789_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE1789_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FCH547\n",
      "+ poretools_dir=FCH547\n",
      "+ sample_tag=FCH547_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH547_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FCH547_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 4080 sequences (4335270 bp)...\n",
      "[M::mem_process_seqs] Processed 4080 reads in 14.656 CPU sec, 14.656 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH547_hq.fasta\n",
      "[main] Real time: 15.509 sec; CPU: 14.688 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH547_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH547_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FORE1789\n",
      "+ poretools_dir=FORE1789\n",
      "+ sample_tag=FORE1789_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE1789_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FORE1789_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3924 sequences (3856528 bp)...\n",
      "[M::mem_process_seqs] Processed 3924 reads in 14.115 CPU sec, 14.121 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE1789_hq.fasta\n",
      "[main] Real time: 14.877 sec; CPU: 14.152 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE1789_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE1789_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FCH547\n",
      "+ poretools_dir=FCH547\n",
      "+ sample_tag=FCH547_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH547_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_FCH547_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4080 sequences (4335270 bp)...\n",
      "[M::mem_process_seqs] Processed 4080 reads in 18.358 CPU sec, 18.359 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH547_hq.fasta\n",
      "[main] Real time: 19.153 sec; CPU: 18.418 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH547_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH547_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=023888\n",
      "+ poretools_dir=023888\n",
      "+ sample_tag=023888_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 023888_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_023888_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 2572 sequences (2445188 bp)...\n",
      "[M::mem_process_seqs] Processed 2572 reads in 8.697 CPU sec, 8.698 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 023888_hq.fasta\n",
      "[main] Real time: 9.099 sec; CPU: 8.724 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_023888_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_023888_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FCH542\n",
      "+ poretools_dir=FCH542\n",
      "+ sample_tag=FCH542_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH542_hq.fasta\n",
      "+ samtools sort - EM_079517_FCH542_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3302 sequences (4996494 bp)...\n",
      "[M::mem_process_seqs] Processed 3302 reads in 23.907 CPU sec, 23.908 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH542_hq.fasta\n",
      "[main] Real time: 24.989 sec; CPU: 23.947 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH542_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH542_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=023888\n",
      "+ poretools_dir=023888\n",
      "+ sample_tag=023888_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 023888_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_023888_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 2572 sequences (2445188 bp)...\n",
      "[M::mem_process_seqs] Processed 2572 reads in 9.523 CPU sec, 9.523 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 023888_hq.fasta\n",
      "[main] Real time: 10.008 sec; CPU: 9.554 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_023888_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_023888_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FCH542\n",
      "+ poretools_dir=FCH542\n",
      "+ sample_tag=FCH542_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH542_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FCH542_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3302 sequences (4996494 bp)...\n",
      "[M::mem_process_seqs] Processed 3302 reads in 24.968 CPU sec, 24.972 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH542_hq.fasta\n",
      "[main] Real time: 25.703 sec; CPU: 25.012 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH542_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH542_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FCH1019\n",
      "+ poretools_dir=FCH1019\n",
      "+ sample_tag=FCH1019_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH1019_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_FCH1019_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3883 sequences (5424906 bp)...\n",
      "[M::mem_process_seqs] Processed 3883 reads in 22.816 CPU sec, 22.817 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH1019_hq.fasta\n",
      "[main] Real time: 24.054 sec; CPU: 22.956 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH1019_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH1019_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC589\n",
      "+ poretools_dir=REDC589\n",
      "+ sample_tag=REDC589_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC589_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_REDC589_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 5086 sequences (7627133 bp)...\n",
      "[M::mem_process_seqs] Processed 5086 reads in 32.682 CPU sec, 32.686 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC589_hq.fasta\n",
      "[main] Real time: 34.177 sec; CPU: 32.770 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC589_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC589_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC545\n",
      "+ poretools_dir=REDC545\n",
      "+ sample_tag=REDC545_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_REDC545_hq_bwa.sorted\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC545_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4856 sequences (7507421 bp)...\n",
      "[M::mem_process_seqs] Processed 4856 reads in 32.037 CPU sec, 32.042 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC545_hq.fasta\n",
      "[main] Real time: 33.676 sec; CPU: 32.146 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC545_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FCH1019\n",
      "+ poretools_dir=FCH1019\n",
      "+ sample_tag=FCH1019_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH1019_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FCH1019_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3883 sequences (5424906 bp)...\n",
      "[M::mem_process_seqs] Processed 3883 reads in 25.543 CPU sec, 25.548 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH1019_hq.fasta\n",
      "[main] Real time: 26.307 sec; CPU: 25.584 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH1019_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH1019_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC545\n",
      "+ poretools_dir=REDC545\n",
      "+ sample_tag=REDC545_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC545_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_REDC545_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4856 sequences (7507421 bp)...\n",
      "[M::mem_process_seqs] Processed 4856 reads in 34.542 CPU sec, 34.574 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC545_hq.fasta\n",
      "[main] Real time: 35.833 sec; CPU: 34.603 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC545_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC589\n",
      "+ poretools_dir=REDC589\n",
      "+ sample_tag=REDC589_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC589_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_REDC589_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 5086 sequences (7627133 bp)...\n",
      "[M::mem_process_seqs] Processed 5086 reads in 35.468 CPU sec, 35.503 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC589_hq.fasta\n",
      "[main] Real time: 36.852 sec; CPU: 35.532 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC589_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC589_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FORE838\n",
      "+ poretools_dir=FORE838\n",
      "+ sample_tag=FORE838_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE838_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FORE838_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4553 sequences (6568886 bp)...\n",
      "[M::mem_process_seqs] Processed 4553 reads in 31.338 CPU sec, 31.391 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE838_hq.fasta\n",
      "[main] Real time: 32.768 sec; CPU: 31.407 sec\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE838_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FORE838\n",
      "+ poretools_dir=FORE838\n",
      "+ sample_tag=FORE838_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE838_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_FORE838_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4553 sequences (6568886 bp)...\n",
      "[M::mem_process_seqs] Processed 4553 reads in 34.567 CPU sec, 34.567 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE838_hq.fasta\n",
      "[main] Real time: 36.313 sec; CPU: 34.652 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE838_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE838_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FCH_732\n",
      "+ poretools_dir=FCH_732\n",
      "+ sample_tag=FCH_732_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH_732_hq.fasta\n",
      "+ samtools sort - EM_079517_FCH_732_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 1777 sequences (1477083 bp)...\n",
      "[M::mem_process_seqs] Processed 1777 reads in 4.703 CPU sec, 4.704 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH_732_hq.fasta\n",
      "[main] Real time: 5.103 sec; CPU: 4.722 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH_732_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH_732_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FCH_732\n",
      "+ poretools_dir=FCH_732\n",
      "+ sample_tag=FCH_732_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH_732_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FCH_732_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 1777 sequences (1477083 bp)...\n",
      "[M::mem_process_seqs] Processed 1777 reads in 5.852 CPU sec, 5.854 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH_732_hq.fasta\n",
      "[main] Real time: 6.115 sec; CPU: 5.865 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH_732_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH_732_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=023888_B\n",
      "+ poretools_dir=023888_B\n",
      "+ sample_tag=023888_B_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 023888_B_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_023888_B_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 7665 sequences (6484388 bp)...\n",
      "[M::mem_process_seqs] Processed 7665 reads in 15.028 CPU sec, 15.036 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 023888_B_hq.fasta\n",
      "[main] Real time: 16.413 sec; CPU: 15.096 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_023888_B_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_023888_B_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=CON11010\n",
      "+ poretools_dir=CON11010\n",
      "+ sample_tag=CON11010_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta CON11010_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_CON11010_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4246 sequences (6888738 bp)...\n",
      "[M::mem_process_seqs] Processed 4246 reads in 30.951 CPU sec, 30.981 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta CON11010_hq.fasta\n",
      "[main] Real time: 31.828 sec; CPU: 31.001 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_CON11010_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_CON11010_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FCH_729\n",
      "+ poretools_dir=FCH_729\n",
      "+ sample_tag=FCH_729_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH_729_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_FCH_729_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3890 sequences (3672398 bp)...\n",
      "[M::mem_process_seqs] Processed 3890 reads in 12.811 CPU sec, 12.836 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH_729_hq.fasta\n",
      "[main] Real time: 13.761 sec; CPU: 12.903 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH_729_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH_729_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=023888_B\n",
      "+ poretools_dir=023888_B\n",
      "+ sample_tag=023888_B_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 023888_B_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_023888_B_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 7665 sequences (6484388 bp)...\n",
      "[M::mem_process_seqs] Processed 7665 reads in 17.875 CPU sec, 17.878 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 023888_B_hq.fasta\n",
      "[main] Real time: 18.913 sec; CPU: 17.934 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_023888_B_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_023888_B_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC494B\n",
      "+ poretools_dir=REDC494B\n",
      "+ sample_tag=REDC494B_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_REDC494B_hq_bwa.sorted\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC494B_hq.fasta\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 5656 sequences (10002168 bp)...\n",
      "[M::process] read 345 sequences (612724 bp)...\n",
      "[M::mem_process_seqs] Processed 5656 reads in 46.151 CPU sec, 46.158 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 345 reads in 3.651 CPU sec, 3.566 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC494B_hq.fasta\n",
      "[main] Real time: 49.852 sec; CPU: 49.858 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC494B_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC494B_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FORE548\n",
      "+ poretools_dir=FORE548\n",
      "+ sample_tag=FORE548_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE548_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FORE548_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5744 sequences (9057709 bp)...\n",
      "[M::mem_process_seqs] Processed 5744 reads in 40.719 CPU sec, 40.738 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE548_hq.fasta\n",
      "[main] Real time: 42.098 sec; CPU: 40.792 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE548_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE548_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FCH_729\n",
      "+ poretools_dir=FCH_729\n",
      "+ sample_tag=FCH_729_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FCH_729_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH_729_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3890 sequences (3672398 bp)...\n",
      "[M::mem_process_seqs] Processed 3890 reads in 13.505 CPU sec, 13.506 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH_729_hq.fasta\n",
      "[main] Real time: 14.444 sec; CPU: 13.551 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH_729_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH_729_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=023747\n",
      "+ poretools_dir=023747\n",
      "+ sample_tag=023747_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 023747_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_023747_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 7039 sequences (8078422 bp)...\n",
      "[M::mem_process_seqs] Processed 7039 reads in 30.662 CPU sec, 30.668 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 023747_hq.fasta\n",
      "[main] Real time: 31.987 sec; CPU: 30.738 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_023747_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_023747_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FCH_731\n",
      "+ poretools_dir=FCH_731\n",
      "+ sample_tag=FCH_731_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH_731_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_FCH_731_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 3027 sequences (3025034 bp)...\n",
      "[M::mem_process_seqs] Processed 3027 reads in 14.504 CPU sec, 14.507 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH_731_hq.fasta\n",
      "[main] Real time: 15.060 sec; CPU: 14.540 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH_731_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "fasta ../refs/EM_079517.fasta --bam EM_079517_FCH_731_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FORE548\n",
      "+ poretools_dir=FORE548\n",
      "+ sample_tag=FORE548_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE548_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_FORE548_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5744 sequences (9057709 bp)...\n",
      "[M::mem_process_seqs] Processed 5744 reads in 43.249 CPU sec, 43.251 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE548_hq.fasta\n",
      "[main] Real time: 44.900 sec; CPU: 43.351 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE548_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "fasta ../refs/EM_079517.fasta --bam EM_079517_FORE548_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FCH_731\n",
      "+ poretools_dir=FCH_731\n",
      "+ sample_tag=FCH_731_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH_731_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FCH_731_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 3027 sequences (3025034 bp)...\n",
      "[M::mem_process_seqs] Processed 3027 reads in 15.655 CPU sec, 15.655 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH_731_hq.fasta\n",
      "[main] Real time: 16.319 sec; CPU: 15.726 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH_731_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH_731_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=023747\n",
      "+ poretools_dir=023747\n",
      "+ sample_tag=023747_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 023747_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_023747_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 7039 sequences (8078422 bp)...\n",
      "[M::mem_process_seqs] Processed 7039 reads in 33.040 CPU sec, 33.044 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 023747_hq.fasta\n",
      "[main] Real time: 34.901 sec; CPU: 33.298 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_023747_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_023747_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=KG168\n",
      "+ poretools_dir=KG168\n",
      "+ sample_tag=KG168_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG168_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_KG168_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 7286 sequences (8165765 bp)...\n",
      "[M::mem_process_seqs] Processed 7286 reads in 28.747 CPU sec, 28.759 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG168_hq.fasta\n",
      "[main] Real time: 30.585 sec; CPU: 28.844 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG168_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG168_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FORE002212\n",
      "+ poretools_dir=FORE002212\n",
      "+ sample_tag=FORE002212_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE002212_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FORE002212_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5873 sequences (9640900 bp)...\n",
      "[M::mem_process_seqs] Processed 5873 reads in 38.414 CPU sec, 38.443 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE002212_hq.fasta\n",
      "[main] Real time: 40.739 sec; CPU: 38.515 sec\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE002212_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=CON11010\n",
      "+ poretools_dir=CON11010\n",
      "+ sample_tag=CON11010_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta CON11010_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_CON11010_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4246 sequences (6888738 bp)...\n",
      "[M::mem_process_seqs] Processed 4246 reads in 38.109 CPU sec, 38.126 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta CON11010_hq.fasta\n",
      "[main] Real time: 39.833 sec; CPU: 38.580 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_CON11010_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_CON11010_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FORE002212\n",
      "+ poretools_dir=FORE002212\n",
      "+ sample_tag=FORE002212_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE002212_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_FORE002212_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5873 sequences (9640900 bp)...\n",
      "[M::mem_process_seqs] Processed 5873 reads in 42.090 CPU sec, 42.104 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE002212_hq.fasta\n",
      "[main] Real time: 44.205 sec; CPU: 42.233 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE002212_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE002212_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=027317\n",
      "+ poretools_dir=027317\n",
      "+ sample_tag=027317_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 027317_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_027317_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5973 sequences (4914541 bp)...\n",
      "[M::mem_process_seqs] Processed 5973 reads in 16.497 CPU sec, 16.497 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 027317_hq.fasta\n",
      "[main] Real time: 17.074 sec; CPU: 16.535 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_027317_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_027317_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=KG168\n",
      "+ poretools_dir=KG168\n",
      "+ sample_tag=KG168_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG168_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_KG168_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 7286 sequences (8165765 bp)...\n",
      "[M::mem_process_seqs] Processed 7286 reads in 34.323 CPU sec, 34.330 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG168_hq.fasta\n",
      "[main] Real time: 35.634 sec; CPU: 34.404 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG168_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG168_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=027317\n",
      "+ poretools_dir=027317\n",
      "+ sample_tag=027317_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 027317_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_027317_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5973 sequences (4914541 bp)...\n",
      "[M::mem_process_seqs] Processed 5973 reads in 19.203 CPU sec, 19.218 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 027317_hq.fasta\n",
      "[main] Real time: 20.010 sec; CPU: 19.277 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_027317_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_027317_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FCH1785_S\n",
      "+ poretools_dir=FCH1785_S\n",
      "+ sample_tag=FCH1785_S_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH1785_S_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_FCH1785_S_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4417 sequences (7131062 bp)...\n",
      "[M::mem_process_seqs] Processed 4417 reads in 30.637 CPU sec, 30.651 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH1785_S_hq.fasta\n",
      "[main] Real time: 31.865 sec; CPU: 30.840 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH1785_S_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH1785_S_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=KG167\n",
      "+ poretools_dir=KG167\n",
      "+ sample_tag=KG167_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG167_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_KG167_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 7821 sequences (8904594 bp)...\n",
      "[M::mem_process_seqs] Processed 7821 reads in 40.389 CPU sec, 40.394 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta KG167_hq.fasta\n",
      "[main] Real time: 41.743 sec; CPU: 40.507 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG167_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_KG167_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=IPD10193\n",
      "+ poretools_dir=IPD10193\n",
      "+ sample_tag=IPD10193_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta IPD10193_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_IPD10193_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 6330 sequences (10000214 bp)...\n",
      "[M::process] read 921 sequences (1531380 bp)...\n",
      "[M::mem_process_seqs] Processed 6330 reads in 39.714 CPU sec, 39.698 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 921 reads in 8.817 CPU sec, 8.780 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta IPD10193_hq.fasta\n",
      "[main] Real time: 48.799 sec; CPU: 48.601 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_IPD10193_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_IPD10193_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=CON10786\n",
      "+ poretools_dir=CON10786\n",
      "+ sample_tag=CON10786_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta CON10786_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_CON10786_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5160 sequences (8370708 bp)...\n",
      "[M::mem_process_seqs] Processed 5160 reads in 40.947 CPU sec, 40.957 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta CON10786_hq.fasta\n",
      "[main] Real time: 42.304 sec; CPU: 41.025 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_CON10786_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_CON10786_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC494B\n",
      "+ poretools_dir=REDC494B\n",
      "+ sample_tag=REDC494B_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC494B_hq.fasta\n",
      "+ samtools sort - EM_079517_REDC494B_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5656 sequences (10002168 bp)...\n",
      "[M::process] read 345 sequences (612724 bp)...\n",
      "[M::mem_process_seqs] Processed 5656 reads in 57.486 CPU sec, 57.495 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 345 reads in 4.016 CPU sec, 3.968 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC494B_hq.fasta\n",
      "[main] Real time: 61.639 sec; CPU: 61.566 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC494B_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC494B_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=CON10786\n",
      "+ poretools_dir=CON10786\n",
      "+ sample_tag=CON10786_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta CON10786_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_CON10786_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5160 sequences (8370708 bp)...\n",
      "[M::mem_process_seqs] Processed 5160 reads in 41.098 CPU sec, 41.120 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta CON10786_hq.fasta\n",
      "[main] Real time: 42.440 sec; CPU: 41.189 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_CON10786_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_CON10786_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=KG167\n",
      "+ poretools_dir=KG167\n",
      "+ sample_tag=KG167_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG167_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_KG167_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 7821 sequences (8904594 bp)...\n",
      "[M::mem_process_seqs] Processed 7821 reads in 40.182 CPU sec, 40.249 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta KG167_hq.fasta\n",
      "[main] Real time: 41.553 sec; CPU: 40.286 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG167_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_KG167_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=IPD10193\n",
      "+ poretools_dir=IPD10193\n",
      "+ sample_tag=IPD10193_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta IPD10193_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_IPD10193_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 6330 sequences (10000214 bp)...\n",
      "[M::process] read 921 sequences (1531380 bp)...\n",
      "[M::mem_process_seqs] Processed 6330 reads in 44.744 CPU sec, 44.713 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 921 reads in 8.522 CPU sec, 8.465 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta IPD10193_hq.fasta\n",
      "[main] Real time: 53.576 sec; CPU: 53.339 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_IPD10193_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_IPD10193_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FCH1785_S\n",
      "+ poretools_dir=FCH1785_S\n",
      "+ sample_tag=FCH1785_S_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH1785_S_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_FCH1785_S_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4417 sequences (7131062 bp)...\n",
      "[M::mem_process_seqs] Processed 4417 reads in 36.007 CPU sec, 36.015 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH1785_S_hq.fasta\n",
      "[main] Real time: 37.084 sec; CPU: 36.276 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH1785_S_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH1785_S_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=CON10590\n",
      "+ poretools_dir=CON10590\n",
      "+ sample_tag=CON10590_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta CON10590_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_CON10590_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 6109 sequences (9943270 bp)...\n",
      "[M::mem_process_seqs] Processed 6109 reads in 50.254 CPU sec, 50.257 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta CON10590_hq.fasta\n",
      "[main] Real time: 52.072 sec; CPU: 50.374 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_CON10590_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_CON10590_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=CON10590\n",
      "+ poretools_dir=CON10590\n",
      "+ sample_tag=CON10590_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta CON10590_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_CON10590_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 6109 sequences (9943270 bp)...\n",
      "[M::mem_process_seqs] Processed 6109 reads in 51.635 CPU sec, 51.655 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta CON10590_hq.fasta\n",
      "[main] Real time: 53.294 sec; CPU: 51.733 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_CON10590_hq_bwa.sorted.bam\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_CON10590_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC_01285\n",
      "+ poretools_dir=REDC_01285\n",
      "+ sample_tag=REDC_01285_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC_01285_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_REDC_01285_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 6026 sequences (10001001 bp)...\n",
      "[M::process] read 1224 sequences (2017982 bp)...\n",
      "[M::mem_process_seqs] Processed 6026 reads in 46.117 CPU sec, 46.131 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 1224 reads in 12.741 CPU sec, 12.723 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC_01285_hq.fasta\n",
      "[main] Real time: 59.250 sec; CPU: 58.919 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC_01285_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC_01285_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FORE_2878\n",
      "+ poretools_dir=FORE_2878\n",
      "+ sample_tag=FORE_2878_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE_2878_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_FORE_2878_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4572 sequences (6844529 bp)...\n",
      "[M::mem_process_seqs] Processed 4572 reads in 28.345 CPU sec, 28.389 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE_2878_hq.fasta\n",
      "[main] Real time: 29.452 sec; CPU: 28.433 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE_2878_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE_2878_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC_01285\n",
      "+ poretools_dir=REDC_01285\n",
      "+ sample_tag=REDC_01285_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC_01285_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_REDC_01285_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 6026 sequences (10001001 bp)...\n",
      "[M::process] read 1224 sequences (2017982 bp)...\n",
      "[M::mem_process_seqs] Processed 6026 reads in 53.365 CPU sec, 53.373 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 1224 reads in 8.886 CPU sec, 8.849 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC_01285_hq.fasta\n",
      "[main] Real time: 62.628 sec; CPU: 62.320 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC_01285_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "fasta ../refs/EM_079517.fasta --bam EM_079517_REDC_01285_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FORE_2878\n",
      "+ poretools_dir=FORE_2878\n",
      "+ sample_tag=FORE_2878_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE_2878_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_FORE_2878_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 4572 sequences (6844529 bp)...\n",
      "[M::mem_process_seqs] Processed 4572 reads in 28.847 CPU sec, 28.858 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE_2878_hq.fasta\n",
      "[main] Real time: 29.924 sec; CPU: 28.981 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE_2878_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE_2878_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC_01165\n",
      "+ poretools_dir=REDC_01165\n",
      "+ sample_tag=REDC_01165_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC_01165_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_REDC_01165_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 7828 sequences (8844247 bp)...\n",
      "[M::mem_process_seqs] Processed 7828 reads in 19.040 CPU sec, 19.056 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC_01165_hq.fasta\n",
      "[main] Real time: 19.942 sec; CPU: 19.107 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC_01165_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC_01165_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC_01165\n",
      "+ poretools_dir=REDC_01165\n",
      "+ sample_tag=REDC_01165_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC_01165_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_REDC_01165_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 7828 sequences (8844247 bp)...\n",
      "[M::mem_process_seqs] Processed 7828 reads in 20.586 CPU sec, 20.587 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC_01165_hq.fasta\n",
      "[main] Real time: 21.803 sec; CPU: 20.733 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC_01165_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC_01165_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=CON10512\n",
      "+ poretools_dir=CON10512\n",
      "+ sample_tag=CON10512_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta CON10512_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_CON10512_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 6088 sequences (10001802 bp)...\n",
      "[M::process] read 989 sequences (1559866 bp)...\n",
      "[M::mem_process_seqs] Processed 6088 reads in 53.895 CPU sec, 53.897 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 989 reads in 7.064 CPU sec, 7.021 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta CON10512_hq.fasta\n",
      "[main] Real time: 61.296 sec; CPU: 61.030 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_CON10512_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_CON10512_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC_01323\n",
      "+ poretools_dir=REDC_01323\n",
      "+ sample_tag=REDC_01323_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC_01323_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_REDC_01323_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 7969 sequences (7807123 bp)...\n",
      "[M::mem_process_seqs] Processed 7969 reads in 23.577 CPU sec, 23.598 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC_01323_hq.fasta\n",
      "[main] Real time: 24.617 sec; CPU: 23.728 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC_01323_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC_01323_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC_01323\n",
      "+ poretools_dir=REDC_01323\n",
      "+ sample_tag=REDC_01323_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC_01323_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_REDC_01323_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 7969 sequences (7807123 bp)...\n",
      "[M::mem_process_seqs] Processed 7969 reads in 23.597 CPU sec, 23.604 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC_01323_hq.fasta\n",
      "[main] Real time: 24.635 sec; CPU: 23.663 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC_01323_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=REDC_01408\n",
      "+ poretools_dir=REDC_01408\n",
      "+ sample_tag=REDC_01408_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC_01408_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_REDC_01408_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 8302 sequences (9724116 bp)...\n",
      "[M::mem_process_seqs] Processed 8302 reads in 26.987 CPU sec, 26.993 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta REDC_01408_hq.fasta\n",
      "[main] Real time: 28.119 sec; CPU: 27.109 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC_01408_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_REDC_01408_hq_marginalign.sorted.bam\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FORE_2781_RX19_FLOWCELL2\n",
      "+ poretools_dir=FORE_2781_RX19_FLOWCELL2\n",
      "+ sample_tag=FORE_2781_RX19_FLOWCELL2_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE_2781_RX19_FLOWCELL2_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_FORE_2781_RX19_FLOWCELL2_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 7955 sequences (8581748 bp)...\n",
      "[M::mem_process_seqs] Processed 7955 reads in 27.523 CPU sec, 27.524 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE_2781_RX19_FLOWCELL2_hq.fasta\n",
      "[main] Real time: 28.880 sec; CPU: 27.598 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE_2781_RX19_FLOWCELL2_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE_2781_RX19_FLOWCELL2_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FORE_2781_RX19_FLOWCELL2\n",
      "+ poretools_dir=FORE_2781_RX19_FLOWCELL2\n",
      "+ sample_tag=FORE_2781_RX19_FLOWCELL2_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE_2781_RX19_FLOWCELL2_hq.fasta\n",
      "+ samtools sort - EM_079517_FORE_2781_RX19_FLOWCELL2_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 7955 sequences (8581748 bp)...\n",
      "[M::mem_process_seqs] Processed 7955 reads in 30.565 CPU sec, 30.574 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE_2781_RX19_FLOWCELL2_hq.fasta\n",
      "[main] Real time: 31.821 sec; CPU: 30.650 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE_2781_RX19_FLOWCELL2_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE_2781_RX19_FLOWCELL2_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=REDC_01408\n",
      "+ poretools_dir=REDC_01408\n",
      "+ sample_tag=REDC_01408_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC_01408_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_REDC_01408_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 8302 sequences (9724116 bp)...\n",
      "[M::mem_process_seqs] Processed 8302 reads in 28.619 CPU sec, 28.620 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta REDC_01408_hq.fasta\n",
      "[main] Real time: 29.843 sec; CPU: 28.823 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC_01408_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_REDC_01408_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=CON10512\n",
      "+ poretools_dir=CON10512\n",
      "+ sample_tag=CON10512_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta CON10512_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_CON10512_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 6088 sequences (10001802 bp)...\n",
      "[M::process] read 989 sequences (1559866 bp)...\n",
      "[M::mem_process_seqs] Processed 6088 reads in 59.972 CPU sec, 59.972 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 989 reads in 7.013 CPU sec, 6.978 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta CON10512_hq.fasta\n",
      "[main] Real time: 67.252 sec; CPU: 67.042 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_CON10512_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_CON10512_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FCH_730\n",
      "+ poretools_dir=FCH_730\n",
      "+ sample_tag=FCH_730_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH_730_hq.fasta\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_FCH_730_hq_bwa.sorted\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5726 sequences (10002645 bp)...\n",
      "[M::process] read 1757 sequences (3055343 bp)...\n",
      "[M::mem_process_seqs] Processed 5726 reads in 43.559 CPU sec, 43.556 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 1757 reads in 11.983 CPU sec, 11.945 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FCH_730_hq.fasta\n",
      "[main] Real time: 56.015 sec; CPU: 55.629 sec\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FCH_730_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FCH_730\n",
      "+ poretools_dir=FCH_730\n",
      "+ sample_tag=FCH_730_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH_730_hq.fasta\n",
      "+ samtools sort - EM_079517_mut30_2_FCH_730_hq_bwa.sorted\n",
      "+ samtools view -bS -\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 5726 sequences (10002645 bp)...\n",
      "[M::process] read 1757 sequences (3055343 bp)...\n",
      "[M::mem_process_seqs] Processed 5726 reads in 44.201 CPU sec, 44.155 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 1757 reads in 12.334 CPU sec, 12.300 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FCH_730_hq.fasta\n",
      "[main] Real time: 57.113 sec; CPU: 56.726 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH_730_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FCH_730_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=027636\n",
      "+ poretools_dir=027636\n",
      "+ sample_tag=027636_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 027636_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_027636_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5760 sequences (10000487 bp)...\n",
      "[M::process] read 1771 sequences (3011775 bp)...\n",
      "[M::mem_process_seqs] Processed 5760 reads in 38.794 CPU sec, 38.804 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 1771 reads in 10.834 CPU sec, 10.802 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta 027636_hq.fasta\n",
      "[main] Real time: 50.137 sec; CPU: 49.739 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_027636_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_027636_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=027636\n",
      "+ poretools_dir=027636\n",
      "+ sample_tag=027636_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ samtools view -bS -\n",
      "+ samtools sort - EM_079517_mut30_2_027636_hq_bwa.sorted\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 027636_hq.fasta\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5760 sequences (10000487 bp)...\n",
      "[M::process] read 1771 sequences (3011775 bp)...\n",
      "[M::mem_process_seqs] Processed 5760 reads in 40.503 CPU sec, 40.496 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 1771 reads in 10.645 CPU sec, 10.619 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta 027636_hq.fasta\n",
      "[main] Real time: 51.627 sec; CPU: 51.192 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_027636_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_027636_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517\n",
      "+ sample=FORE_2781\n",
      "+ poretools_dir=FORE_2781\n",
      "+ sample_tag=FORE_2781_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE_2781_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_FORE_2781_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5982 sequences (10000549 bp)...\n",
      "[M::process] read 3147 sequences (5140250 bp)...\n",
      "[M::mem_process_seqs] Processed 5982 reads in 39.921 CPU sec, 39.893 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 3147 reads in 20.015 CPU sec, 19.990 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517.fasta FORE_2781_hq.fasta\n",
      "[main] Real time: 60.695 sec; CPU: 60.012 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE_2781_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517.fasta --bam EM_079517_FORE_2781_hq_marginalign.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ ref_prefix=EM_079517_mut30_2\n",
      "+ sample=FORE_2781\n",
      "+ poretools_dir=FORE_2781\n",
      "+ sample_tag=FORE_2781_hq\n",
      "+ second_batch=na\n",
      "+ read_type=hq\n",
      "+ bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE_2781_hq.fasta\n",
      "+ samtools view -bS -\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "+ samtools sort - EM_079517_mut30_2_FORE_2781_hq_bwa.sorted\n",
      "[bam_header_read] EOF marker is absent. The input is probably truncated.\n",
      "[M::process] read 5982 sequences (10000549 bp)...\n",
      "[M::process] read 3147 sequences (5140250 bp)...\n",
      "[M::mem_process_seqs] Processed 5982 reads in 44.219 CPU sec, 44.123 real sec\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "[M::mem_process_seqs] Processed 3147 reads in 19.897 CPU sec, 19.877 real sec\n",
      "[main] Version: 0.7.12-r1044\n",
      "[main] CMD: bwa mem -t 1 -x ont2d ../refs/EM_079517_mut30_2.fasta FORE_2781_hq.fasta\n",
      "[main] Real time: 64.920 sec; CPU: 64.270 sec\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE_2781_hq_bwa.sorted.bam\n",
      "+ count-errors.py -\n",
      "+ count-errors.py -\n",
      "+ expand-cigar.py --fasta ../refs/EM_079517_mut30_2.fasta --bam EM_079517_mut30_2_FORE_2781_hq_marginalign.sorted.bam\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "ROOTDIR=\"/mnt/ebola/ebov\"\n",
    "cd $ROOTDIR\n",
    "export PATH=`pwd`/bin/bwa:$PATH\n",
    "export PATH=$PATH:`pwd`/scripts\n",
    "export PATH=$PATH:`pwd`/bin/nanopore-scripts\n",
    "export PATH=$PATH:`pwd`/bin/samtools\n",
    "export PATH=$PATH:`pwd`/bin/marginAlign\n",
    "\n",
    "cd analysis-EMLab\n",
    "makecommands.py ../metadata/metadata.db EMLab qual.sh | parallel -j40\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "## After the raw files are produced, show statistics.\n",
    "\n",
    "%cd analysis-var6mer-aln5mer-reverse-marginalign-all"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!makecommands.py ../metadata/metadata.db EMLab consensus.sh | parallel -j40"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!dump_consensus.sh ../metadata/metadata.db 20151028"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!rm *oct28*\n",
    "!make -f ../scripts/tree.mk PREFIX=oct28 FASTA=consensus-20151028-tpr0.75.tagged.fasta THREADS=40 DIST=0.0002"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!make_stats_file.py ../metadata/metadata.db EMLab | intersection_vcf_stats.py /dev/stdin > tp_stats.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!head -10 tp_stats.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%load_ext rpy2.ipython"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%R\n",
    "stats=read.table(\"tp_stats.txt\", header=T, sep=\"\\t\")\n",
    "a = nrow(subset(stats, tag == 'np-new-filter_qual200-50' & TPR > 0.8))\n",
    "subset(stats, tag == 'np-new-filter_qual200-50' & TPR < 0.2)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%R\n",
    "\n",
    "library(ggplot2)\n",
    "stats=read.table(\"tp_stats.txt\", header=T, sep=\"\\t\")\n",
    "ggplot(stats, aes(x=TPR)) + geom_histogram() + facet_wrap(~tag)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
